{
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "hlUvXdBaO61T",
        "pycharm": {
          "name": "#%% md\n"
        }
      },
      "source": [
        "# Protein design pipeline\n",
        "\n",
        "This is a pipeline for de novo protein design, incorporating the EvoDiff diffusion model for sequence generation, and OmegaFold model for folding prediciton. This was adapted from their Github repos (see below) for the S2DS Summer 2024 course, Team Polyploy. It's been optimized to work in Google Colab.\n",
        "\n",
        "EvoDiff:\n",
        "\n",
        "Diffusion model for de novo generation of protein sequences (outputs are fastA files). It can generate proteins with disordered regions, which is an advatage in relation to structure-based models (like RFDiffusion), while maintaining the capabiling of scaffold motif design.\n",
        "\n",
        "https://github.com/microsoft/evodiff\n",
        "\n",
        "\n",
        "OmegaFold:\n",
        "\n",
        "The first computational method to successfully predict high-resolution protein structure from a single primary sequence alone. Using a new combination of a protein language model that allows us to make predictions from single sequences and a geometry-inspired transformer model trained on protein structures, OmegaFold outperforms RoseTTAFold and achieves similar prediction accuracy to AlphaFold2 on recently released structures.\n",
        "\n",
        "https://github.com/HeliXonProtein/OmegaFold\n",
        "\n",
        "\n",
        "\n",
        "## Connecting to S2DS Gihub Repo (~2min)\n",
        "\n",
        "Cloning github repo is not stricly necessary, unless you want to access specific files other than this script. But the script itself can run on it's own without access to other files in the github repo. Can skip to \"Get reference protein\" if not cloning the repo.\n",
        "\n",
        "To clone the git repo, use private ssh keys to access it. The key generation step only needs to be done once (1st and 2nd cell; now commented out)."
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "izR6BQ2aa9rf"
      },
      "outputs": [],
      "source": [
        "# #first command generates key rsa 4096 long. Puts your email at the end of it, save the public key to id_rsa and does all of this with no passphrase being asked\n",
        "# !ssh-keygen -t rsa -b 4096 -C \"youremail\" -f /root/.ssh/id_rsa -N \"\"\n",
        "\n",
        "# #this command scans GitHub's SSH key and adds it to the known_hosts file, which is used to verify the identity of the remote server.\n",
        "# !ssh-keyscan -t rsa github.com >> ~/.ssh/known_hosts\n",
        "\n",
        "# #print the private key that you need to copy and paste in github (the key is the last line in the output which starts with \"ssh-rsa\" and ends with your email included)\n",
        "# !cat /root/.ssh/id_rsa.pub\n"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "ypG-6atJFInp"
      },
      "source": [
        "Add the public SSH key to GitHub (one-time setup):\n",
        "1. go to your GitHub account settings and navigate to the “SSH and GPG keys” section\n",
        "2. Click “New SSH key” and provide a name for your key, e.g. my_colab_key\n",
        "3. Copy the key that appears as the last line in the output of the previous cell (starts with \"ssh-rsa\" and ends with your email) and paste it into the key text box on GitHub\n",
        "4. Click “Add SSH key” and refresh github\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "fXa5-V7ZF1DG",
        "outputId": "869b6d50-8f0a-44ee-af8f-8a94b04039da"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Mounted at /content/drive\n",
            "id_rsa\tid_rsa.pub  known_hosts\n"
          ]
        }
      ],
      "source": [
        "# from google.colab import drive\n",
        "# drive.mount('/content/drive')\n",
        "\n",
        "# !mkdir /content/drive/MyDrive/ssh_colab\n",
        "# !cp /root/.ssh/* /content/drive/MyDrive/ssh_colab\n",
        "# !ls /content/drive/MyDrive/ssh_colab\n"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "2kpU6wruQTCz"
      },
      "source": [
        "Now from google drive we export the key to connect google colab with github, which will allow use of desired git commands (git-clone, git-commit, etc) and access to all files in the github repo through the google colab."
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 1,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "-sxeIKj-KFby",
        "outputId": "8eecc331-3aa8-4ec3-dba3-b25a7e99ae26"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Mounted at /content/drive\n"
          ]
        }
      ],
      "source": [
        "from google.colab import drive\n",
        "drive.mount('/content/drive')\n",
        "\n",
        "!mkdir /root/.ssh/\n",
        "!cp /content/drive/MyDrive/ssh_colab/* /root/.ssh/"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "Fzk8sGoFDnCb"
      },
      "source": [
        "Note: An alternative to this Google Drive SSH authentication method, which has been tested and shown to work, can be found here, if desired:\n",
        "https://felixbmuller.medium.com/connect-a-private-github-repository-with-google-colab-via-a-deploy-key-cca8ad13007\n"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "Z1Eya6qnTFD7"
      },
      "source": [
        "Git clone the repository and switch to your branch"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 2,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "U5KWa5PvvI69",
        "outputId": "16d480cb-8209-4783-9a9d-3147bc8d8075"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "S2DS Polyploy directory does not exist, cloning from github\n",
            "Cloning into 'S2DS-Summer24-Polyploy'...\n",
            "remote: Enumerating objects: 1896, done.\u001b[K\n",
            "remote: Counting objects: 100% (681/681), done.\u001b[K\n",
            "remote: Compressing objects: 100% (286/286), done.\u001b[K\n",
            "remote: Total 1896 (delta 477), reused 531 (delta 392), pack-reused 1215\u001b[K\n",
            "Receiving objects: 100% (1896/1896), 118.11 MiB | 21.05 MiB/s, done.\n",
            "Resolving deltas: 100% (1021/1021), done.\n",
            "Updating files: 100% (332/332), done.\n"
          ]
        }
      ],
      "source": [
        "import os\n",
        "\n",
        "#Only clone the repository if it doesn't already exist (to avoid duplicates/complications)\n",
        "if ~os.path.isdir(\"S2DS-Summer24-Polyploy\"):\n",
        "    print('S2DS Polyploy directory does not exist, cloning from github')\n",
        "\n",
        "    !git clone git@github.com:S2DSLondon/S2DS-Summer24-Polyploy.git\n"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "X_bzX3D1vr_j"
      },
      "source": [
        "Switch to the branch you would like to work in"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 3,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "6Fqj0OMycS8z",
        "outputId": "6000ca7b-1a0a-4c00-9856-5374c824ce74"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "/content/S2DS-Summer24-Polyploy\n",
            "Branch 'public' set up to track remote branch 'public' from 'origin'.\n",
            "Switched to a new branch 'public'\n"
          ]
        }
      ],
      "source": [
        "%cd S2DS-Summer24-Polyploy\n",
        "#change to branch of interest\n",
        "!git checkout public"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "-mtawkllO61X",
        "pycharm": {
          "name": "#%% md\n"
        }
      },
      "source": [
        "# Get reference protein"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [],
      "metadata": {
        "id": "Xb_2u0_diVWi"
      }
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "NL0Q-qFoXEvb"
      },
      "source": [
        "\n",
        "We need to provide a pdb file of the protein from where our motif is kept. Below we will automatically dowload it and save it to the desired directory."
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 24,
      "metadata": {
        "id": "U_NIN3CGO61Y",
        "pycharm": {
          "name": "#%%\n"
        },
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "b086699c-ab77-4ff9-c064-7c9dd1d877f6"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "PDB file 1SMD downloaded successfully to data/external/pdb/scaffolding-pdbs/1SMD.pdb\n",
            "PDB file found: data/external/pdb/scaffolding-pdbs/1SMD.pdb\n"
          ]
        }
      ],
      "source": [
        "import os\n",
        "import requests\n",
        "\n",
        "def download_pdb(pdb_code, pdb_file_path):\n",
        "    url = f\"https://files.rcsb.org/download/{pdb_code.upper()}.pdb\"\n",
        "    response = requests.get(url)\n",
        "\n",
        "    if response.status_code == 200:\n",
        "        os.makedirs(os.path.dirname(pdb_file_path), exist_ok=True)  # Create directories if not present\n",
        "        with open(pdb_file_path, 'wb') as f:\n",
        "            f.write(response.content)\n",
        "        print(f\"PDB file {pdb_code} downloaded successfully to {pdb_file_path}\")\n",
        "    else:\n",
        "        print(f\"Failed to download PDB file {pdb_code}. Status code: {response.status_code}\")\n",
        "\n",
        "# Variables to define for evodiff\n",
        "pdb_code = '1SMD'  # @param {type:\"string\"}\n",
        "num_seqs = 1  # @param {type:\"integer\"}\n",
        "data_top_dir = 'data/external/pdb/'\n",
        "\n",
        "# Variables to define for extracting motifs\n",
        "pdb_file_name = f'{pdb_code}.pdb'\n",
        "pdb_file_path = os.path.join(data_top_dir, 'scaffolding-pdbs', pdb_file_name)\n",
        "\n",
        "# Download the PDB file from online\n",
        "download_pdb(pdb_code, pdb_file_path)\n",
        "\n",
        "# Check if the file exists\n",
        "if os.path.exists(pdb_file_path):\n",
        "    print(f\"PDB file found: {pdb_file_path}\")\n",
        "else:\n",
        "    print(f\"PDB file not found: {pdb_file_path}\")\n"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "TQCN5D9KBgso"
      },
      "source": [
        " Important: Here you need to inspect the PBD file to see if the first residues starts at 1 or not. If it doesnt then we need to add an offset so the correct residues are extracted. At the moment this needs to be done manually, but in future this could be automated.\n",
        "\n",
        " Provide the start residues for the motif you are interested in scaffolding (the end residue is inclusive) and the offset if there is one. The adjusted start/end residue numbers for evodiff should be the motif site start/end minus the offset.\n",
        "\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 25,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "VEnSMr5VBgsp",
        "outputId": "627cf84c-6405-445f-f1dd-023e1ae4f748"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Adjusted start_residue_number for evodiff: 196\n",
            "Adjusted end_residue_number for evodiff: 217\n"
          ]
        }
      ],
      "source": [
        "# Function to create adjusted indices based on the number of motifs\n",
        "def adjust_indices(motif_start, motif_end, offset):\n",
        "    start_idx = motif_start - offset\n",
        "    end_idx = motif_end - offset\n",
        "    return start_idx, end_idx\n",
        "\n",
        "# Parameters\n",
        "motif_start = 212  # @param {type:\"integer\"}\n",
        "motif_end = 233    # @param {type:\"integer\"}\n",
        "offset = 16        # @param {type:\"integer\"}\n",
        "\n",
        "# Get adjusted indices\n",
        "start_idx, end_idx = adjust_indices(motif_start, motif_end, offset)\n",
        "\n",
        "print(f\"Adjusted start_residue_number for evodiff: {start_idx}\")\n",
        "print(f\"Adjusted end_residue_number for evodiff: {end_idx}\")\n",
        "\n"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "vBsNeAtIBgsp"
      },
      "source": [
        "Now we will extract the motif to be kept as a pdb file, for later comparison with the generated protein."
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 26,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "OKwLw_7NBgsp",
        "outputId": "b9a2f99f-e382-4088-bc1c-d12258a77c49"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Requirement already satisfied: biopython in /usr/local/lib/python3.10/dist-packages (1.81)\n",
            "Requirement already satisfied: numpy in /usr/local/lib/python3.10/dist-packages (from biopython) (1.26.4)\n",
            "Extracted PDB file saved to: data/interim/test_extracted_motif.pdb\n",
            "Motif sequence: DKLHNLNSNWFPEGSKPFIYQE\n"
          ]
        }
      ],
      "source": [
        "# Install Biopython if not already installed\n",
        "!pip install biopython\n",
        "\n",
        "from Bio import PDB\n",
        "from Bio.SeqUtils import seq1\n",
        "import os\n",
        "import warnings\n",
        "warnings.simplefilter('ignore', PDB.PDBExceptions.PDBConstructionWarning)\n",
        "\n",
        "# Parse the PDB file\n",
        "parser = PDB.PDBParser()\n",
        "\n",
        "# Parse the PDB file\n",
        "structure = parser.get_structure('protein', pdb_file_path)\n",
        "\n",
        "# Create a new structure object to store the extracted residues\n",
        "new_structure = PDB.Structure.Structure('new_structure')\n",
        "\n",
        "# Store the sequence of the extracted motif\n",
        "motif_sequence = []\n",
        "\n",
        "for model in structure:\n",
        "    new_model = PDB.Model.Model(model.id)  # Create a new model\n",
        "    for chain in model:\n",
        "        new_chain = PDB.Chain.Chain('A')  # Create a new chain (adjust 'A' as needed)\n",
        "        for residue in chain.get_residues():\n",
        "            residue_id = residue.get_id()[1]  # Get the residue number\n",
        "            if motif_start <= residue_id <= motif_end:\n",
        "                # Create a new residue object and add it to the new chain\n",
        "                new_residue = PDB.Residue.Residue(residue.get_id(), residue.resname, residue.segid)\n",
        "                for atom in residue:\n",
        "                    new_residue.add(atom)\n",
        "                new_chain.add(new_residue)\n",
        "\n",
        "                # Append the residue's one-letter code to the motif sequence\n",
        "                motif_sequence.append(seq1(residue.resname))\n",
        "\n",
        "        # Add the new chain to the new model\n",
        "        new_model.add(new_chain)\n",
        "\n",
        "    # Add the new model to the new structure\n",
        "    new_structure.add(new_model)\n",
        "# Define the path to the output PDB file\n",
        "output_pdb_file = 'data/interim/test_extracted_motif.pdb' #@param {type:\"string\"}\n",
        "\n",
        "# Create the output directory if it doesn't exist\n",
        "os.makedirs(os.path.dirname(output_pdb_file), exist_ok=True)\n",
        "\n",
        "# Save the new structure to a PDB file\n",
        "io = PDB.PDBIO()\n",
        "io.set_structure(new_structure)\n",
        "io.save(output_pdb_file)\n",
        "\n",
        "# Convert the motif sequence list to a string\n",
        "motif_sequence_str = ''.join(motif_sequence)\n",
        "\n",
        "print(f\"Extracted PDB file saved to: {output_pdb_file}\")\n",
        "print(f\"Motif sequence: {motif_sequence_str}\")\n"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "# EvoDiff\n",
        "\n",
        "\n",
        "First step is to generate a protein sequence (fastA file), using the Evodiff diffusion model.\n",
        "\n",
        "Install required libraries as below:"
      ],
      "metadata": {
        "id": "qT1LTBkkacLE"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "!pip install evodiff\n",
        "!pip install git+https://github.com/microsoft/evodiff.git\n",
        "\n",
        "!pip install torch\n",
        "!pip install torch-geometric\n",
        "!pip install torch-scatter -f https://data.pyg.org/whl/torch-2.3.0+cpu.html\n",
        "\n"
      ],
      "metadata": {
        "id": "-4xoAo3JaXod",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "f7175d20-048e-478b-e167-5ae81a9b8019"
      },
      "execution_count": 23,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Requirement already satisfied: evodiff in /usr/local/lib/python3.10/dist-packages (1.1.0)\n",
            "Requirement already satisfied: pandas in /usr/local/lib/python3.10/dist-packages (from evodiff) (2.1.4)\n",
            "Requirement already satisfied: lmdb in /usr/local/lib/python3.10/dist-packages (from evodiff) (1.5.1)\n",
            "Requirement already satisfied: numpy in /usr/local/lib/python3.10/dist-packages (from evodiff) (1.26.4)\n",
            "Requirement already satisfied: sequence-models in /usr/local/lib/python3.10/dist-packages (from evodiff) (1.8.0)\n",
            "Requirement already satisfied: mlflow in /usr/local/lib/python3.10/dist-packages (from evodiff) (2.15.0)\n",
            "Requirement already satisfied: scikit-learn in /usr/local/lib/python3.10/dist-packages (from evodiff) (1.3.2)\n",
            "Requirement already satisfied: blosum in /usr/local/lib/python3.10/dist-packages (from evodiff) (2.0.3)\n",
            "Requirement already satisfied: seaborn in /usr/local/lib/python3.10/dist-packages (from evodiff) (0.13.1)\n",
            "Requirement already satisfied: matplotlib in /usr/local/lib/python3.10/dist-packages (from evodiff) (3.7.1)\n",
            "Requirement already satisfied: fair-esm in /usr/local/lib/python3.10/dist-packages (from evodiff) (2.0.0)\n",
            "Requirement already satisfied: tqdm in /usr/local/lib/python3.10/dist-packages (from evodiff) (4.66.4)\n",
            "Requirement already satisfied: biotite in /usr/local/lib/python3.10/dist-packages (from evodiff) (0.41.2)\n",
            "Requirement already satisfied: requests in /usr/local/lib/python3.10/dist-packages (from evodiff) (2.31.0)\n",
            "Requirement already satisfied: mdanalysis in /usr/local/lib/python3.10/dist-packages (from evodiff) (2.7.0)\n",
            "Requirement already satisfied: pdb-tools in /usr/local/lib/python3.10/dist-packages (from evodiff) (2.5.0)\n",
            "Requirement already satisfied: msgpack>=0.5.6 in /usr/local/lib/python3.10/dist-packages (from biotite->evodiff) (1.0.8)\n",
            "Requirement already satisfied: networkx>=2.0 in /usr/local/lib/python3.10/dist-packages (from biotite->evodiff) (3.3)\n",
            "Requirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.10/dist-packages (from requests->evodiff) (3.3.2)\n",
            "Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.10/dist-packages (from requests->evodiff) (3.7)\n",
            "Requirement already satisfied: urllib3<3,>=1.21.1 in /usr/local/lib/python3.10/dist-packages (from requests->evodiff) (2.0.7)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.10/dist-packages (from requests->evodiff) (2024.7.4)\n",
            "Requirement already satisfied: contourpy>=1.0.1 in /usr/local/lib/python3.10/dist-packages (from matplotlib->evodiff) (1.2.1)\n",
            "Requirement already satisfied: cycler>=0.10 in /usr/local/lib/python3.10/dist-packages (from matplotlib->evodiff) (0.12.1)\n",
            "Requirement already satisfied: fonttools>=4.22.0 in /usr/local/lib/python3.10/dist-packages (from matplotlib->evodiff) (4.53.1)\n",
            "Requirement already satisfied: kiwisolver>=1.0.1 in /usr/local/lib/python3.10/dist-packages (from matplotlib->evodiff) (1.4.5)\n",
            "Requirement already satisfied: packaging>=20.0 in /usr/local/lib/python3.10/dist-packages (from matplotlib->evodiff) (24.1)\n",
            "Requirement already satisfied: pillow>=6.2.0 in /usr/local/lib/python3.10/dist-packages (from matplotlib->evodiff) (9.4.0)\n",
            "Requirement already satisfied: pyparsing>=2.3.1 in /usr/local/lib/python3.10/dist-packages (from matplotlib->evodiff) (3.1.2)\n",
            "Requirement already satisfied: python-dateutil>=2.7 in /usr/local/lib/python3.10/dist-packages (from matplotlib->evodiff) (2.8.2)\n",
            "Requirement already satisfied: GridDataFormats>=0.4.0 in /usr/local/lib/python3.10/dist-packages (from mdanalysis->evodiff) (1.0.2)\n",
            "Requirement already satisfied: mmtf-python>=1.0.0 in /usr/local/lib/python3.10/dist-packages (from mdanalysis->evodiff) (1.1.3)\n",
            "Requirement already satisfied: joblib>=0.12 in /usr/local/lib/python3.10/dist-packages (from mdanalysis->evodiff) (1.4.2)\n",
            "Requirement already satisfied: scipy>=1.5.0 in /usr/local/lib/python3.10/dist-packages (from mdanalysis->evodiff) (1.13.1)\n",
            "Requirement already satisfied: threadpoolctl in /usr/local/lib/python3.10/dist-packages (from mdanalysis->evodiff) (3.5.0)\n",
            "Requirement already satisfied: fasteners in /usr/local/lib/python3.10/dist-packages (from mdanalysis->evodiff) (0.19)\n",
            "Requirement already satisfied: mda-xdrlib in /usr/local/lib/python3.10/dist-packages (from mdanalysis->evodiff) (0.2.0)\n",
            "Requirement already satisfied: mlflow-skinny==2.15.0 in /usr/local/lib/python3.10/dist-packages (from mlflow->evodiff) (2.15.0)\n",
            "Requirement already satisfied: Flask<4 in /usr/local/lib/python3.10/dist-packages (from mlflow->evodiff) (2.2.5)\n",
            "Requirement already satisfied: alembic!=1.10.0,<2 in /usr/local/lib/python3.10/dist-packages (from mlflow->evodiff) (1.13.2)\n",
            "Requirement already satisfied: docker<8,>=4.0.0 in /usr/local/lib/python3.10/dist-packages (from mlflow->evodiff) (7.1.0)\n",
            "Requirement already satisfied: graphene<4 in /usr/local/lib/python3.10/dist-packages (from mlflow->evodiff) (3.3)\n",
            "Requirement already satisfied: markdown<4,>=3.3 in /usr/local/lib/python3.10/dist-packages (from mlflow->evodiff) (3.6)\n",
            "Requirement already satisfied: pyarrow<16,>=4.0.0 in /usr/local/lib/python3.10/dist-packages (from mlflow->evodiff) (14.0.2)\n",
            "Requirement already satisfied: querystring-parser<2 in /usr/local/lib/python3.10/dist-packages (from mlflow->evodiff) (1.2.4)\n",
            "Requirement already satisfied: sqlalchemy<3,>=1.4.0 in /usr/local/lib/python3.10/dist-packages (from mlflow->evodiff) (2.0.31)\n",
            "Requirement already satisfied: Jinja2<4,>=2.11 in /usr/local/lib/python3.10/dist-packages (from mlflow->evodiff) (3.1.4)\n",
            "Requirement already satisfied: gunicorn<23 in /usr/local/lib/python3.10/dist-packages (from mlflow->evodiff) (22.0.0)\n",
            "Requirement already satisfied: cachetools<6,>=5.0.0 in /usr/local/lib/python3.10/dist-packages (from mlflow-skinny==2.15.0->mlflow->evodiff) (5.4.0)\n",
            "Requirement already satisfied: click<9,>=7.0 in /usr/local/lib/python3.10/dist-packages (from mlflow-skinny==2.15.0->mlflow->evodiff) (8.1.7)\n",
            "Requirement already satisfied: cloudpickle<4 in /usr/local/lib/python3.10/dist-packages (from mlflow-skinny==2.15.0->mlflow->evodiff) (2.2.1)\n",
            "Requirement already satisfied: databricks-sdk<1,>=0.20.0 in /usr/local/lib/python3.10/dist-packages (from mlflow-skinny==2.15.0->mlflow->evodiff) (0.29.0)\n",
            "Requirement already satisfied: entrypoints<1 in /usr/local/lib/python3.10/dist-packages (from mlflow-skinny==2.15.0->mlflow->evodiff) (0.4)\n",
            "Requirement already satisfied: gitpython<4,>=3.1.9 in /usr/local/lib/python3.10/dist-packages (from mlflow-skinny==2.15.0->mlflow->evodiff) (3.1.43)\n",
            "Requirement already satisfied: importlib-metadata!=4.7.0,<8,>=3.7.0 in /usr/local/lib/python3.10/dist-packages (from mlflow-skinny==2.15.0->mlflow->evodiff) (7.2.1)\n",
            "Requirement already satisfied: opentelemetry-api<3,>=1.9.0 in /usr/local/lib/python3.10/dist-packages (from mlflow-skinny==2.15.0->mlflow->evodiff) (1.26.0)\n",
            "Requirement already satisfied: opentelemetry-sdk<3,>=1.9.0 in /usr/local/lib/python3.10/dist-packages (from mlflow-skinny==2.15.0->mlflow->evodiff) (1.26.0)\n",
            "Requirement already satisfied: protobuf<6,>=3.12.0 in /usr/local/lib/python3.10/dist-packages (from mlflow-skinny==2.15.0->mlflow->evodiff) (3.20.3)\n",
            "Requirement already satisfied: pytz<2025 in /usr/local/lib/python3.10/dist-packages (from mlflow-skinny==2.15.0->mlflow->evodiff) (2024.1)\n",
            "Requirement already satisfied: pyyaml<7,>=5.1 in /usr/local/lib/python3.10/dist-packages (from mlflow-skinny==2.15.0->mlflow->evodiff) (6.0.1)\n",
            "Requirement already satisfied: sqlparse<1,>=0.4.0 in /usr/local/lib/python3.10/dist-packages (from mlflow-skinny==2.15.0->mlflow->evodiff) (0.5.1)\n",
            "Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.10/dist-packages (from pandas->evodiff) (2024.1)\n",
            "Requirement already satisfied: Mako in /usr/local/lib/python3.10/dist-packages (from alembic!=1.10.0,<2->mlflow->evodiff) (1.3.5)\n",
            "Requirement already satisfied: typing-extensions>=4 in /usr/local/lib/python3.10/dist-packages (from alembic!=1.10.0,<2->mlflow->evodiff) (4.12.2)\n",
            "Requirement already satisfied: Werkzeug>=2.2.2 in /usr/local/lib/python3.10/dist-packages (from Flask<4->mlflow->evodiff) (3.0.3)\n",
            "Requirement already satisfied: itsdangerous>=2.0 in /usr/local/lib/python3.10/dist-packages (from Flask<4->mlflow->evodiff) (2.2.0)\n",
            "Requirement already satisfied: graphql-core<3.3,>=3.1 in /usr/local/lib/python3.10/dist-packages (from graphene<4->mlflow->evodiff) (3.2.3)\n",
            "Requirement already satisfied: graphql-relay<3.3,>=3.1 in /usr/local/lib/python3.10/dist-packages (from graphene<4->mlflow->evodiff) (3.2.0)\n",
            "Requirement already satisfied: aniso8601<10,>=8 in /usr/local/lib/python3.10/dist-packages (from graphene<4->mlflow->evodiff) (9.0.1)\n",
            "Requirement already satisfied: mrcfile in /usr/local/lib/python3.10/dist-packages (from GridDataFormats>=0.4.0->mdanalysis->evodiff) (1.5.3)\n",
            "Requirement already satisfied: MarkupSafe>=2.0 in /usr/local/lib/python3.10/dist-packages (from Jinja2<4,>=2.11->mlflow->evodiff) (2.1.5)\n",
            "Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.10/dist-packages (from python-dateutil>=2.7->matplotlib->evodiff) (1.16.0)\n",
            "Requirement already satisfied: greenlet!=0.4.17 in /usr/local/lib/python3.10/dist-packages (from sqlalchemy<3,>=1.4.0->mlflow->evodiff) (3.0.3)\n",
            "Requirement already satisfied: google-auth~=2.0 in /usr/local/lib/python3.10/dist-packages (from databricks-sdk<1,>=0.20.0->mlflow-skinny==2.15.0->mlflow->evodiff) (2.27.0)\n",
            "Requirement already satisfied: gitdb<5,>=4.0.1 in /usr/local/lib/python3.10/dist-packages (from gitpython<4,>=3.1.9->mlflow-skinny==2.15.0->mlflow->evodiff) (4.0.11)\n",
            "Requirement already satisfied: zipp>=0.5 in /usr/local/lib/python3.10/dist-packages (from importlib-metadata!=4.7.0,<8,>=3.7.0->mlflow-skinny==2.15.0->mlflow->evodiff) (3.19.2)\n",
            "Requirement already satisfied: deprecated>=1.2.6 in /usr/local/lib/python3.10/dist-packages (from opentelemetry-api<3,>=1.9.0->mlflow-skinny==2.15.0->mlflow->evodiff) (1.2.14)\n",
            "Requirement already satisfied: opentelemetry-semantic-conventions==0.47b0 in /usr/local/lib/python3.10/dist-packages (from opentelemetry-sdk<3,>=1.9.0->mlflow-skinny==2.15.0->mlflow->evodiff) (0.47b0)\n",
            "Requirement already satisfied: wrapt<2,>=1.10 in /usr/local/lib/python3.10/dist-packages (from deprecated>=1.2.6->opentelemetry-api<3,>=1.9.0->mlflow-skinny==2.15.0->mlflow->evodiff) (1.16.0)\n",
            "Requirement already satisfied: smmap<6,>=3.0.1 in /usr/local/lib/python3.10/dist-packages (from gitdb<5,>=4.0.1->gitpython<4,>=3.1.9->mlflow-skinny==2.15.0->mlflow->evodiff) (5.0.1)\n",
            "Requirement already satisfied: pyasn1-modules>=0.2.1 in /usr/local/lib/python3.10/dist-packages (from google-auth~=2.0->databricks-sdk<1,>=0.20.0->mlflow-skinny==2.15.0->mlflow->evodiff) (0.4.0)\n",
            "Requirement already satisfied: rsa<5,>=3.1.4 in /usr/local/lib/python3.10/dist-packages (from google-auth~=2.0->databricks-sdk<1,>=0.20.0->mlflow-skinny==2.15.0->mlflow->evodiff) (4.9)\n",
            "Requirement already satisfied: pyasn1<0.7.0,>=0.4.6 in /usr/local/lib/python3.10/dist-packages (from pyasn1-modules>=0.2.1->google-auth~=2.0->databricks-sdk<1,>=0.20.0->mlflow-skinny==2.15.0->mlflow->evodiff) (0.6.0)\n",
            "Collecting git+https://github.com/microsoft/evodiff.git\n",
            "  Cloning https://github.com/microsoft/evodiff.git to /tmp/pip-req-build-4qtan4ou\n",
            "  Running command git clone --filter=blob:none --quiet https://github.com/microsoft/evodiff.git /tmp/pip-req-build-4qtan4ou\n",
            "  Resolved https://github.com/microsoft/evodiff.git to commit f813abd06baa3e31098b46c67dd2d3f0cbc65b7d\n",
            "  Installing build dependencies ... \u001b[?25l\u001b[?25hdone\n",
            "  Getting requirements to build wheel ... \u001b[?25l\u001b[?25hdone\n",
            "  Preparing metadata (pyproject.toml) ... \u001b[?25l\u001b[?25hdone\n",
            "Requirement already satisfied: torch in /usr/local/lib/python3.10/dist-packages (2.3.1+cu121)\n",
            "Requirement already satisfied: filelock in /usr/local/lib/python3.10/dist-packages (from torch) (3.15.4)\n",
            "Requirement already satisfied: typing-extensions>=4.8.0 in /usr/local/lib/python3.10/dist-packages (from torch) (4.12.2)\n",
            "Requirement already satisfied: sympy in /usr/local/lib/python3.10/dist-packages (from torch) (1.13.1)\n",
            "Requirement already satisfied: networkx in /usr/local/lib/python3.10/dist-packages (from torch) (3.3)\n",
            "Requirement already satisfied: jinja2 in /usr/local/lib/python3.10/dist-packages (from torch) (3.1.4)\n",
            "Requirement already satisfied: fsspec in /usr/local/lib/python3.10/dist-packages (from torch) (2024.6.1)\n",
            "Requirement already satisfied: nvidia-cuda-nvrtc-cu12==12.1.105 in /usr/local/lib/python3.10/dist-packages (from torch) (12.1.105)\n",
            "Requirement already satisfied: nvidia-cuda-runtime-cu12==12.1.105 in /usr/local/lib/python3.10/dist-packages (from torch) (12.1.105)\n",
            "Requirement already satisfied: nvidia-cuda-cupti-cu12==12.1.105 in /usr/local/lib/python3.10/dist-packages (from torch) (12.1.105)\n",
            "Requirement already satisfied: nvidia-cudnn-cu12==8.9.2.26 in /usr/local/lib/python3.10/dist-packages (from torch) (8.9.2.26)\n",
            "Requirement already satisfied: nvidia-cublas-cu12==12.1.3.1 in /usr/local/lib/python3.10/dist-packages (from torch) (12.1.3.1)\n",
            "Requirement already satisfied: nvidia-cufft-cu12==11.0.2.54 in /usr/local/lib/python3.10/dist-packages (from torch) (11.0.2.54)\n",
            "Requirement already satisfied: nvidia-curand-cu12==10.3.2.106 in /usr/local/lib/python3.10/dist-packages (from torch) (10.3.2.106)\n",
            "Requirement already satisfied: nvidia-cusolver-cu12==11.4.5.107 in /usr/local/lib/python3.10/dist-packages (from torch) (11.4.5.107)\n",
            "Requirement already satisfied: nvidia-cusparse-cu12==12.1.0.106 in /usr/local/lib/python3.10/dist-packages (from torch) (12.1.0.106)\n",
            "Requirement already satisfied: nvidia-nccl-cu12==2.20.5 in /usr/local/lib/python3.10/dist-packages (from torch) (2.20.5)\n",
            "Requirement already satisfied: nvidia-nvtx-cu12==12.1.105 in /usr/local/lib/python3.10/dist-packages (from torch) (12.1.105)\n",
            "Requirement already satisfied: triton==2.3.1 in /usr/local/lib/python3.10/dist-packages (from torch) (2.3.1)\n",
            "Requirement already satisfied: nvidia-nvjitlink-cu12 in /usr/local/lib/python3.10/dist-packages (from nvidia-cusolver-cu12==11.4.5.107->torch) (12.5.82)\n",
            "Requirement already satisfied: MarkupSafe>=2.0 in /usr/local/lib/python3.10/dist-packages (from jinja2->torch) (2.1.5)\n",
            "Requirement already satisfied: mpmath<1.4,>=1.1.0 in /usr/local/lib/python3.10/dist-packages (from sympy->torch) (1.3.0)\n",
            "Requirement already satisfied: torch-geometric in /usr/local/lib/python3.10/dist-packages (2.5.3)\n",
            "Requirement already satisfied: tqdm in /usr/local/lib/python3.10/dist-packages (from torch-geometric) (4.66.4)\n",
            "Requirement already satisfied: numpy in /usr/local/lib/python3.10/dist-packages (from torch-geometric) (1.26.4)\n",
            "Requirement already satisfied: scipy in /usr/local/lib/python3.10/dist-packages (from torch-geometric) (1.13.1)\n",
            "Requirement already satisfied: fsspec in /usr/local/lib/python3.10/dist-packages (from torch-geometric) (2024.6.1)\n",
            "Requirement already satisfied: jinja2 in /usr/local/lib/python3.10/dist-packages (from torch-geometric) (3.1.4)\n",
            "Requirement already satisfied: aiohttp in /usr/local/lib/python3.10/dist-packages (from torch-geometric) (3.9.5)\n",
            "Requirement already satisfied: requests in /usr/local/lib/python3.10/dist-packages (from torch-geometric) (2.31.0)\n",
            "Requirement already satisfied: pyparsing in /usr/local/lib/python3.10/dist-packages (from torch-geometric) (3.1.2)\n",
            "Requirement already satisfied: scikit-learn in /usr/local/lib/python3.10/dist-packages (from torch-geometric) (1.3.2)\n",
            "Requirement already satisfied: psutil>=5.8.0 in /usr/local/lib/python3.10/dist-packages (from torch-geometric) (5.9.5)\n",
            "Requirement already satisfied: aiosignal>=1.1.2 in /usr/local/lib/python3.10/dist-packages (from aiohttp->torch-geometric) (1.3.1)\n",
            "Requirement already satisfied: attrs>=17.3.0 in /usr/local/lib/python3.10/dist-packages (from aiohttp->torch-geometric) (23.2.0)\n",
            "Requirement already satisfied: frozenlist>=1.1.1 in /usr/local/lib/python3.10/dist-packages (from aiohttp->torch-geometric) (1.4.1)\n",
            "Requirement already satisfied: multidict<7.0,>=4.5 in /usr/local/lib/python3.10/dist-packages (from aiohttp->torch-geometric) (6.0.5)\n",
            "Requirement already satisfied: yarl<2.0,>=1.0 in /usr/local/lib/python3.10/dist-packages (from aiohttp->torch-geometric) (1.9.4)\n",
            "Requirement already satisfied: async-timeout<5.0,>=4.0 in /usr/local/lib/python3.10/dist-packages (from aiohttp->torch-geometric) (4.0.3)\n",
            "Requirement already satisfied: MarkupSafe>=2.0 in /usr/local/lib/python3.10/dist-packages (from jinja2->torch-geometric) (2.1.5)\n",
            "Requirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.10/dist-packages (from requests->torch-geometric) (3.3.2)\n",
            "Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.10/dist-packages (from requests->torch-geometric) (3.7)\n",
            "Requirement already satisfied: urllib3<3,>=1.21.1 in /usr/local/lib/python3.10/dist-packages (from requests->torch-geometric) (2.0.7)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.10/dist-packages (from requests->torch-geometric) (2024.7.4)\n",
            "Requirement already satisfied: joblib>=1.1.1 in /usr/local/lib/python3.10/dist-packages (from scikit-learn->torch-geometric) (1.4.2)\n",
            "Requirement already satisfied: threadpoolctl>=2.0.0 in /usr/local/lib/python3.10/dist-packages (from scikit-learn->torch-geometric) (3.5.0)\n",
            "Looking in links: https://data.pyg.org/whl/torch-2.3.0+cpu.html\n",
            "Requirement already satisfied: torch-scatter in /usr/local/lib/python3.10/dist-packages (2.1.2+pt23cpu)\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "ygX0GNR1O61Y",
        "pycharm": {
          "name": "#%% md\n"
        }
      },
      "source": [
        "\n",
        "\n",
        " We need to download model information from zenodo. For demonstration purposes, we show an example using the lightest model - 38M model here, and generation on a CPU.\n",
        "[EvoDiff has a bunch of different models, some require more memory and GPU.]\n",
        "\n",
        "\n",
        "Next, we run the generation and can specify a few things:\n",
        "\n",
        "* `scaffold_length` we want to generate. The code will randomly sample the location of the motif within the specified scaffold length.\n",
        "\n",
        "* number of sequences to be designed under: `batch_size`=1\n",
        "\n",
        "* whether to run on cpu/gpu etc under: `device`='cpu'\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 28,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "PsVgPV3aO61Y",
        "outputId": "15cc62a4-c529-4a64-8782-6b035154244a",
        "pycharm": {
          "name": "#%%\n"
        }
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "ALREADY DOWNLOADED\n",
            "CLEANING PDB\n",
            "WARNING: USING CHAIN A FROM PDB FILE\n",
            "sequence extracted from pdb YSSNTQQGRTSIVHLFEWRWVDIALECERYLAPKGFGGVQVSPPNENVAIHNPFRPWWERYQPVSYKLCTRSGNEDEFRNMVTRCNNVGVRIYVDAVINHMCGNAVSAGTSSTCGSYFNPGSRDFPAVPYSGWDFNDGKCKTGSGDIENYNDATQVRDCRLSGLLDLALGKDYVRSKIAEYMNHLIDIGVAGFRIDASKHMWPGDIKAILDKLHNLNSNWFPEGSKPFIYQEVIDLGGEPIKSSDYFGNGRVTEFKYGAKLGTVIRKWNGEKMSYLKNWGEGWGFMPSDRALVFVDNHDNQRGHGAGGASILTFWDARLYKMAVGFMLAHPYGFTRVMSSYRWPRYFENGKDVNDWVGPPNDNGVTKEVTINPDTTCGNDWVCEHRWRQIRNMVNFRNVVDGQPFTNWYDNGSNQVAFGRGNRGFIVFNNDDWTFSLTLQTGLPAGTYCDVISGDKINGNCTGIKIYVSDDGKAHFSISNSAEDPFIAIHAESKL\n",
            "sequence length 495\n",
            "motif extracted from indexes supplied: ASKHMWPGDIKAILDKLHNLNS\n",
            "Generated sequence: ['MNFSSKDRGEFLTFAAKCGKKHKRKAIFVTGSRGTRSDEMAGYEQFRGKPNAIIGAGIIFLDEASKHMWPGDIKAILDKLHNLNSTTLIKEKCIEVCLEDIPAYGRDSWKLLLEDGGCPRNSGTKTEAIHHILELIELEQHEERTQLISYGRNLGLDDIIYGSDSEEEKAEE', 'MDDQTIQKKIAASLLSLILTTYEANEADTLTEENKNPIIDDEWARLFYLGLAILGKLSVKHDSASKHMWPGDIKAILDKLHNLNSNILANFFLQHICTLGKGQQKIKALDSALKAFTATLDSEEPTYRLGKKSNSGSTKVKKGAAPKSKPVTQRVEEDNTRGIVSQNQATVP']\n",
            "motif start indices [63]\n",
            "motif end indices [84]\n"
          ]
        }
      ],
      "source": [
        "#load the model\n",
        "from evodiff.pretrained import OA_DM_38M\n",
        "\n",
        "checkpoint = OA_DM_38M()\n",
        "model, collater, tokenizer, scheme = checkpoint\n",
        "\n",
        "\n",
        "from evodiff.conditional_generation import generate_scaffold\n",
        "\n",
        "scaffold_length = 150 #@param {type:\"integer\"}\n",
        "batch_size = 2 #@param {type:\"integer\"}\n",
        "device = \"cpu\" #@param{type:\"string\"}\n",
        "\n",
        "# Make sure start_idx and end_idx are lists, even if they contain a single element\n",
        "start_idx = [start_idx]  # Assuming start_idx is an integer representing the start index\n",
        "end_idx = [end_idx]  # Assuming end_idx is an integer representing the end index\n",
        "\n",
        "generated_sequence, new_start_idx, new_end_idx = generate_scaffold(model, pdb_code, start_idx, end_idx, scaffold_length, data_top_dir, tokenizer, batch_size, device)\n",
        "\n",
        "print(\"motif start indices\", new_start_idx)\n",
        "print(\"motif end indices\", new_end_idx)\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 29,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "jeuZQ7J7O61Z",
        "outputId": "f4d2d397-9d22-44cd-bb83-6316816ae4ce"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Generated sequence saved to: data/interim/evo_generated_sequences/sequence_0.fasta\n",
            "Generated sequence saved to: data/interim/evo_generated_sequences/sequence_1.fasta\n"
          ]
        }
      ],
      "source": [
        "#Save fasta files for these generated sequences:\n",
        "\n",
        "file_paths=[] #A list to keep track of the paths to the fasta files\n",
        "\n",
        "output_dir = 'data/interim/evo_generated_sequences/'\n",
        "os.makedirs(output_dir, exist_ok=True)\n",
        "for i, sequence in enumerate(generated_sequence):\n",
        "    # Create a file path for each sequence file\n",
        "    file_path = os.path.join(output_dir, f'sequence_{i}.fasta')\n",
        "    file_paths.append(file_path) #Add this path to the list\n",
        "    # Open the file in write mode and write the sequence in FASTA format\n",
        "    with open(file_path, 'w') as f:\n",
        "        f.write(f\">SEQUENCE_{i}\\n{sequence}\\n\")\n",
        "\n",
        "    print(f\"Generated sequence saved to: {file_path}\")\n",
        "\n"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "OrmH2wOE9qhG"
      },
      "source": [
        "Now we go back to the file saved by evodiff and we retrieve again the sequence that then will serve as input for omegafold"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 30,
      "metadata": {
        "id": "oyHTzqt0wYHy"
      },
      "outputs": [],
      "source": [
        "# Function to read the sequence from a FASTA file\n",
        "def read_fasta(file_path):\n",
        "    with open(file_path, 'r') as file:\n",
        "        lines = file.readlines()\n",
        "        # Assuming the sequence is in the second line of the FASTA file\n",
        "        sequence = lines[1].strip()\n",
        "    return sequence\n"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "wELNWL9e6-X2"
      },
      "source": [
        "# OmegaFold\n",
        "\n",
        "## Installation"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 31,
      "metadata": {
        "id": "J3YIr_nS1w33"
      },
      "outputs": [],
      "source": [
        "\n",
        "#import os,sys,re\n",
        "import sys,re\n",
        "from IPython.utils import io\n",
        "if \"SETUP_DONE\" not in dir():\n",
        "  import torch\n",
        "  #Try to run on GPU, if it's available\n",
        "  device = \"cuda\" if torch.cuda.is_available() else \"cpu\"\n",
        "  with io.capture_output() as captured:\n",
        "    #If OmegaFold is not already installed locally, install it\n",
        "    if not os.path.isdir(\"OmegaFold\"):\n",
        "      %shell git clone --branch beta --quiet https://github.com/sokrypton/OmegaFold.git\n",
        "      # %shell cd OmegaFold; pip -q install -r requirements.txt\n",
        "      %shell pip -q install py3Dmol biopython==1.81\n",
        "      %shell apt-get install aria2 -qq > /dev/null\n",
        "      %shell aria2c -q -x 16 https://helixon.s3.amazonaws.com/release1.pt\n",
        "      %shell mkdir -p ~/.cache/omegafold_ckpt\n",
        "      %shell mv release1.pt ~/.cache/omegafold_ckpt/model.pt\n",
        "  SETUP_DONE = True"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 32,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "rue45zg-1xoi",
        "outputId": "6ede2fc8-a72e-4912-9811-a85721afd7ac",
        "collapsed": true
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/Bio/Data/SCOPData.py:18: BiopythonDeprecationWarning: The 'Bio.Data.SCOPData' module will be deprecated in a future release of Biopython in favor of 'Bio.Data.PDBData.\n",
            "  warnings.warn(\n",
            "/content/S2DS-Summer24-Polyploy/OmegaFold/omegafold/utils/protein_utils/residue_constants.py:483: UserWarning: Using torch.cross without specifying the dim arg is deprecated.\n",
            "Please either pass the dim explicitly or simply use torch.linalg.cross.\n",
            "The default value of dim will change to agree with that of linalg.cross in a future release. (Triggered internally at ../aten/src/ATen/native/Cross.cpp:62.)\n",
            "  eznorm = torch.cross(ex_normalized, ey_normalized)\n",
            "INFO:root:Loading weights from /root/.cache/omegafold_ckpt/model.pt\n",
            "INFO:root:Constructing OmegaFold\n",
            "INFO:root:Reading evodiff_seq_0_2d04c.fasta\n",
            "INFO:root:Predicting 1th chain in evodiff_seq_0_2d04c.fasta\n",
            "INFO:root:172 residues in this chain.\n",
            "INFO:root:Finished prediction in 179.47 seconds.\n",
            "INFO:root:Saving prediction to ./evodiff_seq_0_2d04c.pdb\n",
            "INFO:root:Saved\n",
            "INFO:root:Done!\n",
            "/usr/local/lib/python3.10/dist-packages/Bio/Data/SCOPData.py:18: BiopythonDeprecationWarning: The 'Bio.Data.SCOPData' module will be deprecated in a future release of Biopython in favor of 'Bio.Data.PDBData.\n",
            "  warnings.warn(\n",
            "/content/S2DS-Summer24-Polyploy/OmegaFold/omegafold/utils/protein_utils/residue_constants.py:483: UserWarning: Using torch.cross without specifying the dim arg is deprecated.\n",
            "Please either pass the dim explicitly or simply use torch.linalg.cross.\n",
            "The default value of dim will change to agree with that of linalg.cross in a future release. (Triggered internally at ../aten/src/ATen/native/Cross.cpp:62.)\n",
            "  eznorm = torch.cross(ex_normalized, ey_normalized)\n",
            "INFO:root:Loading weights from /root/.cache/omegafold_ckpt/model.pt\n",
            "INFO:root:Constructing OmegaFold\n",
            "INFO:root:Reading evodiff_seq_1_824dc.fasta\n",
            "INFO:root:Predicting 1th chain in evodiff_seq_1_824dc.fasta\n",
            "INFO:root:172 residues in this chain.\n",
            "INFO:root:Finished prediction in 180.29 seconds.\n",
            "INFO:root:Saving prediction to ./evodiff_seq_1_824dc.pdb\n",
            "INFO:root:Saved\n",
            "INFO:root:Done!\n"
          ]
        }
      ],
      "source": [
        "import numpy as np\n",
        "\n",
        "#@markdown ##Run **OmegaFold**\n",
        "from string import ascii_uppercase, ascii_lowercase\n",
        "import hashlib\n",
        "import shutil\n",
        "\n",
        "def get_hash(x): return hashlib.sha1(x.encode()).hexdigest()\n",
        "alphabet_list = list(ascii_uppercase+ascii_lowercase)\n",
        "\n",
        "def get_subbatch_size(L):\n",
        "  if L <  500: return 500\n",
        "  if L < 1000: return 200\n",
        "  return 150\n",
        "\n",
        "def renum_pdb_str(pdb_str, Ls=None, renum=True, offset=1):\n",
        "  if Ls is not None:\n",
        "    L_init = 0\n",
        "    new_chain = {}\n",
        "    for L,c in zip(Ls, alphabet_list):\n",
        "      new_chain.update({i:c for i in range(L_init,L_init+L)})\n",
        "      L_init += L\n",
        "\n",
        "  n,num,pdb_out = 0,offset,[]\n",
        "  resnum_ = None\n",
        "  chain_ = None\n",
        "  new_chain_ = new_chain[0]\n",
        "  for line in pdb_str.split(\"\\n\"):\n",
        "    if line[:4] == \"ATOM\":\n",
        "      chain = line[21:22]\n",
        "      resnum = int(line[22:22+5])\n",
        "      if resnum_ is None: resnum_ = resnum\n",
        "      if chain_ is None: chain_ = chain\n",
        "      if resnum != resnum_ or chain != chain_:\n",
        "        num += (resnum - resnum_)\n",
        "        n += 1\n",
        "        resnum_,chain_ = resnum,chain\n",
        "      if Ls is not None:\n",
        "        if new_chain[n] != new_chain_:\n",
        "          num = offset\n",
        "          new_chain_ = new_chain[n]\n",
        "      N = num if renum else resnum\n",
        "      if Ls is None: pdb_out.append(\"%s%4i%s\" % (line[:22],N,line[26:]))\n",
        "      else: pdb_out.append(\"%s%s%4i%s\" % (line[:21],new_chain[n],N,line[26:]))\n",
        "  return \"\\n\".join(pdb_out)\n",
        "\n",
        "pdb_strs = [] #A list for keeping track of the pdb strings, for visualization\n",
        "fasta_fns = [] #A list for keeping track of the output fasta filenames\n",
        "pdb_fns = [] #A list for keeping track of the output pdb filenames\n",
        "\n",
        "#Loop through each of the created fasta files:\n",
        "for i in np.arange(len(generated_sequence)):\n",
        "  jobname = \"evodiff_seq_\" + str(i) #@param {type:\"string\"}\n",
        "  jobname = re.sub(r'\\W+', '', jobname)[:50]\n",
        "\n",
        "  copies = 1 #@param {type:\"integer\"}\n",
        "  #Note that \"sequence\" had been pulled directly from the fasta before, now make copies if needed\n",
        "  sequence = read_fasta(file_paths[i])\n",
        "  sequence = \":\".join([sequence] * copies)\n",
        "\n",
        "\n",
        "\n",
        "  output_path = \"data/processed/\"\n",
        "  os.makedirs(output_path, exist_ok=True)\n",
        "\n",
        "  #@markdown **Advanced Options**\n",
        "  num_cycle = 1 #@param [\"1\", \"2\", \"4\", \"8\", \"16\", \"32\"] {type:\"raw\"}\n",
        "  offset_rope = False #@param {type:\"boolean\"}\n",
        "\n",
        "  ID = jobname+\"_\"+get_hash(sequence)[:5]\n",
        "  seqs = sequence.split(\":\")\n",
        "  lengths = [len(s) for s in seqs]\n",
        "\n",
        "  subbatch_size = get_subbatch_size(sum(lengths))\n",
        "\n",
        "  u_seqs = list(set(seqs))\n",
        "\n",
        "  if len(seqs) == 1: mode = \"mono\"\n",
        "  elif len(u_seqs) == 1: mode = \"homo\"\n",
        "  else: mode = \"hetero\"\n",
        "\n",
        "  with open(f\"{ID}.fasta\",\"w\") as out:\n",
        "    out.write(f\">{ID}\\n{sequence}\\n\")\n",
        "\n",
        "  %shell python OmegaFold/main.py --offset_rope={offset_rope} --device={device} --subbatch_size={subbatch_size} --num_cycle={num_cycle} {ID}.fasta .\n",
        "\n",
        "  pdb_str = renum_pdb_str(open(f\"{ID}.pdb\",'r').read(), Ls=lengths)\n",
        "  with open(f\"{ID}.pdb\",\"w\") as out:\n",
        "    out.write(pdb_str)\n",
        "  pdb_strs.append(pdb_str)\n",
        "\n",
        "  #Now move these files to the desired output directory\n",
        "  fasta_fn = str(ID) + \".fasta\"\n",
        "  pdb_fn = str(ID) + \".pdb\"\n",
        "  shutil.move(f\"{ID}.fasta\",output_path + fasta_fn)\n",
        "  shutil.move(f\"{ID}.pdb\",output_path + pdb_fn)\n",
        "  fasta_fns.append(fasta_fn) #Add the fasta filename to the list\n",
        "  pdb_fns.append(pdb_fn) #Add the pdb filename to the list\n",
        "\n"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "VgajX2R0ZJ0I"
      },
      "source": [
        "## Visualise folded protein using Py3Dmol\n",
        "\n",
        "If you have generated multiple proteins select which one you want to visualise in seq_idx by typing the index of the specific protein stored in pdb_strs.\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 33,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 497
        },
        "id": "Ald5LfwE2OxW",
        "outputId": "ebffd506-fe9d-4c33-ec3b-74d997bf1d75"
      },
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_17225171870958037\"  style=\"position: relative; width: 800px; height: 480px;\">\n        <p id=\"3dmolwarning_17225171870958037\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    //this is to ignore the existence of requirejs amd\n    var savedexports, savedmodule;\n    if (typeof exports !== 'undefined') savedexports = exports;\n    else exports = {}\n    if (typeof module !== 'undefined') savedmodule = module;\n    else module = {}\n\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n        exports = savedexports;\n        module = savedmodule;\n        resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n}\n\nvar viewer_17225171870958037 = null;\nvar warn = document.getElementById(\"3dmolwarning_17225171870958037\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_17225171870958037 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_17225171870958037\"),{backgroundColor:\"white\"});\nviewer_17225171870958037.zoomTo();\n\tviewer_17225171870958037.addModel(\"ATOM      1  N   MET A   1      20.342  -5.221  -3.009  1.00 19.26           N  \\nATOM      2  CA  MET A   1      20.535  -3.913  -2.390  1.00 19.26           C  \\nATOM      3  C   MET A   1      19.208  -3.338  -1.907  1.00 19.26           C  \\nATOM      4  O   MET A   1      18.222  -3.334  -2.647  1.00 19.26           O  \\nATOM      5  CB  MET A   1      21.198  -2.945  -3.373  1.00 19.26           C  \\nATOM      6  CG  MET A   1      22.687  -2.756  -3.137  1.00 19.26           C  \\nATOM      7  SD  MET A   1      23.414  -1.470  -4.227  1.00 19.26           S  \\nATOM      8  CE  MET A   1      25.093  -2.130  -4.425  1.00 19.26           C  \\nATOM      9  N   ASN A   2      18.936  -3.437  -0.635  1.00 24.82           N  \\nATOM     10  CA  ASN A   2      18.253  -2.636   0.376  1.00 24.82           C  \\nATOM     11  C   ASN A   2      17.861  -1.265  -0.167  1.00 24.82           C  \\nATOM     12  O   ASN A   2      18.621  -0.648  -0.915  1.00 24.82           O  \\nATOM     13  CB  ASN A   2      19.128  -2.484   1.622  1.00 24.82           C  \\nATOM     14  CG  ASN A   2      18.782  -3.488   2.704  1.00 24.82           C  \\nATOM     15  OD1 ASN A   2      17.699  -4.080   2.695  1.00 24.82           O  \\nATOM     16  ND2 ASN A   2      19.698  -3.686   3.644  1.00 24.82           N  \\nATOM     17  N   PHE A   3      16.733  -1.136  -0.851  1.00 26.73           N  \\nATOM     18  CA  PHE A   3      16.253   0.239  -0.920  1.00 26.73           C  \\nATOM     19  C   PHE A   3      17.150   1.167  -0.111  1.00 26.73           C  \\nATOM     20  O   PHE A   3      17.429   0.904   1.061  1.00 26.73           O  \\nATOM     21  CB  PHE A   3      14.810   0.332  -0.416  1.00 26.73           C  \\nATOM     22  CG  PHE A   3      13.794   0.515  -1.511  1.00 26.73           C  \\nATOM     23  CD1 PHE A   3      13.466   1.786  -1.967  1.00 26.73           C  \\nATOM     24  CD2 PHE A   3      13.168  -0.584  -2.084  1.00 26.73           C  \\nATOM     25  CE1 PHE A   3      12.526   1.959  -2.981  1.00 26.73           C  \\nATOM     26  CE2 PHE A   3      12.228  -0.419  -3.097  1.00 26.73           C  \\nATOM     27  CZ  PHE A   3      11.908   0.854  -3.544  1.00 26.73           C  \\nATOM     28  N   SER A   4      18.058   1.794  -0.677  1.00 31.72           N  \\nATOM     29  CA  SER A   4      18.738   2.788   0.146  1.00 31.72           C  \\nATOM     30  C   SER A   4      17.741   3.719   0.827  1.00 31.72           C  \\nATOM     31  O   SER A   4      16.580   3.798   0.421  1.00 31.72           O  \\nATOM     32  CB  SER A   4      19.715   3.607  -0.700  1.00 31.72           C  \\nATOM     33  OG  SER A   4      19.059   4.710  -1.300  1.00 31.72           O  \\nATOM     34  N   SER A   5      17.853   3.851   2.111  1.00 33.25           N  \\nATOM     35  CA  SER A   5      17.126   4.857   2.879  1.00 33.25           C  \\nATOM     36  C   SER A   5      16.843   6.097   2.037  1.00 33.25           C  \\nATOM     37  O   SER A   5      15.799   6.734   2.191  1.00 33.25           O  \\nATOM     38  CB  SER A   5      17.913   5.249   4.130  1.00 33.25           C  \\nATOM     39  OG  SER A   5      18.026   4.150   5.018  1.00 33.25           O  \\nATOM     40  N   LYS A   6      17.716   6.326   1.037  1.00 36.79           N  \\nATOM     41  CA  LYS A   6      17.521   7.550   0.266  1.00 36.79           C  \\nATOM     42  C   LYS A   6      16.348   7.411  -0.699  1.00 36.79           C  \\nATOM     43  O   LYS A   6      15.500   8.302  -0.787  1.00 36.79           O  \\nATOM     44  CB  LYS A   6      18.793   7.909  -0.503  1.00 36.79           C  \\nATOM     45  CG  LYS A   6      19.007   9.404  -0.684  1.00 36.79           C  \\nATOM     46  CD  LYS A   6      20.292   9.697  -1.447  1.00 36.79           C  \\nATOM     47  CE  LYS A   6      20.458  11.188  -1.713  1.00 36.79           C  \\nATOM     48  NZ  LYS A   6      21.741  11.485  -2.418  1.00 36.79           N  \\nATOM     49  N   ASP A   7      16.314   6.310  -1.417  1.00 35.84           N  \\nATOM     50  CA  ASP A   7      15.257   6.105  -2.403  1.00 35.84           C  \\nATOM     51  C   ASP A   7      13.889   6.015  -1.730  1.00 35.84           C  \\nATOM     52  O   ASP A   7      12.904   6.553  -2.240  1.00 35.84           O  \\nATOM     53  CB  ASP A   7      15.525   4.841  -3.222  1.00 35.84           C  \\nATOM     54  CG  ASP A   7      16.715   4.979  -4.155  1.00 35.84           C  \\nATOM     55  OD1 ASP A   7      17.032   6.113  -4.575  1.00 35.84           O  \\nATOM     56  OD2 ASP A   7      17.340   3.945  -4.475  1.00 35.84           O  \\nATOM     57  N   ARG A   8      13.909   5.362  -0.564  1.00 35.59           N  \\nATOM     58  CA  ARG A   8      12.718   5.338   0.278  1.00 35.59           C  \\nATOM     59  C   ARG A   8      12.256   6.753   0.613  1.00 35.59           C  \\nATOM     60  O   ARG A   8      11.061   7.051   0.558  1.00 35.59           O  \\nATOM     61  CB  ARG A   8      12.984   4.555   1.566  1.00 35.59           C  \\nATOM     62  CG  ARG A   8      12.770   3.056   1.430  1.00 35.59           C  \\nATOM     63  CD  ARG A   8      13.043   2.327   2.739  1.00 35.59           C  \\nATOM     64  NE  ARG A   8      13.268   0.901   2.524  1.00 35.59           N  \\nATOM     65  CZ  ARG A   8      14.007   0.123   3.310  1.00 35.59           C  \\nATOM     66  NH1 ARG A   8      14.609   0.621   4.384  1.00 35.59           N  \\nATOM     67  NH2 ARG A   8      14.145  -1.163   3.022  1.00 35.59           N  \\nATOM     68  N   GLY A   9      13.346   7.498   0.933  1.00 33.98           N  \\nATOM     69  CA  GLY A   9      13.037   8.875   1.285  1.00 33.98           C  \\nATOM     70  C   GLY A   9      12.453   9.669   0.132  1.00 33.98           C  \\nATOM     71  O   GLY A   9      11.467  10.389   0.303  1.00 33.98           O  \\nATOM     72  N   GLU A  10      13.106   9.485  -1.017  1.00 35.96           N  \\nATOM     73  CA  GLU A  10      12.625  10.251  -2.163  1.00 35.96           C  \\nATOM     74  C   GLU A  10      11.196   9.861  -2.527  1.00 35.96           C  \\nATOM     75  O   GLU A  10      10.372  10.724  -2.839  1.00 35.96           O  \\nATOM     76  CB  GLU A  10      13.548  10.052  -3.368  1.00 35.96           C  \\nATOM     77  CG  GLU A  10      14.817  10.891  -3.317  1.00 35.96           C  \\nATOM     78  CD  GLU A  10      15.622  10.841  -4.605  1.00 35.96           C  \\nATOM     79  OE1 GLU A  10      16.582  11.630  -4.752  1.00 35.96           O  \\nATOM     80  OE2 GLU A  10      15.289  10.005  -5.475  1.00 35.96           O  \\nATOM     81  N   PHE A  11      10.908   8.518  -2.432  1.00 33.60           N  \\nATOM     82  CA  PHE A  11       9.574   8.015  -2.736  1.00 33.60           C  \\nATOM     83  C   PHE A  11       8.558   8.527  -1.722  1.00 33.60           C  \\nATOM     84  O   PHE A  11       7.482   8.999  -2.097  1.00 33.60           O  \\nATOM     85  CB  PHE A  11       9.567   6.484  -2.759  1.00 33.60           C  \\nATOM     86  CG  PHE A  11       8.250   5.888  -3.176  1.00 33.60           C  \\nATOM     87  CD1 PHE A  11       7.377   5.364  -2.230  1.00 33.60           C  \\nATOM     88  CD2 PHE A  11       7.883   5.853  -4.515  1.00 33.60           C  \\nATOM     89  CE1 PHE A  11       6.157   4.812  -2.613  1.00 33.60           C  \\nATOM     90  CE2 PHE A  11       6.666   5.303  -4.906  1.00 33.60           C  \\nATOM     91  CZ  PHE A  11       5.804   4.783  -3.953  1.00 33.60           C  \\nATOM     92  N   LEU A  12       8.915   8.417  -0.481  1.00 35.05           N  \\nATOM     93  CA  LEU A  12       8.045   8.927   0.573  1.00 35.05           C  \\nATOM     94  C   LEU A  12       7.765  10.413   0.377  1.00 35.05           C  \\nATOM     95  O   LEU A  12       6.638  10.869   0.581  1.00 35.05           O  \\nATOM     96  CB  LEU A  12       8.675   8.692   1.949  1.00 35.05           C  \\nATOM     97  CG  LEU A  12       8.477   7.303   2.557  1.00 35.05           C  \\nATOM     98  CD1 LEU A  12       9.563   7.016   3.589  1.00 35.05           C  \\nATOM     99  CD2 LEU A  12       7.092   7.186   3.184  1.00 35.05           C  \\nATOM    100  N   THR A  13       8.824  11.120  -0.076  1.00 37.91           N  \\nATOM    101  CA  THR A  13       8.690  12.553  -0.316  1.00 37.91           C  \\nATOM    102  C   THR A  13       7.724  12.821  -1.466  1.00 37.91           C  \\nATOM    103  O   THR A  13       6.872  13.707  -1.374  1.00 37.91           O  \\nATOM    104  CB  THR A  13      10.054  13.197  -0.627  1.00 37.91           C  \\nATOM    105  OG1 THR A  13      10.983  12.858   0.410  1.00 37.91           O  \\nATOM    106  CG2 THR A  13       9.938  14.715  -0.713  1.00 37.91           C  \\nATOM    107  N   PHE A  14       7.950  12.117  -2.587  1.00 34.13           N  \\nATOM    108  CA  PHE A  14       7.063  12.261  -3.735  1.00 34.13           C  \\nATOM    109  C   PHE A  14       5.612  12.026  -3.331  1.00 34.13           C  \\nATOM    110  O   PHE A  14       4.732  12.818  -3.672  1.00 34.13           O  \\nATOM    111  CB  PHE A  14       7.461  11.287  -4.849  1.00 34.13           C  \\nATOM    112  CG  PHE A  14       6.545  11.322  -6.042  1.00 34.13           C  \\nATOM    113  CD1 PHE A  14       5.556  10.359  -6.205  1.00 34.13           C  \\nATOM    114  CD2 PHE A  14       6.671  12.319  -7.001  1.00 34.13           C  \\nATOM    115  CE1 PHE A  14       4.706  10.389  -7.309  1.00 34.13           C  \\nATOM    116  CE2 PHE A  14       5.826  12.356  -8.106  1.00 34.13           C  \\nATOM    117  CZ  PHE A  14       4.845  11.389  -8.258  1.00 34.13           C  \\nATOM    118  N   ALA A  15       5.321  11.009  -2.556  1.00 35.05           N  \\nATOM    119  CA  ALA A  15       3.966  10.659  -2.137  1.00 35.05           C  \\nATOM    120  C   ALA A  15       3.388  11.723  -1.208  1.00 35.05           C  \\nATOM    121  O   ALA A  15       2.221  12.101  -1.337  1.00 35.05           O  \\nATOM    122  CB  ALA A  15       3.957   9.295  -1.451  1.00 35.05           C  \\nATOM    123  N   ALA A  16       4.214  12.010  -0.186  1.00 36.31           N  \\nATOM    124  CA  ALA A  16       3.776  13.072   0.716  1.00 36.31           C  \\nATOM    125  C   ALA A  16       3.390  14.327  -0.060  1.00 36.31           C  \\nATOM    126  O   ALA A  16       2.389  14.975   0.252  1.00 36.31           O  \\nATOM    127  CB  ALA A  16       4.871  13.393   1.731  1.00 36.31           C  \\nATOM    128  N   LYS A  17       4.177  14.552  -1.125  1.00 37.13           N  \\nATOM    129  CA  LYS A  17       3.895  15.732  -1.936  1.00 37.13           C  \\nATOM    130  C   LYS A  17       2.565  15.589  -2.671  1.00 37.13           C  \\nATOM    131  O   LYS A  17       1.791  16.545  -2.757  1.00 37.13           O  \\nATOM    132  CB  LYS A  17       5.024  15.977  -2.939  1.00 37.13           C  \\nATOM    133  CG  LYS A  17       6.106  16.919  -2.434  1.00 37.13           C  \\nATOM    134  CD  LYS A  17       7.173  17.164  -3.493  1.00 37.13           C  \\nATOM    135  CE  LYS A  17       8.324  17.999  -2.947  1.00 37.13           C  \\nATOM    136  NZ  LYS A  17       9.372  18.243  -3.983  1.00 37.13           N  \\nATOM    137  N   CYS A  18       2.370  14.413  -3.193  1.00 33.27           N  \\nATOM    138  CA  CYS A  18       1.122  14.193  -3.916  1.00 33.27           C  \\nATOM    139  C   CYS A  18      -0.074  14.272  -2.976  1.00 33.27           C  \\nATOM    140  O   CYS A  18      -1.113  14.831  -3.332  1.00 33.27           O  \\nATOM    141  CB  CYS A  18       1.142  12.837  -4.620  1.00 33.27           C  \\nATOM    142  SG  CYS A  18       2.165  12.803  -6.108  1.00 33.27           S  \\nATOM    143  N   GLY A  19       0.168  13.624  -1.777  1.00 33.08           N  \\nATOM    144  CA  GLY A  19      -0.874  13.715  -0.767  1.00 33.08           C  \\nATOM    145  C   GLY A  19      -1.165  15.141  -0.337  1.00 33.08           C  \\nATOM    146  O   GLY A  19      -2.327  15.522  -0.177  1.00 33.08           O  \\nATOM    147  N   LYS A  20      -0.084  15.861  -0.183  1.00 36.99           N  \\nATOM    148  CA  LYS A  20      -0.295  17.256   0.192  1.00 36.99           C  \\nATOM    149  C   LYS A  20      -1.025  18.018  -0.911  1.00 36.99           C  \\nATOM    150  O   LYS A  20      -1.909  18.830  -0.631  1.00 36.99           O  \\nATOM    151  CB  LYS A  20       1.039  17.936   0.504  1.00 36.99           C  \\nATOM    152  CG  LYS A  20       1.473  17.811   1.957  1.00 36.99           C  \\nATOM    153  CD  LYS A  20       2.714  18.647   2.243  1.00 36.99           C  \\nATOM    154  CE  LYS A  20       3.215  18.436   3.666  1.00 36.99           C  \\nATOM    155  NZ  LYS A  20       4.449  19.230   3.942  1.00 36.99           N  \\nATOM    156  N   LYS A  21      -0.487  17.855  -2.114  1.00 36.37           N  \\nATOM    157  CA  LYS A  21      -1.130  18.551  -3.225  1.00 36.37           C  \\nATOM    158  C   LYS A  21      -2.621  18.232  -3.284  1.00 36.37           C  \\nATOM    159  O   LYS A  21      -3.441  19.114  -3.545  1.00 36.37           O  \\nATOM    160  CB  LYS A  21      -0.462  18.182  -4.550  1.00 36.37           C  \\nATOM    161  CG  LYS A  21      -0.677  19.203  -5.657  1.00 36.37           C  \\nATOM    162  CD  LYS A  21       0.053  18.805  -6.934  1.00 36.37           C  \\nATOM    163  CE  LYS A  21      -0.241  19.774  -8.072  1.00 36.37           C  \\nATOM    164  NZ  LYS A  21       0.485  19.397  -9.321  1.00 36.37           N  \\nATOM    165  N   HIS A  22      -2.923  16.976  -2.857  1.00 31.47           N  \\nATOM    166  CA  HIS A  22      -4.329  16.624  -3.016  1.00 31.47           C  \\nATOM    167  C   HIS A  22      -5.024  16.506  -1.664  1.00 31.47           C  \\nATOM    168  O   HIS A  22      -6.209  16.169  -1.597  1.00 31.47           O  \\nATOM    169  CB  HIS A  22      -4.469  15.315  -3.795  1.00 31.47           C  \\nATOM    170  CG  HIS A  22      -4.007  15.408  -5.214  1.00 31.47           C  \\nATOM    171  ND1 HIS A  22      -4.714  16.088  -6.183  1.00 31.47           N  \\nATOM    172  CD2 HIS A  22      -2.908  14.909  -5.826  1.00 31.47           C  \\nATOM    173  CE1 HIS A  22      -4.066  16.002  -7.333  1.00 31.47           C  \\nATOM    174  NE2 HIS A  22      -2.967  15.292  -7.144  1.00 31.47           N  \\nATOM    175  N   LYS A  23      -4.304  16.943  -0.647  1.00 33.11           N  \\nATOM    176  CA  LYS A  23      -4.820  16.880   0.717  1.00 33.11           C  \\nATOM    177  C   LYS A  23      -5.295  15.472   1.061  1.00 33.11           C  \\nATOM    178  O   LYS A  23      -6.340  15.300   1.692  1.00 33.11           O  \\nATOM    179  CB  LYS A  23      -5.963  17.879   0.905  1.00 33.11           C  \\nATOM    180  CG  LYS A  23      -5.518  19.335   0.907  1.00 33.11           C  \\nATOM    181  CD  LYS A  23      -6.662  20.268   1.282  1.00 33.11           C  \\nATOM    182  CE  LYS A  23      -6.234  21.728   1.230  1.00 33.11           C  \\nATOM    183  NZ  LYS A  23      -7.350  22.647   1.605  1.00 33.11           N  \\nATOM    184  N   ARG A  24      -4.766  14.528   0.459  1.00 32.11           N  \\nATOM    185  CA  ARG A  24      -5.162  13.129   0.584  1.00 32.11           C  \\nATOM    186  C   ARG A  24      -4.118  12.332   1.358  1.00 32.11           C  \\nATOM    187  O   ARG A  24      -2.919  12.599   1.249  1.00 32.11           O  \\nATOM    188  CB  ARG A  24      -5.379  12.506  -0.797  1.00 32.11           C  \\nATOM    189  CG  ARG A  24      -6.703  12.885  -1.442  1.00 32.11           C  \\nATOM    190  CD  ARG A  24      -6.875  12.226  -2.804  1.00 32.11           C  \\nATOM    191  NE  ARG A  24      -7.613  13.083  -3.728  1.00 32.11           N  \\nATOM    192  CZ  ARG A  24      -7.873  12.783  -4.997  1.00 32.11           C  \\nATOM    193  NH1 ARG A  24      -7.458  11.634  -5.520  1.00 32.11           N  \\nATOM    194  NH2 ARG A  24      -8.552  13.636  -5.749  1.00 32.11           N  \\nATOM    195  N   LYS A  25      -4.513  11.775   2.477  1.00 28.54           N  \\nATOM    196  CA  LYS A  25      -3.564  10.892   3.148  1.00 28.54           C  \\nATOM    197  C   LYS A  25      -3.399   9.580   2.386  1.00 28.54           C  \\nATOM    198  O   LYS A  25      -4.378   9.015   1.893  1.00 28.54           O  \\nATOM    199  CB  LYS A  25      -4.013  10.611   4.583  1.00 28.54           C  \\nATOM    200  CG  LYS A  25      -3.541  11.647   5.593  1.00 28.54           C  \\nATOM    201  CD  LYS A  25      -3.863  11.222   7.020  1.00 28.54           C  \\nATOM    202  CE  LYS A  25      -3.480  12.301   8.025  1.00 28.54           C  \\nATOM    203  NZ  LYS A  25      -3.807  11.895   9.425  1.00 28.54           N  \\nATOM    204  N   ALA A  26      -2.226   9.407   1.842  1.00 28.49           N  \\nATOM    205  CA  ALA A  26      -1.820   8.259   1.034  1.00 28.49           C  \\nATOM    206  C   ALA A  26      -1.350   7.106   1.916  1.00 28.49           C  \\nATOM    207  O   ALA A  26      -0.676   7.324   2.926  1.00 28.49           O  \\nATOM    208  CB  ALA A  26      -0.719   8.660   0.055  1.00 28.49           C  \\nATOM    209  N   ILE A  27      -1.995   6.070   1.927  1.00 28.34           N  \\nATOM    210  CA  ILE A  27      -1.384   4.929   2.601  1.00 28.34           C  \\nATOM    211  C   ILE A  27      -0.755   3.996   1.569  1.00 28.34           C  \\nATOM    212  O   ILE A  27      -1.353   3.721   0.526  1.00 28.34           O  \\nATOM    213  CB  ILE A  27      -2.414   4.162   3.460  1.00 28.34           C  \\nATOM    214  CG1 ILE A  27      -3.061   5.101   4.484  1.00 28.34           C  \\nATOM    215  CG2 ILE A  27      -1.755   2.966   4.153  1.00 28.34           C  \\nATOM    216  CD1 ILE A  27      -4.155   4.448   5.319  1.00 28.34           C  \\nATOM    217  N   PHE A  28       0.214   3.179   2.000  1.00 26.57           N  \\nATOM    218  CA  PHE A  28       1.266   2.559   1.202  1.00 26.57           C  \\nATOM    219  C   PHE A  28       1.099   1.044   1.170  1.00 26.57           C  \\nATOM    220  O   PHE A  28       0.980   0.405   2.218  1.00 26.57           O  \\nATOM    221  CB  PHE A  28       2.647   2.924   1.755  1.00 26.57           C  \\nATOM    222  CG  PHE A  28       3.103   4.309   1.384  1.00 26.57           C  \\nATOM    223  CD1 PHE A  28       3.851   4.526   0.234  1.00 26.57           C  \\nATOM    224  CD2 PHE A  28       2.782   5.396   2.187  1.00 26.57           C  \\nATOM    225  CE1 PHE A  28       4.274   5.807  -0.112  1.00 26.57           C  \\nATOM    226  CE2 PHE A  28       3.201   6.680   1.849  1.00 26.57           C  \\nATOM    227  CZ  PHE A  28       3.948   6.883   0.699  1.00 26.57           C  \\nATOM    228  N   VAL A  29       1.193   0.417   0.120  1.00 26.65           N  \\nATOM    229  CA  VAL A  29       1.338  -1.030   0.242  1.00 26.65           C  \\nATOM    230  C   VAL A  29       2.568  -1.493  -0.536  1.00 26.65           C  \\nATOM    231  O   VAL A  29       2.718  -1.176  -1.718  1.00 26.65           O  \\nATOM    232  CB  VAL A  29       0.079  -1.771  -0.260  1.00 26.65           C  \\nATOM    233  CG1 VAL A  29       0.140  -3.251   0.112  1.00 26.65           C  \\nATOM    234  CG2 VAL A  29      -1.182  -1.125   0.310  1.00 26.65           C  \\nATOM    235  N   THR A  30       3.658  -1.545   0.037  1.00 27.23           N  \\nATOM    236  CA  THR A  30       4.823  -2.065  -0.671  1.00 27.23           C  \\nATOM    237  C   THR A  30       4.768  -3.587  -0.758  1.00 27.23           C  \\nATOM    238  O   THR A  30       4.133  -4.239   0.073  1.00 27.23           O  \\nATOM    239  CB  THR A  30       6.132  -1.634   0.017  1.00 27.23           C  \\nATOM    240  OG1 THR A  30       6.681  -2.751   0.729  1.00 27.23           O  \\nATOM    241  CG2 THR A  30       5.888  -0.492   0.997  1.00 27.23           C  \\nATOM    242  N   GLY A  31       4.924  -4.134  -1.893  1.00 26.19           N  \\nATOM    243  CA  GLY A  31       5.068  -5.560  -2.142  1.00 26.19           C  \\nATOM    244  C   GLY A  31       6.496  -6.050  -1.998  1.00 26.19           C  \\nATOM    245  O   GLY A  31       7.414  -5.489  -2.600  1.00 26.19           O  \\nATOM    246  N   SER A  32       7.047  -6.276  -0.823  1.00 24.92           N  \\nATOM    247  CA  SER A  32       8.323  -6.969  -0.970  1.00 24.92           C  \\nATOM    248  C   SER A  32       8.182  -8.458  -0.670  1.00 24.92           C  \\nATOM    249  O   SER A  32       7.392  -8.851   0.191  1.00 24.92           O  \\nATOM    250  CB  SER A  32       9.377  -6.353  -0.049  1.00 24.92           C  \\nATOM    251  OG  SER A  32       9.065  -6.604   1.311  1.00 24.92           O  \\nATOM    252  N   ARG A  33       8.693  -9.399  -1.484  1.00 24.59           N  \\nATOM    253  CA  ARG A  33       9.346 -10.608  -0.992  1.00 24.59           C  \\nATOM    254  C   ARG A  33      10.065 -11.340  -2.121  1.00 24.59           C  \\nATOM    255  O   ARG A  33       9.440 -11.738  -3.106  1.00 24.59           O  \\nATOM    256  CB  ARG A  33       8.328 -11.538  -0.329  1.00 24.59           C  \\nATOM    257  CG  ARG A  33       8.186 -11.327   1.170  1.00 24.59           C  \\nATOM    258  CD  ARG A  33       7.311 -12.396   1.810  1.00 24.59           C  \\nATOM    259  NE  ARG A  33       7.206 -12.213   3.255  1.00 24.59           N  \\nATOM    260  CZ  ARG A  33       6.553 -13.030   4.076  1.00 24.59           C  \\nATOM    261  NH1 ARG A  33       5.930 -14.106   3.610  1.00 24.59           N  \\nATOM    262  NH2 ARG A  33       6.521 -12.769   5.375  1.00 24.59           N  \\nATOM    263  N   GLY A  34      11.362 -11.122  -2.264  1.00 22.55           N  \\nATOM    264  CA  GLY A  34      12.088 -12.185  -2.940  1.00 22.55           C  \\nATOM    265  C   GLY A  34      13.337 -12.620  -2.198  1.00 22.55           C  \\nATOM    266  O   GLY A  34      13.778 -11.945  -1.266  1.00 22.55           O  \\nATOM    267  N   THR A  35      13.476 -13.813  -1.827  1.00 22.69           N  \\nATOM    268  CA  THR A  35      14.695 -14.564  -1.547  1.00 22.69           C  \\nATOM    269  C   THR A  35      15.899 -13.913  -2.222  1.00 22.69           C  \\nATOM    270  O   THR A  35      15.775 -13.346  -3.310  1.00 22.69           O  \\nATOM    271  CB  THR A  35      14.570 -16.026  -2.013  1.00 22.69           C  \\nATOM    272  OG1 THR A  35      14.055 -16.053  -3.350  1.00 22.69           O  \\nATOM    273  CG2 THR A  35      13.634 -16.815  -1.104  1.00 22.69           C  \\nATOM    274  N   ARG A  36      16.632 -13.027  -1.470  1.00 23.57           N  \\nATOM    275  CA  ARG A  36      17.886 -12.281  -1.524  1.00 23.57           C  \\nATOM    276  C   ARG A  36      18.638 -12.568  -2.819  1.00 23.57           C  \\nATOM    277  O   ARG A  36      19.173 -11.652  -3.448  1.00 23.57           O  \\nATOM    278  CB  ARG A  36      18.766 -12.621  -0.320  1.00 23.57           C  \\nATOM    279  CG  ARG A  36      18.402 -11.860   0.945  1.00 23.57           C  \\nATOM    280  CD  ARG A  36      19.277 -12.269   2.122  1.00 23.57           C  \\nATOM    281  NE  ARG A  36      19.369 -11.208   3.120  1.00 23.57           N  \\nATOM    282  CZ  ARG A  36      20.028 -11.305   4.271  1.00 23.57           C  \\nATOM    283  NH1 ARG A  36      20.670 -12.422   4.593  1.00 23.57           N  \\nATOM    284  NH2 ARG A  36      20.047 -10.277   5.107  1.00 23.57           N  \\nATOM    285  N   SER A  37      18.651 -13.793  -3.376  1.00 23.37           N  \\nATOM    286  CA  SER A  37      19.697 -14.026  -4.366  1.00 23.37           C  \\nATOM    287  C   SER A  37      19.174 -13.818  -5.783  1.00 23.37           C  \\nATOM    288  O   SER A  37      19.904 -13.348  -6.658  1.00 23.37           O  \\nATOM    289  CB  SER A  37      20.263 -15.440  -4.226  1.00 23.37           C  \\nATOM    290  OG  SER A  37      19.434 -16.381  -4.885  1.00 23.37           O  \\nATOM    291  N   ASP A  38      17.924 -14.032  -5.972  1.00 26.30           N  \\nATOM    292  CA  ASP A  38      17.442 -13.880  -7.341  1.00 26.30           C  \\nATOM    293  C   ASP A  38      17.088 -12.425  -7.641  1.00 26.30           C  \\nATOM    294  O   ASP A  38      16.953 -12.041  -8.804  1.00 26.30           O  \\nATOM    295  CB  ASP A  38      16.227 -14.778  -7.585  1.00 26.30           C  \\nATOM    296  CG  ASP A  38      16.592 -16.247  -7.714  1.00 26.30           C  \\nATOM    297  OD1 ASP A  38      17.761 -16.560  -8.028  1.00 26.30           O  \\nATOM    298  OD2 ASP A  38      15.702 -17.098  -7.503  1.00 26.30           O  \\nATOM    299  N   GLU A  39      17.116 -11.671  -6.538  1.00 25.48           N  \\nATOM    300  CA  GLU A  39      16.894 -10.230  -6.609  1.00 25.48           C  \\nATOM    301  C   GLU A  39      17.946  -9.553  -7.482  1.00 25.48           C  \\nATOM    302  O   GLU A  39      17.635  -8.622  -8.229  1.00 25.48           O  \\nATOM    303  CB  GLU A  39      16.899  -9.616  -5.207  1.00 25.48           C  \\nATOM    304  CG  GLU A  39      15.528  -9.576  -4.548  1.00 25.48           C  \\nATOM    305  CD  GLU A  39      15.552  -8.984  -3.148  1.00 25.48           C  \\nATOM    306  OE1 GLU A  39      14.472  -8.839  -2.532  1.00 25.48           O  \\nATOM    307  OE2 GLU A  39      16.660  -8.662  -2.664  1.00 25.48           O  \\nATOM    308  N   MET A  40      19.098 -10.098  -7.437  1.00 22.65           N  \\nATOM    309  CA  MET A  40      20.196  -9.370  -8.067  1.00 22.65           C  \\nATOM    310  C   MET A  40      20.121  -9.480  -9.586  1.00 22.65           C  \\nATOM    311  O   MET A  40      20.442  -8.527 -10.298  1.00 22.65           O  \\nATOM    312  CB  MET A  40      21.544  -9.894  -7.569  1.00 22.65           C  \\nATOM    313  CG  MET A  40      22.393  -8.840  -6.877  1.00 22.65           C  \\nATOM    314  SD  MET A  40      23.746  -9.573  -5.877  1.00 22.65           S  \\nATOM    315  CE  MET A  40      25.033  -9.732  -7.147  1.00 22.65           C  \\nATOM    316  N   ALA A  41      19.631 -10.621  -9.985  1.00 25.44           N  \\nATOM    317  CA  ALA A  41      19.750 -10.800 -11.430  1.00 25.44           C  \\nATOM    318  C   ALA A  41      18.748  -9.922 -12.175  1.00 25.44           C  \\nATOM    319  O   ALA A  41      19.074  -9.344 -13.215  1.00 25.44           O  \\nATOM    320  CB  ALA A  41      19.548 -12.266 -11.802  1.00 25.44           C  \\nATOM    321  N   GLY A  42      17.666  -9.760 -11.538  1.00 26.60           N  \\nATOM    322  CA  GLY A  42      16.696  -9.030 -12.339  1.00 26.60           C  \\nATOM    323  C   GLY A  42      16.845  -7.524 -12.230  1.00 26.60           C  \\nATOM    324  O   GLY A  42      16.531  -6.794 -13.172  1.00 26.60           O  \\nATOM    325  N   TYR A  43      17.698  -7.174 -11.288  1.00 24.07           N  \\nATOM    326  CA  TYR A  43      17.898  -5.762 -10.983  1.00 24.07           C  \\nATOM    327  C   TYR A  43      18.847  -5.116 -11.984  1.00 24.07           C  \\nATOM    328  O   TYR A  43      18.656  -3.963 -12.376  1.00 24.07           O  \\nATOM    329  CB  TYR A  43      18.442  -5.591  -9.561  1.00 24.07           C  \\nATOM    330  CG  TYR A  43      17.852  -4.414  -8.823  1.00 24.07           C  \\nATOM    331  CD1 TYR A  43      18.441  -3.154  -8.897  1.00 24.07           C  \\nATOM    332  CD2 TYR A  43      16.706  -4.561  -8.049  1.00 24.07           C  \\nATOM    333  CE1 TYR A  43      17.901  -2.067  -8.216  1.00 24.07           C  \\nATOM    334  CE2 TYR A  43      16.158  -3.481  -7.364  1.00 24.07           C  \\nATOM    335  CZ  TYR A  43      16.761  -2.240  -7.454  1.00 24.07           C  \\nATOM    336  OH  TYR A  43      16.223  -1.168  -6.778  1.00 24.07           O  \\nATOM    337  N   GLU A  44      20.053  -5.666 -12.051  1.00 26.05           N  \\nATOM    338  CA  GLU A  44      20.923  -4.988 -13.008  1.00 26.05           C  \\nATOM    339  C   GLU A  44      20.165  -4.627 -14.282  1.00 26.05           C  \\nATOM    340  O   GLU A  44      20.485  -3.636 -14.941  1.00 26.05           O  \\nATOM    341  CB  GLU A  44      22.135  -5.860 -13.345  1.00 26.05           C  \\nATOM    342  CG  GLU A  44      23.270  -5.753 -12.337  1.00 26.05           C  \\nATOM    343  CD  GLU A  44      24.624  -6.131 -12.916  1.00 26.05           C  \\nATOM    344  OE1 GLU A  44      25.653  -5.930 -12.232  1.00 26.05           O  \\nATOM    345  OE2 GLU A  44      24.656  -6.631 -14.062  1.00 26.05           O  \\nATOM    346  N   GLN A  45      19.059  -5.270 -14.483  1.00 26.60           N  \\nATOM    347  CA  GLN A  45      18.412  -4.773 -15.693  1.00 26.60           C  \\nATOM    348  C   GLN A  45      17.568  -3.537 -15.397  1.00 26.60           C  \\nATOM    349  O   GLN A  45      17.360  -2.696 -16.274  1.00 26.60           O  \\nATOM    350  CB  GLN A  45      17.544  -5.862 -16.325  1.00 26.60           C  \\nATOM    351  CG  GLN A  45      18.321  -6.839 -17.197  1.00 26.60           C  \\nATOM    352  CD  GLN A  45      17.446  -7.940 -17.767  1.00 26.60           C  \\nATOM    353  OE1 GLN A  45      16.285  -8.094 -17.376  1.00 26.60           O  \\nATOM    354  NE2 GLN A  45      17.998  -8.714 -18.696  1.00 26.60           N  \\nATOM    355  N   PHE A  46      17.314  -3.170 -14.142  1.00 22.47           N  \\nATOM    356  CA  PHE A  46      16.252  -2.178 -14.265  1.00 22.47           C  \\nATOM    357  C   PHE A  46      16.679  -0.849 -13.652  1.00 22.47           C  \\nATOM    358  O   PHE A  46      15.841  -0.083 -13.172  1.00 22.47           O  \\nATOM    359  CB  PHE A  46      14.968  -2.676 -13.595  1.00 22.47           C  \\nATOM    360  CG  PHE A  46      14.238  -3.726 -14.388  1.00 22.47           C  \\nATOM    361  CD1 PHE A  46      13.443  -3.372 -15.471  1.00 22.47           C  \\nATOM    362  CD2 PHE A  46      14.347  -5.069 -14.051  1.00 22.47           C  \\nATOM    363  CE1 PHE A  46      12.766  -4.342 -16.207  1.00 22.47           C  \\nATOM    364  CE2 PHE A  46      13.674  -6.044 -14.781  1.00 22.47           C  \\nATOM    365  CZ  PHE A  46      12.883  -5.678 -15.859  1.00 22.47           C  \\nATOM    366  N   ARG A  47      18.041  -0.763 -13.387  1.00 26.21           N  \\nATOM    367  CA  ARG A  47      18.601   0.573 -13.210  1.00 26.21           C  \\nATOM    368  C   ARG A  47      17.892   1.587 -14.101  1.00 26.21           C  \\nATOM    369  O   ARG A  47      17.663   1.328 -15.285  1.00 26.21           O  \\nATOM    370  CB  ARG A  47      20.102   0.570 -13.510  1.00 26.21           C  \\nATOM    371  CG  ARG A  47      20.978   0.544 -12.268  1.00 26.21           C  \\nATOM    372  CD  ARG A  47      22.447   0.753 -12.610  1.00 26.21           C  \\nATOM    373  NE  ARG A  47      23.273   0.834 -11.409  1.00 26.21           N  \\nATOM    374  CZ  ARG A  47      24.589   1.026 -11.406  1.00 26.21           C  \\nATOM    375  NH1 ARG A  47      25.257   1.163 -12.546  1.00 26.21           N  \\nATOM    376  NH2 ARG A  47      25.243   1.083 -10.255  1.00 26.21           N  \\nATOM    377  N   GLY A  48      17.188   2.564 -13.588  1.00 26.84           N  \\nATOM    378  CA  GLY A  48      16.745   3.762 -14.283  1.00 26.84           C  \\nATOM    379  C   GLY A  48      15.394   3.596 -14.955  1.00 26.84           C  \\nATOM    380  O   GLY A  48      14.969   4.459 -15.726  1.00 26.84           O  \\nATOM    381  N   LYS A  49      14.758   2.475 -14.847  1.00 24.35           N  \\nATOM    382  CA  LYS A  49      13.395   2.418 -15.368  1.00 24.35           C  \\nATOM    383  C   LYS A  49      12.379   2.785 -14.291  1.00 24.35           C  \\nATOM    384  O   LYS A  49      12.368   2.190 -13.211  1.00 24.35           O  \\nATOM    385  CB  LYS A  49      13.091   1.026 -15.923  1.00 24.35           C  \\nATOM    386  CG  LYS A  49      13.787   0.719 -17.241  1.00 24.35           C  \\nATOM    387  CD  LYS A  49      13.298  -0.594 -17.838  1.00 24.35           C  \\nATOM    388  CE  LYS A  49      14.020  -0.921 -19.139  1.00 24.35           C  \\nATOM    389  NZ  LYS A  49      13.574  -2.228 -19.707  1.00 24.35           N  \\nATOM    390  N   PRO A  50      12.038   4.055 -14.194  1.00 23.93           N  \\nATOM    391  CA  PRO A  50      11.130   4.564 -13.164  1.00 23.93           C  \\nATOM    392  C   PRO A  50       9.960   3.620 -12.890  1.00 23.93           C  \\nATOM    393  O   PRO A  50       9.214   3.819 -11.928  1.00 23.93           O  \\nATOM    394  CB  PRO A  50      10.637   5.888 -13.752  1.00 23.93           C  \\nATOM    395  CG  PRO A  50      11.203   5.927 -15.134  1.00 23.93           C  \\nATOM    396  CD  PRO A  50      12.106   4.740 -15.314  1.00 23.93           C  \\nATOM    397  N   ASN A  51       9.856   2.506 -13.542  1.00 23.66           N  \\nATOM    398  CA  ASN A  51       8.621   1.745 -13.388  1.00 23.66           C  \\nATOM    399  C   ASN A  51       8.659   0.860 -12.144  1.00 23.66           C  \\nATOM    400  O   ASN A  51       7.688   0.164 -11.843  1.00 23.66           O  \\nATOM    401  CB  ASN A  51       8.352   0.898 -14.633  1.00 23.66           C  \\nATOM    402  CG  ASN A  51       7.750   1.703 -15.768  1.00 23.66           C  \\nATOM    403  OD1 ASN A  51       7.207   2.790 -15.554  1.00 23.66           O  \\nATOM    404  ND2 ASN A  51       7.842   1.176 -16.983  1.00 23.66           N  \\nATOM    405  N   ALA A  52       9.828   0.659 -11.617  1.00 23.02           N  \\nATOM    406  CA  ALA A  52       9.854  -0.401 -10.613  1.00 23.02           C  \\nATOM    407  C   ALA A  52       9.137   0.033  -9.339  1.00 23.02           C  \\nATOM    408  O   ALA A  52       8.710  -0.806  -8.542  1.00 23.02           O  \\nATOM    409  CB  ALA A  52      11.294  -0.802 -10.300  1.00 23.02           C  \\nATOM    410  N   ILE A  53       8.752   1.254  -9.265  1.00 22.86           N  \\nATOM    411  CA  ILE A  53       8.136   1.761  -8.044  1.00 22.86           C  \\nATOM    412  C   ILE A  53       6.658   1.378  -8.014  1.00 22.86           C  \\nATOM    413  O   ILE A  53       5.992   1.525  -6.986  1.00 22.86           O  \\nATOM    414  CB  ILE A  53       8.295   3.293  -7.925  1.00 22.86           C  \\nATOM    415  CG1 ILE A  53       9.779   3.677  -7.923  1.00 22.86           C  \\nATOM    416  CG2 ILE A  53       7.593   3.813  -6.666  1.00 22.86           C  \\nATOM    417  CD1 ILE A  53      10.031   5.176  -8.021  1.00 22.86           C  \\nATOM    418  N   ILE A  54       6.226   0.544  -8.942  1.00 24.11           N  \\nATOM    419  CA  ILE A  54       4.776   0.549  -9.101  1.00 24.11           C  \\nATOM    420  C   ILE A  54       4.160  -0.561  -8.253  1.00 24.11           C  \\nATOM    421  O   ILE A  54       2.936  -0.694  -8.188  1.00 24.11           O  \\nATOM    422  CB  ILE A  54       4.370   0.380 -10.583  1.00 24.11           C  \\nATOM    423  CG1 ILE A  54       4.980   1.498 -11.435  1.00 24.11           C  \\nATOM    424  CG2 ILE A  54       2.845   0.353 -10.725  1.00 24.11           C  \\nATOM    425  CD1 ILE A  54       4.699   1.364 -12.925  1.00 24.11           C  \\nATOM    426  N   GLY A  55       4.935  -1.285  -7.507  1.00 25.57           N  \\nATOM    427  CA  GLY A  55       4.109  -2.193  -6.729  1.00 25.57           C  \\nATOM    428  C   GLY A  55       3.530  -1.553  -5.481  1.00 25.57           C  \\nATOM    429  O   GLY A  55       2.831  -2.210  -4.707  1.00 25.57           O  \\nATOM    430  N   ALA A  56       3.666  -0.269  -5.338  1.00 25.26           N  \\nATOM    431  CA  ALA A  56       3.179   0.402  -4.136  1.00 25.26           C  \\nATOM    432  C   ALA A  56       1.816   1.044  -4.379  1.00 25.26           C  \\nATOM    433  O   ALA A  56       1.599   1.681  -5.413  1.00 25.26           O  \\nATOM    434  CB  ALA A  56       4.183   1.454  -3.670  1.00 25.26           C  \\nATOM    435  N   GLY A  57       0.778   0.228  -4.369  1.00 27.90           N  \\nATOM    436  CA  GLY A  57      -0.533   0.851  -4.456  1.00 27.90           C  \\nATOM    437  C   GLY A  57      -0.731   1.969  -3.451  1.00 27.90           C  \\nATOM    438  O   GLY A  57      -0.176   1.928  -2.351  1.00 27.90           O  \\nATOM    439  N   ILE A  58      -0.900   3.122  -3.945  1.00 27.63           N  \\nATOM    440  CA  ILE A  58      -1.239   4.328  -3.198  1.00 27.63           C  \\nATOM    441  C   ILE A  58      -2.740   4.356  -2.917  1.00 27.63           C  \\nATOM    442  O   ILE A  58      -3.552   4.221  -3.835  1.00 27.63           O  \\nATOM    443  CB  ILE A  58      -0.812   5.603  -3.959  1.00 27.63           C  \\nATOM    444  CG1 ILE A  58       0.687   5.558  -4.276  1.00 27.63           C  \\nATOM    445  CG2 ILE A  58      -1.164   6.857  -3.152  1.00 27.63           C  \\nATOM    446  CD1 ILE A  58       1.163   6.690  -5.177  1.00 27.63           C  \\nATOM    447  N   ILE A  59      -3.117   4.183  -1.694  1.00 30.18           N  \\nATOM    448  CA  ILE A  59      -4.528   4.395  -1.389  1.00 30.18           C  \\nATOM    449  C   ILE A  59      -4.734   5.815  -0.866  1.00 30.18           C  \\nATOM    450  O   ILE A  59      -4.039   6.251   0.055  1.00 30.18           O  \\nATOM    451  CB  ILE A  59      -5.045   3.364  -0.362  1.00 30.18           C  \\nATOM    452  CG1 ILE A  59      -4.815   1.938  -0.874  1.00 30.18           C  \\nATOM    453  CG2 ILE A  59      -6.527   3.604  -0.056  1.00 30.18           C  \\nATOM    454  CD1 ILE A  59      -5.166   0.854   0.136  1.00 30.18           C  \\nATOM    455  N   PHE A  60      -5.431   6.566  -1.626  1.00 27.15           N  \\nATOM    456  CA  PHE A  60      -5.690   7.944  -1.226  1.00 27.15           C  \\nATOM    457  C   PHE A  60      -6.935   8.030  -0.352  1.00 27.15           C  \\nATOM    458  O   PHE A  60      -7.978   7.466  -0.690  1.00 27.15           O  \\nATOM    459  CB  PHE A  60      -5.850   8.842  -2.457  1.00 27.15           C  \\nATOM    460  CG  PHE A  60      -4.610   8.935  -3.306  1.00 27.15           C  \\nATOM    461  CD1 PHE A  60      -3.615   9.858  -3.008  1.00 27.15           C  \\nATOM    462  CD2 PHE A  60      -4.440   8.099  -4.401  1.00 27.15           C  \\nATOM    463  CE1 PHE A  60      -2.467   9.947  -3.792  1.00 27.15           C  \\nATOM    464  CE2 PHE A  60      -3.296   8.181  -5.189  1.00 27.15           C  \\nATOM    465  CZ  PHE A  60      -2.311   9.107  -4.883  1.00 27.15           C  \\nATOM    466  N   LEU A  61      -6.792   8.558   0.800  1.00 28.66           N  \\nATOM    467  CA  LEU A  61      -7.975   8.703   1.642  1.00 28.66           C  \\nATOM    468  C   LEU A  61      -8.442  10.154   1.679  1.00 28.66           C  \\nATOM    469  O   LEU A  61      -7.623  11.075   1.639  1.00 28.66           O  \\nATOM    470  CB  LEU A  61      -7.685   8.211   3.063  1.00 28.66           C  \\nATOM    471  CG  LEU A  61      -7.315   6.734   3.207  1.00 28.66           C  \\nATOM    472  CD1 LEU A  61      -6.608   6.492   4.536  1.00 28.66           C  \\nATOM    473  CD2 LEU A  61      -8.557   5.858   3.089  1.00 28.66           C  \\nATOM    474  N   ASP A  62      -9.616  10.389   1.187  1.00 28.79           N  \\nATOM    475  CA  ASP A  62     -10.344  11.652   1.256  1.00 28.79           C  \\nATOM    476  C   ASP A  62     -10.496  12.121   2.701  1.00 28.79           C  \\nATOM    477  O   ASP A  62     -10.486  11.307   3.628  1.00 28.79           O  \\nATOM    478  CB  ASP A  62     -11.720  11.515   0.601  1.00 28.79           C  \\nATOM    479  CG  ASP A  62     -12.425  12.847   0.417  1.00 28.79           C  \\nATOM    480  OD1 ASP A  62     -11.890  13.886   0.861  1.00 28.79           O  \\nATOM    481  OD2 ASP A  62     -13.526  12.858  -0.175  1.00 28.79           O  \\nATOM    482  N   GLU A  63      -9.959  13.193   3.025  1.00 28.13           N  \\nATOM    483  CA  GLU A  63     -10.331  13.866   4.265  1.00 28.13           C  \\nATOM    484  C   GLU A  63     -11.779  13.564   4.644  1.00 28.13           C  \\nATOM    485  O   GLU A  63     -12.118  13.513   5.828  1.00 28.13           O  \\nATOM    486  CB  GLU A  63     -10.125  15.378   4.140  1.00 28.13           C  \\nATOM    487  CG  GLU A  63      -8.961  15.910   4.964  1.00 28.13           C  \\nATOM    488  CD  GLU A  63      -8.883  17.428   4.981  1.00 28.13           C  \\nATOM    489  OE1 GLU A  63      -8.025  17.984   5.703  1.00 28.13           O  \\nATOM    490  OE2 GLU A  63      -9.688  18.067   4.267  1.00 28.13           O  \\nATOM    491  N   ALA A  64     -12.668  13.684   3.656  1.00 26.80           N  \\nATOM    492  CA  ALA A  64     -14.004  13.353   4.145  1.00 26.80           C  \\nATOM    493  C   ALA A  64     -13.989  12.055   4.946  1.00 26.80           C  \\nATOM    494  O   ALA A  64     -14.824  11.855   5.832  1.00 26.80           O  \\nATOM    495  CB  ALA A  64     -14.986  13.245   2.980  1.00 26.80           C  \\nATOM    496  N   SER A  65     -12.929  11.339   4.776  1.00 27.10           N  \\nATOM    497  CA  SER A  65     -12.904  10.114   5.568  1.00 27.10           C  \\nATOM    498  C   SER A  65     -12.344  10.370   6.964  1.00 27.10           C  \\nATOM    499  O   SER A  65     -12.158   9.434   7.745  1.00 27.10           O  \\nATOM    500  CB  SER A  65     -12.075   9.038   4.867  1.00 27.10           C  \\nATOM    501  OG  SER A  65     -10.863   9.580   4.370  1.00 27.10           O  \\nATOM    502  N   LYS A  66     -11.629  11.465   7.109  1.00 27.42           N  \\nATOM    503  CA  LYS A  66     -11.222  11.763   8.479  1.00 27.42           C  \\nATOM    504  C   LYS A  66     -12.287  11.319   9.477  1.00 27.42           C  \\nATOM    505  O   LYS A  66     -11.964  10.866  10.577  1.00 27.42           O  \\nATOM    506  CB  LYS A  66     -10.941  13.257   8.643  1.00 27.42           C  \\nATOM    507  CG  LYS A  66      -9.531  13.669   8.245  1.00 27.42           C  \\nATOM    508  CD  LYS A  66      -9.274  15.141   8.541  1.00 27.42           C  \\nATOM    509  CE  LYS A  66      -7.876  15.563   8.110  1.00 27.42           C  \\nATOM    510  NZ  LYS A  66      -7.628  17.012   8.375  1.00 27.42           N  \\nATOM    511  N   HIS A  67     -13.413  11.807   9.159  1.00 24.43           N  \\nATOM    512  CA  HIS A  67     -14.326  11.473  10.246  1.00 24.43           C  \\nATOM    513  C   HIS A  67     -14.614   9.976  10.283  1.00 24.43           C  \\nATOM    514  O   HIS A  67     -15.337   9.500  11.162  1.00 24.43           O  \\nATOM    515  CB  HIS A  67     -15.633  12.256  10.107  1.00 24.43           C  \\nATOM    516  CG  HIS A  67     -15.436  13.731   9.954  1.00 24.43           C  \\nATOM    517  ND1 HIS A  67     -14.918  14.520  10.958  1.00 24.43           N  \\nATOM    518  CD2 HIS A  67     -15.687  14.558   8.912  1.00 24.43           C  \\nATOM    519  CE1 HIS A  67     -14.858  15.773  10.538  1.00 24.43           C  \\nATOM    520  NE2 HIS A  67     -15.319  15.822   9.300  1.00 24.43           N  \\nATOM    521  N   MET A  68     -13.890   9.296   9.509  1.00 22.35           N  \\nATOM    522  CA  MET A  68     -14.343   7.911   9.600  1.00 22.35           C  \\nATOM    523  C   MET A  68     -13.209   6.996  10.049  1.00 22.35           C  \\nATOM    524  O   MET A  68     -13.047   5.895   9.520  1.00 22.35           O  \\nATOM    525  CB  MET A  68     -14.897   7.436   8.255  1.00 22.35           C  \\nATOM    526  CG  MET A  68     -16.278   7.984   7.934  1.00 22.35           C  \\nATOM    527  SD  MET A  68     -16.839   7.528   6.247  1.00 22.35           S  \\nATOM    528  CE  MET A  68     -18.202   6.402   6.657  1.00 22.35           C  \\nATOM    529  N   TRP A  69     -12.254   7.629  10.737  1.00 23.82           N  \\nATOM    530  CA  TRP A  69     -11.306   6.713  11.363  1.00 23.82           C  \\nATOM    531  C   TRP A  69     -12.016   5.777  12.335  1.00 23.82           C  \\nATOM    532  O   TRP A  69     -12.974   6.174  13.002  1.00 23.82           O  \\nATOM    533  CB  TRP A  69     -10.208   7.491  12.092  1.00 23.82           C  \\nATOM    534  CG  TRP A  69      -8.818   7.142  11.650  1.00 23.82           C  \\nATOM    535  CD1 TRP A  69      -8.201   5.927  11.756  1.00 23.82           C  \\nATOM    536  CD2 TRP A  69      -7.871   8.021  11.036  1.00 23.82           C  \\nATOM    537  NE1 TRP A  69      -6.926   5.997  11.243  1.00 23.82           N  \\nATOM    538  CE2 TRP A  69      -6.699   7.271  10.795  1.00 23.82           C  \\nATOM    539  CE3 TRP A  69      -7.901   9.372  10.664  1.00 23.82           C  \\nATOM    540  CZ2 TRP A  69      -5.565   7.828  10.199  1.00 23.82           C  \\nATOM    541  CZ3 TRP A  69      -6.771   9.924  10.071  1.00 23.82           C  \\nATOM    542  CH2 TRP A  69      -5.621   9.151   9.845  1.00 23.82           C  \\nATOM    543  N   PRO A  70     -11.435   4.711  12.760  1.00 24.68           N  \\nATOM    544  CA  PRO A  70     -11.103   3.418  12.157  1.00 24.68           C  \\nATOM    545  C   PRO A  70     -12.289   2.457  12.130  1.00 24.68           C  \\nATOM    546  O   PRO A  70     -12.106   1.249  11.953  1.00 24.68           O  \\nATOM    547  CB  PRO A  70      -9.988   2.884  13.059  1.00 24.68           C  \\nATOM    548  CG  PRO A  70     -10.227   3.531  14.385  1.00 24.68           C  \\nATOM    549  CD  PRO A  70     -11.484   4.348  14.297  1.00 24.68           C  \\nATOM    550  N   GLY A  71     -13.464   2.935  12.256  1.00 26.84           N  \\nATOM    551  CA  GLY A  71     -14.462   1.923  11.947  1.00 26.84           C  \\nATOM    552  C   GLY A  71     -14.660   1.715  10.458  1.00 26.84           C  \\nATOM    553  O   GLY A  71     -15.062   0.633  10.025  1.00 26.84           O  \\nATOM    554  N   ASP A  72     -14.191   2.627   9.693  1.00 30.30           N  \\nATOM    555  CA  ASP A  72     -14.499   2.522   8.270  1.00 30.30           C  \\nATOM    556  C   ASP A  72     -13.496   1.618   7.556  1.00 30.30           C  \\nATOM    557  O   ASP A  72     -13.811   1.032   6.518  1.00 30.30           O  \\nATOM    558  CB  ASP A  72     -14.513   3.908   7.620  1.00 30.30           C  \\nATOM    559  CG  ASP A  72     -15.903   4.351   7.199  1.00 30.30           C  \\nATOM    560  OD1 ASP A  72     -16.826   3.510   7.163  1.00 30.30           O  \\nATOM    561  OD2 ASP A  72     -16.075   5.552   6.898  1.00 30.30           O  \\nATOM    562  N   ILE A  73     -12.272   1.560   8.064  1.00 28.40           N  \\nATOM    563  CA  ILE A  73     -11.586   0.493   7.344  1.00 28.40           C  \\nATOM    564  C   ILE A  73     -12.305  -0.833   7.581  1.00 28.40           C  \\nATOM    565  O   ILE A  73     -12.470  -1.631   6.655  1.00 28.40           O  \\nATOM    566  CB  ILE A  73     -10.105   0.384   7.771  1.00 28.40           C  \\nATOM    567  CG1 ILE A  73      -9.326   1.624   7.318  1.00 28.40           C  \\nATOM    568  CG2 ILE A  73      -9.474  -0.894   7.211  1.00 28.40           C  \\nATOM    569  CD1 ILE A  73      -7.813   1.462   7.380  1.00 28.40           C  \\nATOM    570  N   LYS A  74     -12.518  -1.142   8.818  1.00 32.10           N  \\nATOM    571  CA  LYS A  74     -13.345  -2.337   8.961  1.00 32.10           C  \\nATOM    572  C   LYS A  74     -14.589  -2.253   8.081  1.00 32.10           C  \\nATOM    573  O   LYS A  74     -14.941  -3.220   7.402  1.00 32.10           O  \\nATOM    574  CB  LYS A  74     -13.750  -2.537  10.423  1.00 32.10           C  \\nATOM    575  CG  LYS A  74     -14.260  -3.936  10.738  1.00 32.10           C  \\nATOM    576  CD  LYS A  74     -14.560  -4.100  12.222  1.00 32.10           C  \\nATOM    577  CE  LYS A  74     -15.009  -5.518  12.549  1.00 32.10           C  \\nATOM    578  NZ  LYS A  74     -15.464  -5.642  13.966  1.00 32.10           N  \\nATOM    579  N   ALA A  75     -15.297  -1.175   8.192  1.00 34.65           N  \\nATOM    580  CA  ALA A  75     -16.462  -1.043   7.321  1.00 34.65           C  \\nATOM    581  C   ALA A  75     -16.061  -1.128   5.852  1.00 34.65           C  \\nATOM    582  O   ALA A  75     -16.712  -1.817   5.063  1.00 34.65           O  \\nATOM    583  CB  ALA A  75     -17.186   0.273   7.597  1.00 34.65           C  \\nATOM    584  N   ILE A  76     -15.079  -0.313   5.469  1.00 33.14           N  \\nATOM    585  CA  ILE A  76     -14.609  -0.460   4.096  1.00 33.14           C  \\nATOM    586  C   ILE A  76     -14.183  -1.906   3.849  1.00 33.14           C  \\nATOM    587  O   ILE A  76     -14.581  -2.515   2.852  1.00 33.14           O  \\nATOM    588  CB  ILE A  76     -13.440   0.503   3.793  1.00 33.14           C  \\nATOM    589  CG1 ILE A  76     -13.929   1.955   3.797  1.00 33.14           C  \\nATOM    590  CG2 ILE A  76     -12.783   0.150   2.455  1.00 33.14           C  \\nATOM    591  CD1 ILE A  76     -12.814   2.985   3.674  1.00 33.14           C  \\nATOM    592  N   LEU A  77     -13.415  -2.342   4.774  1.00 32.98           N  \\nATOM    593  CA  LEU A  77     -13.022  -3.733   4.582  1.00 32.98           C  \\nATOM    594  C   LEU A  77     -14.227  -4.660   4.701  1.00 32.98           C  \\nATOM    595  O   LEU A  77     -14.347  -5.628   3.947  1.00 32.98           O  \\nATOM    596  CB  LEU A  77     -11.953  -4.133   5.603  1.00 32.98           C  \\nATOM    597  CG  LEU A  77     -10.540  -3.605   5.347  1.00 32.98           C  \\nATOM    598  CD1 LEU A  77      -9.665  -3.816   6.578  1.00 32.98           C  \\nATOM    599  CD2 LEU A  77      -9.928  -4.282   4.126  1.00 32.98           C  \\nATOM    600  N   ASP A  78     -15.000  -4.403   5.733  1.00 36.22           N  \\nATOM    601  CA  ASP A  78     -16.248  -5.158   5.801  1.00 36.22           C  \\nATOM    602  C   ASP A  78     -17.068  -4.979   4.526  1.00 36.22           C  \\nATOM    603  O   ASP A  78     -17.628  -5.944   4.002  1.00 36.22           O  \\nATOM    604  CB  ASP A  78     -17.069  -4.729   7.019  1.00 36.22           C  \\nATOM    605  CG  ASP A  78     -16.682  -5.470   8.287  1.00 36.22           C  \\nATOM    606  OD1 ASP A  78     -15.903  -6.445   8.210  1.00 36.22           O  \\nATOM    607  OD2 ASP A  78     -17.163  -5.079   9.372  1.00 36.22           O  \\nATOM    608  N   LYS A  79     -17.266  -3.711   4.228  1.00 34.76           N  \\nATOM    609  CA  LYS A  79     -17.937  -3.521   2.945  1.00 34.76           C  \\nATOM    610  C   LYS A  79     -17.203  -4.256   1.827  1.00 34.76           C  \\nATOM    611  O   LYS A  79     -17.834  -4.854   0.952  1.00 34.76           O  \\nATOM    612  CB  LYS A  79     -18.045  -2.032   2.610  1.00 34.76           C  \\nATOM    613  CG  LYS A  79     -19.429  -1.445   2.842  1.00 34.76           C  \\nATOM    614  CD  LYS A  79     -19.444   0.060   2.605  1.00 34.76           C  \\nATOM    615  CE  LYS A  79     -20.794   0.669   2.960  1.00 34.76           C  \\nATOM    616  NZ  LYS A  79     -20.797   2.152   2.781  1.00 34.76           N  \\nATOM    617  N   LEU A  80     -15.910  -4.128   1.923  1.00 32.78           N  \\nATOM    618  CA  LEU A  80     -15.074  -4.881   0.995  1.00 32.78           C  \\nATOM    619  C   LEU A  80     -15.320  -6.380   1.134  1.00 32.78           C  \\nATOM    620  O   LEU A  80     -15.440  -7.089   0.132  1.00 32.78           O  \\nATOM    621  CB  LEU A  80     -13.594  -4.572   1.234  1.00 32.78           C  \\nATOM    622  CG  LEU A  80     -13.080  -3.243   0.677  1.00 32.78           C  \\nATOM    623  CD1 LEU A  80     -11.763  -2.862   1.344  1.00 32.78           C  \\nATOM    624  CD2 LEU A  80     -12.914  -3.327  -0.837  1.00 32.78           C  \\nATOM    625  N   HIS A  81     -15.318  -6.775   2.396  1.00 33.28           N  \\nATOM    626  CA  HIS A  81     -15.602  -8.172   2.704  1.00 33.28           C  \\nATOM    627  C   HIS A  81     -17.042  -8.531   2.354  1.00 33.28           C  \\nATOM    628  O   HIS A  81     -17.315  -9.641   1.890  1.00 33.28           O  \\nATOM    629  CB  HIS A  81     -15.334  -8.461   4.182  1.00 33.28           C  \\nATOM    630  CG  HIS A  81     -14.655  -9.771   4.424  1.00 33.28           C  \\nATOM    631  ND1 HIS A  81     -15.345 -10.960   4.518  1.00 33.28           N  \\nATOM    632  CD2 HIS A  81     -13.346 -10.078   4.588  1.00 33.28           C  \\nATOM    633  CE1 HIS A  81     -14.487 -11.944   4.731  1.00 33.28           C  \\nATOM    634  NE2 HIS A  81     -13.268 -11.436   4.777  1.00 33.28           N  \\nATOM    635  N   ASN A  82     -17.946  -7.765   2.930  1.00 34.62           N  \\nATOM    636  CA  ASN A  82     -19.343  -8.104   2.683  1.00 34.62           C  \\nATOM    637  C   ASN A  82     -19.685  -8.027   1.198  1.00 34.62           C  \\nATOM    638  O   ASN A  82     -20.668  -8.621   0.751  1.00 34.62           O  \\nATOM    639  CB  ASN A  82     -20.268  -7.188   3.489  1.00 34.62           C  \\nATOM    640  CG  ASN A  82     -20.351  -7.582   4.950  1.00 34.62           C  \\nATOM    641  OD1 ASN A  82     -20.024  -8.713   5.320  1.00 34.62           O  \\nATOM    642  ND2 ASN A  82     -20.789  -6.652   5.791  1.00 34.62           N  \\nATOM    643  N   LEU A  83     -18.929  -7.250   0.539  1.00 28.86           N  \\nATOM    644  CA  LEU A  83     -19.284  -7.177  -0.874  1.00 28.86           C  \\nATOM    645  C   LEU A  83     -18.758  -8.393  -1.630  1.00 28.86           C  \\nATOM    646  O   LEU A  83     -17.583  -8.439  -1.999  1.00 28.86           O  \\nATOM    647  CB  LEU A  83     -18.731  -5.894  -1.500  1.00 28.86           C  \\nATOM    648  CG  LEU A  83     -19.530  -4.615  -1.245  1.00 28.86           C  \\nATOM    649  CD1 LEU A  83     -18.606  -3.402  -1.247  1.00 28.86           C  \\nATOM    650  CD2 LEU A  83     -20.632  -4.456  -2.287  1.00 28.86           C  \\nATOM    651  N   ASN A  84     -18.816  -9.487  -0.983  1.00 29.74           N  \\nATOM    652  CA  ASN A  84     -18.746 -10.752  -1.707  1.00 29.74           C  \\nATOM    653  C   ASN A  84     -19.534 -10.695  -3.012  1.00 29.74           C  \\nATOM    654  O   ASN A  84     -19.795 -11.729  -3.630  1.00 29.74           O  \\nATOM    655  CB  ASN A  84     -19.251 -11.900  -0.831  1.00 29.74           C  \\nATOM    656  CG  ASN A  84     -18.127 -12.639  -0.131  1.00 29.74           C  \\nATOM    657  OD1 ASN A  84     -16.948 -12.334  -0.332  1.00 29.74           O  \\nATOM    658  ND2 ASN A  84     -18.483 -13.614   0.696  1.00 29.74           N  \\nATOM    659  N   SER A  85     -20.065  -9.548  -3.356  1.00 26.98           N  \\nATOM    660  CA  SER A  85     -20.500  -9.557  -4.748  1.00 26.98           C  \\nATOM    661  C   SER A  85     -19.336  -9.274  -5.693  1.00 26.98           C  \\nATOM    662  O   SER A  85     -18.702  -8.220  -5.606  1.00 26.98           O  \\nATOM    663  CB  SER A  85     -21.609  -8.528  -4.970  1.00 26.98           C  \\nATOM    664  OG  SER A  85     -22.234  -8.726  -6.227  1.00 26.98           O  \\nATOM    665  N   THR A  86     -18.290 -10.028  -5.606  1.00 28.60           N  \\nATOM    666  CA  THR A  86     -17.216 -10.093  -6.590  1.00 28.60           C  \\nATOM    667  C   THR A  86     -17.409  -9.035  -7.673  1.00 28.60           C  \\nATOM    668  O   THR A  86     -16.447  -8.396  -8.103  1.00 28.60           O  \\nATOM    669  CB  THR A  86     -17.137 -11.487  -7.239  1.00 28.60           C  \\nATOM    670  OG1 THR A  86     -18.416 -12.127  -7.141  1.00 28.60           O  \\nATOM    671  CG2 THR A  86     -16.092 -12.357  -6.549  1.00 28.60           C  \\nATOM    672  N   THR A  87     -18.698  -8.668  -7.947  1.00 27.56           N  \\nATOM    673  CA  THR A  87     -18.824  -7.790  -9.105  1.00 27.56           C  \\nATOM    674  C   THR A  87     -18.652  -6.329  -8.698  1.00 27.56           C  \\nATOM    675  O   THR A  87     -17.992  -5.558  -9.397  1.00 27.56           O  \\nATOM    676  CB  THR A  87     -20.186  -7.974  -9.800  1.00 27.56           C  \\nATOM    677  OG1 THR A  87     -20.488  -9.373  -9.882  1.00 27.56           O  \\nATOM    678  CG2 THR A  87     -20.169  -7.387 -11.208  1.00 27.56           C  \\nATOM    679  N   LEU A  88     -19.110  -6.028  -7.516  1.00 25.56           N  \\nATOM    680  CA  LEU A  88     -19.039  -4.586  -7.310  1.00 25.56           C  \\nATOM    681  C   LEU A  88     -17.646  -4.170  -6.850  1.00 25.56           C  \\nATOM    682  O   LEU A  88     -17.174  -3.083  -7.190  1.00 25.56           O  \\nATOM    683  CB  LEU A  88     -20.083  -4.139  -6.283  1.00 25.56           C  \\nATOM    684  CG  LEU A  88     -21.102  -3.102  -6.759  1.00 25.56           C  \\nATOM    685  CD1 LEU A  88     -22.451  -3.766  -7.016  1.00 25.56           C  \\nATOM    686  CD2 LEU A  88     -21.239  -1.978  -5.737  1.00 25.56           C  \\nATOM    687  N   ILE A  89     -17.014  -5.046  -6.141  1.00 26.85           N  \\nATOM    688  CA  ILE A  89     -15.613  -4.765  -5.846  1.00 26.85           C  \\nATOM    689  C   ILE A  89     -14.855  -4.506  -7.146  1.00 26.85           C  \\nATOM    690  O   ILE A  89     -14.019  -3.602  -7.214  1.00 26.85           O  \\nATOM    691  CB  ILE A  89     -14.957  -5.925  -5.065  1.00 26.85           C  \\nATOM    692  CG1 ILE A  89     -15.548  -6.020  -3.653  1.00 26.85           C  \\nATOM    693  CG2 ILE A  89     -13.436  -5.748  -5.011  1.00 26.85           C  \\nATOM    694  CD1 ILE A  89     -15.074  -7.233  -2.864  1.00 26.85           C  \\nATOM    695  N   LYS A  90     -15.163  -5.349  -8.108  1.00 27.72           N  \\nATOM    696  CA  LYS A  90     -14.685  -5.275  -9.485  1.00 27.72           C  \\nATOM    697  C   LYS A  90     -14.716  -3.839 -10.002  1.00 27.72           C  \\nATOM    698  O   LYS A  90     -13.717  -3.339 -10.521  1.00 27.72           O  \\nATOM    699  CB  LYS A  90     -15.521  -6.177 -10.394  1.00 27.72           C  \\nATOM    700  CG  LYS A  90     -14.965  -7.585 -10.549  1.00 27.72           C  \\nATOM    701  CD  LYS A  90     -15.813  -8.419 -11.501  1.00 27.72           C  \\nATOM    702  CE  LYS A  90     -15.346  -9.867 -11.545  1.00 27.72           C  \\nATOM    703  NZ  LYS A  90     -16.191 -10.695 -12.456  1.00 27.72           N  \\nATOM    704  N   GLU A  91     -15.856  -3.236  -9.700  1.00 27.23           N  \\nATOM    705  CA  GLU A  91     -16.189  -1.999 -10.401  1.00 27.23           C  \\nATOM    706  C   GLU A  91     -15.733  -0.776  -9.610  1.00 27.23           C  \\nATOM    707  O   GLU A  91     -15.195   0.174 -10.182  1.00 27.23           O  \\nATOM    708  CB  GLU A  91     -17.695  -1.922 -10.668  1.00 27.23           C  \\nATOM    709  CG  GLU A  91     -18.126  -2.601 -11.959  1.00 27.23           C  \\nATOM    710  CD  GLU A  91     -19.621  -2.505 -12.218  1.00 27.23           C  \\nATOM    711  OE1 GLU A  91     -20.080  -2.940 -13.298  1.00 27.23           O  \\nATOM    712  OE2 GLU A  91     -20.340  -1.989 -11.333  1.00 27.23           O  \\nATOM    713  N   LYS A  92     -15.817  -0.923  -8.281  1.00 25.89           N  \\nATOM    714  CA  LYS A  92     -15.603   0.369  -7.634  1.00 25.89           C  \\nATOM    715  C   LYS A  92     -14.134   0.565  -7.271  1.00 25.89           C  \\nATOM    716  O   LYS A  92     -13.625   1.687  -7.308  1.00 25.89           O  \\nATOM    717  CB  LYS A  92     -16.474   0.493  -6.383  1.00 25.89           C  \\nATOM    718  CG  LYS A  92     -17.771   1.257  -6.605  1.00 25.89           C  \\nATOM    719  CD  LYS A  92     -18.499   1.515  -5.292  1.00 25.89           C  \\nATOM    720  CE  LYS A  92     -19.777   2.314  -5.508  1.00 25.89           C  \\nATOM    721  NZ  LYS A  92     -20.495   2.569  -4.223  1.00 25.89           N  \\nATOM    722  N   CYS A  93     -13.493  -0.553  -6.875  1.00 27.40           N  \\nATOM    723  CA  CYS A  93     -12.079  -0.337  -6.588  1.00 27.40           C  \\nATOM    724  C   CYS A  93     -11.314   0.025  -7.855  1.00 27.40           C  \\nATOM    725  O   CYS A  93     -10.408   0.859  -7.822  1.00 27.40           O  \\nATOM    726  CB  CYS A  93     -11.465  -1.582  -5.949  1.00 27.40           C  \\nATOM    727  SG  CYS A  93     -11.481  -1.557  -4.143  1.00 27.40           S  \\nATOM    728  N   ILE A  94     -11.724  -0.569  -8.864  1.00 24.70           N  \\nATOM    729  CA  ILE A  94     -11.052  -0.324 -10.135  1.00 24.70           C  \\nATOM    730  C   ILE A  94     -11.310   1.112 -10.586  1.00 24.70           C  \\nATOM    731  O   ILE A  94     -10.413   1.774 -11.114  1.00 24.70           O  \\nATOM    732  CB  ILE A  94     -11.517  -1.319 -11.222  1.00 24.70           C  \\nATOM    733  CG1 ILE A  94     -11.127  -2.751 -10.837  1.00 24.70           C  \\nATOM    734  CG2 ILE A  94     -10.934  -0.941 -12.586  1.00 24.70           C  \\nATOM    735  CD1 ILE A  94     -11.692  -3.817 -11.765  1.00 24.70           C  \\nATOM    736  N   GLU A  95     -12.540   1.565 -10.356  1.00 23.35           N  \\nATOM    737  CA  GLU A  95     -12.775   2.916 -10.856  1.00 23.35           C  \\nATOM    738  C   GLU A  95     -11.888   3.932 -10.141  1.00 23.35           C  \\nATOM    739  O   GLU A  95     -11.378   4.866 -10.763  1.00 23.35           O  \\nATOM    740  CB  GLU A  95     -14.248   3.301 -10.697  1.00 23.35           C  \\nATOM    741  CG  GLU A  95     -14.991   3.451 -12.016  1.00 23.35           C  \\nATOM    742  CD  GLU A  95     -16.446   3.856 -11.843  1.00 23.35           C  \\nATOM    743  OE1 GLU A  95     -17.149   4.045 -12.862  1.00 23.35           O  \\nATOM    744  OE2 GLU A  95     -16.886   3.987 -10.679  1.00 23.35           O  \\nATOM    745  N   VAL A  96     -11.762   3.683  -8.856  1.00 24.17           N  \\nATOM    746  CA  VAL A  96     -11.095   4.784  -8.170  1.00 24.17           C  \\nATOM    747  C   VAL A  96      -9.613   4.801  -8.537  1.00 24.17           C  \\nATOM    748  O   VAL A  96      -9.034   5.868  -8.760  1.00 24.17           O  \\nATOM    749  CB  VAL A  96     -11.262   4.682  -6.637  1.00 24.17           C  \\nATOM    750  CG1 VAL A  96     -10.565   5.848  -5.938  1.00 24.17           C  \\nATOM    751  CG2 VAL A  96     -12.743   4.640  -6.264  1.00 24.17           C  \\nATOM    752  N   CYS A  97      -9.177   3.717  -9.021  1.00 26.35           N  \\nATOM    753  CA  CYS A  97      -7.727   3.796  -9.161  1.00 26.35           C  \\nATOM    754  C   CYS A  97      -7.330   4.010 -10.617  1.00 26.35           C  \\nATOM    755  O   CYS A  97      -6.314   4.647 -10.900  1.00 26.35           O  \\nATOM    756  CB  CYS A  97      -7.066   2.526  -8.624  1.00 26.35           C  \\nATOM    757  SG  CYS A  97      -6.827   2.532  -6.834  1.00 26.35           S  \\nATOM    758  N   LEU A  98      -8.206   3.682 -11.467  1.00 24.29           N  \\nATOM    759  CA  LEU A  98      -7.744   3.699 -12.851  1.00 24.29           C  \\nATOM    760  C   LEU A  98      -7.745   5.119 -13.407  1.00 24.29           C  \\nATOM    761  O   LEU A  98      -6.915   5.460 -14.253  1.00 24.29           O  \\nATOM    762  CB  LEU A  98      -8.623   2.797 -13.721  1.00 24.29           C  \\nATOM    763  CG  LEU A  98      -8.376   1.292 -13.604  1.00 24.29           C  \\nATOM    764  CD1 LEU A  98      -9.614   0.516 -14.044  1.00 24.29           C  \\nATOM    765  CD2 LEU A  98      -7.161   0.884 -14.430  1.00 24.29           C  \\nATOM    766  N   GLU A  99      -8.744   5.873 -12.999  1.00 25.16           N  \\nATOM    767  CA  GLU A  99      -8.764   7.128 -13.745  1.00 25.16           C  \\nATOM    768  C   GLU A  99      -7.526   7.968 -13.445  1.00 25.16           C  \\nATOM    769  O   GLU A  99      -7.053   8.715 -14.304  1.00 25.16           O  \\nATOM    770  CB  GLU A  99     -10.031   7.924 -13.424  1.00 25.16           C  \\nATOM    771  CG  GLU A  99     -11.274   7.420 -14.144  1.00 25.16           C  \\nATOM    772  CD  GLU A  99     -12.480   8.329 -13.969  1.00 25.16           C  \\nATOM    773  OE1 GLU A  99     -13.532   8.069 -14.595  1.00 25.16           O  \\nATOM    774  OE2 GLU A  99     -12.371   9.310 -13.199  1.00 25.16           O  \\nATOM    775  N   ASP A 100      -6.870   7.665 -12.366  1.00 26.54           N  \\nATOM    776  CA  ASP A 100      -5.791   8.628 -12.172  1.00 26.54           C  \\nATOM    777  C   ASP A 100      -4.427   7.974 -12.382  1.00 26.54           C  \\nATOM    778  O   ASP A 100      -3.394   8.563 -12.058  1.00 26.54           O  \\nATOM    779  CB  ASP A 100      -5.868   9.246 -10.774  1.00 26.54           C  \\nATOM    780  CG  ASP A 100      -6.703  10.514 -10.731  1.00 26.54           C  \\nATOM    781  OD1 ASP A 100      -6.921  11.136 -11.793  1.00 26.54           O  \\nATOM    782  OD2 ASP A 100      -7.143  10.895  -9.625  1.00 26.54           O  \\nATOM    783  N   ILE A 101      -4.476   6.774 -12.769  1.00 24.34           N  \\nATOM    784  CA  ILE A 101      -3.147   6.239 -13.044  1.00 24.34           C  \\nATOM    785  C   ILE A 101      -2.693   6.676 -14.435  1.00 24.34           C  \\nATOM    786  O   ILE A 101      -3.385   6.433 -15.426  1.00 24.34           O  \\nATOM    787  CB  ILE A 101      -3.125   4.698 -12.932  1.00 24.34           C  \\nATOM    788  CG1 ILE A 101      -3.615   4.256 -11.549  1.00 24.34           C  \\nATOM    789  CG2 ILE A 101      -1.720   4.156 -13.214  1.00 24.34           C  \\nATOM    790  CD1 ILE A 101      -3.604   2.748 -11.341  1.00 24.34           C  \\nATOM    791  N   PRO A 102      -2.075   7.867 -14.523  1.00 24.89           N  \\nATOM    792  CA  PRO A 102      -1.660   8.281 -15.866  1.00 24.89           C  \\nATOM    793  C   PRO A 102      -1.451   7.098 -16.809  1.00 24.89           C  \\nATOM    794  O   PRO A 102      -1.213   5.976 -16.354  1.00 24.89           O  \\nATOM    795  CB  PRO A 102      -0.343   9.020 -15.616  1.00 24.89           C  \\nATOM    796  CG  PRO A 102      -0.121   8.916 -14.141  1.00 24.89           C  \\nATOM    797  CD  PRO A 102      -1.261   8.146 -13.540  1.00 24.89           C  \\nATOM    798  N   ALA A 103      -2.269   6.962 -17.739  1.00 24.93           N  \\nATOM    799  CA  ALA A 103      -2.271   6.244 -19.011  1.00 24.93           C  \\nATOM    800  C   ALA A 103      -1.049   5.338 -19.130  1.00 24.93           C  \\nATOM    801  O   ALA A 103      -1.043   4.395 -19.926  1.00 24.93           O  \\nATOM    802  CB  ALA A 103      -2.315   7.228 -20.177  1.00 24.93           C  \\nATOM    803  N   TYR A 104      -0.096   5.493 -18.214  1.00 22.82           N  \\nATOM    804  CA  TYR A 104       1.080   4.666 -18.459  1.00 22.82           C  \\nATOM    805  C   TYR A 104       0.882   3.259 -17.907  1.00 22.82           C  \\nATOM    806  O   TYR A 104       1.597   2.330 -18.289  1.00 22.82           O  \\nATOM    807  CB  TYR A 104       2.325   5.302 -17.832  1.00 22.82           C  \\nATOM    808  CG  TYR A 104       3.152   6.105 -18.806  1.00 22.82           C  \\nATOM    809  CD1 TYR A 104       4.157   5.501 -19.559  1.00 22.82           C  \\nATOM    810  CD2 TYR A 104       2.932   7.467 -18.975  1.00 22.82           C  \\nATOM    811  CE1 TYR A 104       4.924   6.238 -20.456  1.00 22.82           C  \\nATOM    812  CE2 TYR A 104       3.693   8.213 -19.869  1.00 22.82           C  \\nATOM    813  CZ  TYR A 104       4.684   7.591 -20.605  1.00 22.82           C  \\nATOM    814  OH  TYR A 104       5.440   8.324 -21.492  1.00 22.82           O  \\nATOM    815  N   GLY A 105      -0.374   2.860 -17.580  1.00 28.74           N  \\nATOM    816  CA  GLY A 105      -0.552   1.696 -16.728  1.00 28.74           C  \\nATOM    817  C   GLY A 105      -1.470   0.649 -17.331  1.00 28.74           C  \\nATOM    818  O   GLY A 105      -1.915  -0.267 -16.638  1.00 28.74           O  \\nATOM    819  N   ARG A 106      -1.855   0.828 -18.582  1.00 28.92           N  \\nATOM    820  CA  ARG A 106      -2.588  -0.329 -19.085  1.00 28.92           C  \\nATOM    821  C   ARG A 106      -1.736  -1.592 -19.009  1.00 28.92           C  \\nATOM    822  O   ARG A 106      -2.227  -2.655 -18.623  1.00 28.92           O  \\nATOM    823  CB  ARG A 106      -3.045  -0.091 -20.526  1.00 28.92           C  \\nATOM    824  CG  ARG A 106      -4.534   0.182 -20.662  1.00 28.92           C  \\nATOM    825  CD  ARG A 106      -5.023  -0.058 -22.084  1.00 28.92           C  \\nATOM    826  NE  ARG A 106      -5.450   1.184 -22.724  1.00 28.92           N  \\nATOM    827  CZ  ARG A 106      -6.413   1.274 -23.637  1.00 28.92           C  \\nATOM    828  NH1 ARG A 106      -7.071   0.192 -24.038  1.00 28.92           N  \\nATOM    829  NH2 ARG A 106      -6.720   2.454 -24.154  1.00 28.92           N  \\nATOM    830  N   ASP A 107      -0.458  -1.355 -19.241  1.00 32.22           N  \\nATOM    831  CA  ASP A 107       0.356  -2.564 -19.334  1.00 32.22           C  \\nATOM    832  C   ASP A 107       0.992  -2.902 -17.988  1.00 32.22           C  \\nATOM    833  O   ASP A 107       1.500  -4.009 -17.797  1.00 32.22           O  \\nATOM    834  CB  ASP A 107       1.439  -2.400 -20.402  1.00 32.22           C  \\nATOM    835  CG  ASP A 107       0.883  -2.396 -21.815  1.00 32.22           C  \\nATOM    836  OD1 ASP A 107      -0.208  -2.962 -22.043  1.00 32.22           O  \\nATOM    837  OD2 ASP A 107       1.543  -1.824 -22.710  1.00 32.22           O  \\nATOM    838  N   SER A 108       1.069  -1.902 -17.152  1.00 31.34           N  \\nATOM    839  CA  SER A 108       1.707  -2.123 -15.858  1.00 31.34           C  \\nATOM    840  C   SER A 108       0.921  -3.124 -15.018  1.00 31.34           C  \\nATOM    841  O   SER A 108       1.502  -3.871 -14.228  1.00 31.34           O  \\nATOM    842  CB  SER A 108       1.848  -0.805 -15.097  1.00 31.34           C  \\nATOM    843  OG  SER A 108       2.703   0.088 -15.790  1.00 31.34           O  \\nATOM    844  N   TRP A 109      -0.314  -3.225 -15.293  1.00 31.60           N  \\nATOM    845  CA  TRP A 109      -1.001  -4.276 -14.548  1.00 31.60           C  \\nATOM    846  C   TRP A 109      -0.397  -5.642 -14.853  1.00 31.60           C  \\nATOM    847  O   TRP A 109      -0.276  -6.487 -13.963  1.00 31.60           O  \\nATOM    848  CB  TRP A 109      -2.496  -4.279 -14.878  1.00 31.60           C  \\nATOM    849  CG  TRP A 109      -3.329  -3.467 -13.933  1.00 31.60           C  \\nATOM    850  CD1 TRP A 109      -3.742  -3.826 -12.680  1.00 31.60           C  \\nATOM    851  CD2 TRP A 109      -3.856  -2.157 -14.168  1.00 31.60           C  \\nATOM    852  NE1 TRP A 109      -4.494  -2.819 -12.121  1.00 31.60           N  \\nATOM    853  CE2 TRP A 109      -4.579  -1.784 -13.013  1.00 31.60           C  \\nATOM    854  CE3 TRP A 109      -3.787  -1.262 -15.244  1.00 31.60           C  \\nATOM    855  CZ2 TRP A 109      -5.228  -0.552 -12.905  1.00 31.60           C  \\nATOM    856  CZ3 TRP A 109      -4.434  -0.037 -15.134  1.00 31.60           C  \\nATOM    857  CH2 TRP A 109      -5.145   0.305 -13.972  1.00 31.60           C  \\nATOM    858  N   LYS A 110      -0.436  -5.933 -16.229  1.00 34.07           N  \\nATOM    859  CA  LYS A 110       0.140  -7.210 -16.640  1.00 34.07           C  \\nATOM    860  C   LYS A 110       1.427  -7.504 -15.874  1.00 34.07           C  \\nATOM    861  O   LYS A 110       1.665  -8.641 -15.464  1.00 34.07           O  \\nATOM    862  CB  LYS A 110       0.412  -7.216 -18.145  1.00 34.07           C  \\nATOM    863  CG  LYS A 110      -0.286  -8.341 -18.895  1.00 34.07           C  \\nATOM    864  CD  LYS A 110      -0.057  -8.236 -20.398  1.00 34.07           C  \\nATOM    865  CE  LYS A 110      -0.780  -9.344 -21.152  1.00 34.07           C  \\nATOM    866  NZ  LYS A 110      -0.656  -9.179 -22.632  1.00 34.07           N  \\nATOM    867  N   LEU A 111       2.050  -6.401 -15.560  1.00 31.56           N  \\nATOM    868  CA  LEU A 111       3.321  -6.564 -14.862  1.00 31.56           C  \\nATOM    869  C   LEU A 111       3.095  -6.835 -13.378  1.00 31.56           C  \\nATOM    870  O   LEU A 111       3.824  -7.622 -12.769  1.00 31.56           O  \\nATOM    871  CB  LEU A 111       4.194  -5.319 -15.039  1.00 31.56           C  \\nATOM    872  CG  LEU A 111       5.028  -5.252 -16.319  1.00 31.56           C  \\nATOM    873  CD1 LEU A 111       5.317  -3.800 -16.687  1.00 31.56           C  \\nATOM    874  CD2 LEU A 111       6.326  -6.035 -16.153  1.00 31.56           C  \\nATOM    875  N   LEU A 112       2.200  -6.016 -12.854  1.00 31.16           N  \\nATOM    876  CA  LEU A 112       1.821  -6.328 -11.481  1.00 31.16           C  \\nATOM    877  C   LEU A 112       1.500  -7.811 -11.329  1.00 31.16           C  \\nATOM    878  O   LEU A 112       1.898  -8.440 -10.346  1.00 31.16           O  \\nATOM    879  CB  LEU A 112       0.617  -5.487 -11.052  1.00 31.16           C  \\nATOM    880  CG  LEU A 112       0.908  -4.316 -10.111  1.00 31.16           C  \\nATOM    881  CD1 LEU A 112       0.053  -3.110 -10.486  1.00 31.16           C  \\nATOM    882  CD2 LEU A 112       0.663  -4.722  -8.662  1.00 31.16           C  \\nATOM    883  N   LEU A 113       0.694  -8.184 -12.265  1.00 31.10           N  \\nATOM    884  CA  LEU A 113       0.323  -9.594 -12.260  1.00 31.10           C  \\nATOM    885  C   LEU A 113       1.559 -10.482 -12.360  1.00 31.10           C  \\nATOM    886  O   LEU A 113       1.656 -11.499 -11.669  1.00 31.10           O  \\nATOM    887  CB  LEU A 113      -0.635  -9.901 -13.415  1.00 31.10           C  \\nATOM    888  CG  LEU A 113      -2.099  -9.513 -13.203  1.00 31.10           C  \\nATOM    889  CD1 LEU A 113      -2.799  -9.330 -14.545  1.00 31.10           C  \\nATOM    890  CD2 LEU A 113      -2.814 -10.565 -12.361  1.00 31.10           C  \\nATOM    891  N   GLU A 114       2.405 -10.033 -13.233  1.00 34.49           N  \\nATOM    892  CA  GLU A 114       3.623 -10.802 -13.469  1.00 34.49           C  \\nATOM    893  C   GLU A 114       4.569 -10.717 -12.275  1.00 34.49           C  \\nATOM    894  O   GLU A 114       5.144 -11.725 -11.859  1.00 34.49           O  \\nATOM    895  CB  GLU A 114       4.330 -10.313 -14.736  1.00 34.49           C  \\nATOM    896  CG  GLU A 114       3.777 -10.915 -16.020  1.00 34.49           C  \\nATOM    897  CD  GLU A 114       4.522 -10.460 -17.265  1.00 34.49           C  \\nATOM    898  OE1 GLU A 114       4.181 -10.921 -18.378  1.00 34.49           O  \\nATOM    899  OE2 GLU A 114       5.454  -9.637 -17.126  1.00 34.49           O  \\nATOM    900  N   ASP A 115       4.610  -9.539 -11.731  1.00 33.62           N  \\nATOM    901  CA  ASP A 115       5.580  -9.339 -10.659  1.00 33.62           C  \\nATOM    902  C   ASP A 115       5.029  -9.830  -9.322  1.00 33.62           C  \\nATOM    903  O   ASP A 115       5.791 -10.236  -8.442  1.00 33.62           O  \\nATOM    904  CB  ASP A 115       5.971  -7.864 -10.557  1.00 33.62           C  \\nATOM    905  CG  ASP A 115       7.340  -7.571 -11.147  1.00 33.62           C  \\nATOM    906  OD1 ASP A 115       7.989  -8.501 -11.672  1.00 33.62           O  \\nATOM    907  OD2 ASP A 115       7.774  -6.400 -11.083  1.00 33.62           O  \\nATOM    908  N   GLY A 116       3.802  -9.584  -9.134  1.00 33.02           N  \\nATOM    909  CA  GLY A 116       3.309 -10.001  -7.830  1.00 33.02           C  \\nATOM    910  C   GLY A 116       3.146 -11.504  -7.707  1.00 33.02           C  \\nATOM    911  O   GLY A 116       2.819 -12.012  -6.632  1.00 33.02           O  \\nATOM    912  N   GLY A 117       3.798 -12.256  -8.608  1.00 33.20           N  \\nATOM    913  CA  GLY A 117       3.718 -13.695  -8.417  1.00 33.20           C  \\nATOM    914  C   GLY A 117       2.327 -14.252  -8.657  1.00 33.20           C  \\nATOM    915  O   GLY A 117       1.928 -15.233  -8.026  1.00 33.20           O  \\nATOM    916  N   CYS A 118       1.515 -13.377  -9.122  1.00 29.76           N  \\nATOM    917  CA  CYS A 118       0.220 -13.948  -9.474  1.00 29.76           C  \\nATOM    918  C   CYS A 118       0.376 -15.067 -10.497  1.00 29.76           C  \\nATOM    919  O   CYS A 118       1.168 -14.950 -11.434  1.00 29.76           O  \\nATOM    920  CB  CYS A 118      -0.711 -12.867 -10.025  1.00 29.76           C  \\nATOM    921  SG  CYS A 118      -2.409 -13.428 -10.276  1.00 29.76           S  \\nATOM    922  N   PRO A 119       0.147 -16.280 -10.015  1.00 31.04           N  \\nATOM    923  CA  PRO A 119       0.256 -17.295 -11.065  1.00 31.04           C  \\nATOM    924  C   PRO A 119      -0.242 -16.799 -12.420  1.00 31.04           C  \\nATOM    925  O   PRO A 119      -1.214 -16.042 -12.485  1.00 31.04           O  \\nATOM    926  CB  PRO A 119      -0.623 -18.435 -10.545  1.00 31.04           C  \\nATOM    927  CG  PRO A 119      -1.217 -17.912  -9.277  1.00 31.04           C  \\nATOM    928  CD  PRO A 119      -0.669 -16.536  -9.026  1.00 31.04           C  \\nATOM    929  N   ARG A 120       0.627 -16.348 -13.238  1.00 34.00           N  \\nATOM    930  CA  ARG A 120       0.318 -16.194 -14.656  1.00 34.00           C  \\nATOM    931  C   ARG A 120      -1.065 -16.751 -14.978  1.00 34.00           C  \\nATOM    932  O   ARG A 120      -1.830 -16.135 -15.724  1.00 34.00           O  \\nATOM    933  CB  ARG A 120       1.376 -16.889 -15.516  1.00 34.00           C  \\nATOM    934  CG  ARG A 120       2.159 -15.942 -16.413  1.00 34.00           C  \\nATOM    935  CD  ARG A 120       3.092 -16.696 -17.349  1.00 34.00           C  \\nATOM    936  NE  ARG A 120       3.721 -15.803 -18.318  1.00 34.00           N  \\nATOM    937  CZ  ARG A 120       4.762 -16.124 -19.081  1.00 34.00           C  \\nATOM    938  NH1 ARG A 120       5.314 -17.329 -19.004  1.00 34.00           N  \\nATOM    939  NH2 ARG A 120       5.255 -15.233 -19.929  1.00 34.00           N  \\nATOM    940  N   ASN A 121      -1.306 -17.883 -14.489  1.00 34.78           N  \\nATOM    941  CA  ASN A 121      -2.472 -18.562 -15.043  1.00 34.78           C  \\nATOM    942  C   ASN A 121      -3.679 -18.450 -14.116  1.00 34.78           C  \\nATOM    943  O   ASN A 121      -4.516 -19.353 -14.068  1.00 34.78           O  \\nATOM    944  CB  ASN A 121      -2.154 -20.031 -15.326  1.00 34.78           C  \\nATOM    945  CG  ASN A 121      -1.678 -20.263 -16.747  1.00 34.78           C  \\nATOM    946  OD1 ASN A 121      -1.729 -19.360 -17.586  1.00 34.78           O  \\nATOM    947  ND2 ASN A 121      -1.211 -21.474 -17.026  1.00 34.78           N  \\nATOM    948  N   SER A 122      -3.596 -17.709 -13.082  1.00 35.29           N  \\nATOM    949  CA  SER A 122      -4.869 -17.440 -12.421  1.00 35.29           C  \\nATOM    950  C   SER A 122      -5.772 -16.573 -13.293  1.00 35.29           C  \\nATOM    951  O   SER A 122      -5.395 -15.463 -13.675  1.00 35.29           O  \\nATOM    952  CB  SER A 122      -4.640 -16.756 -11.073  1.00 35.29           C  \\nATOM    953  OG  SER A 122      -3.600 -15.798 -11.165  1.00 35.29           O  \\nATOM    954  N   GLY A 123      -5.745 -16.878 -14.580  1.00 34.67           N  \\nATOM    955  CA  GLY A 123      -6.494 -16.408 -15.734  1.00 34.67           C  \\nATOM    956  C   GLY A 123      -7.565 -15.394 -15.379  1.00 34.67           C  \\nATOM    957  O   GLY A 123      -7.831 -14.469 -16.148  1.00 34.67           O  \\nATOM    958  N   THR A 124      -7.870 -15.193 -14.152  1.00 37.20           N  \\nATOM    959  CA  THR A 124      -8.918 -14.178 -14.147  1.00 37.20           C  \\nATOM    960  C   THR A 124      -8.509 -12.983 -13.291  1.00 37.20           C  \\nATOM    961  O   THR A 124      -7.626 -13.097 -12.438  1.00 37.20           O  \\nATOM    962  CB  THR A 124     -10.250 -14.752 -13.629  1.00 37.20           C  \\nATOM    963  OG1 THR A 124     -10.048 -15.308 -12.324  1.00 37.20           O  \\nATOM    964  CG2 THR A 124     -10.777 -15.842 -14.557  1.00 37.20           C  \\nATOM    965  N   LYS A 125      -8.484 -11.830 -13.923  1.00 35.39           N  \\nATOM    966  CA  LYS A 125      -8.479 -10.553 -13.216  1.00 35.39           C  \\nATOM    967  C   LYS A 125      -8.897 -10.730 -11.759  1.00 35.39           C  \\nATOM    968  O   LYS A 125      -8.308 -10.125 -10.860  1.00 35.39           O  \\nATOM    969  CB  LYS A 125      -9.405  -9.552 -13.909  1.00 35.39           C  \\nATOM    970  CG  LYS A 125      -8.705  -8.663 -14.926  1.00 35.39           C  \\nATOM    971  CD  LYS A 125      -9.681  -7.705 -15.596  1.00 35.39           C  \\nATOM    972  CE  LYS A 125      -9.019  -6.934 -16.730  1.00 35.39           C  \\nATOM    973  NZ  LYS A 125      -9.970  -5.988 -17.386  1.00 35.39           N  \\nATOM    974  N   THR A 126      -9.706 -11.726 -11.548  1.00 36.34           N  \\nATOM    975  CA  THR A 126     -10.302 -12.023 -10.251  1.00 36.34           C  \\nATOM    976  C   THR A 126      -9.260 -12.598  -9.296  1.00 36.34           C  \\nATOM    977  O   THR A 126      -9.172 -12.180  -8.139  1.00 36.34           O  \\nATOM    978  CB  THR A 126     -11.476 -13.010 -10.389  1.00 36.34           C  \\nATOM    979  OG1 THR A 126     -12.388 -12.522 -11.381  1.00 36.34           O  \\nATOM    980  CG2 THR A 126     -12.219 -13.169  -9.066  1.00 36.34           C  \\nATOM    981  N   GLU A 127      -8.621 -13.590  -9.794  1.00 39.03           N  \\nATOM    982  CA  GLU A 127      -7.674 -14.304  -8.943  1.00 39.03           C  \\nATOM    983  C   GLU A 127      -6.513 -13.403  -8.531  1.00 39.03           C  \\nATOM    984  O   GLU A 127      -6.037 -13.475  -7.397  1.00 39.03           O  \\nATOM    985  CB  GLU A 127      -7.144 -15.551  -9.656  1.00 39.03           C  \\nATOM    986  CG  GLU A 127      -8.072 -16.754  -9.562  1.00 39.03           C  \\nATOM    987  CD  GLU A 127      -7.522 -17.991 -10.253  1.00 39.03           C  \\nATOM    988  OE1 GLU A 127      -8.013 -19.110  -9.978  1.00 39.03           O  \\nATOM    989  OE2 GLU A 127      -6.592 -17.842 -11.077  1.00 39.03           O  \\nATOM    990  N   ALA A 128      -6.179 -12.575  -9.468  1.00 36.53           N  \\nATOM    991  CA  ALA A 128      -5.092 -11.637  -9.202  1.00 36.53           C  \\nATOM    992  C   ALA A 128      -5.464 -10.669  -8.083  1.00 36.53           C  \\nATOM    993  O   ALA A 128      -4.666 -10.420  -7.176  1.00 36.53           O  \\nATOM    994  CB  ALA A 128      -4.732 -10.866 -10.470  1.00 36.53           C  \\nATOM    995  N   ILE A 129      -6.675 -10.219  -8.188  1.00 38.12           N  \\nATOM    996  CA  ILE A 129      -7.181  -9.276  -7.196  1.00 38.12           C  \\nATOM    997  C   ILE A 129      -7.234  -9.948  -5.826  1.00 38.12           C  \\nATOM    998  O   ILE A 129      -6.819  -9.361  -4.823  1.00 38.12           O  \\nATOM    999  CB  ILE A 129      -8.577  -8.741  -7.585  1.00 38.12           C  \\nATOM   1000  CG1 ILE A 129      -8.481  -7.859  -8.835  1.00 38.12           C  \\nATOM   1001  CG2 ILE A 129      -9.205  -7.972  -6.419  1.00 38.12           C  \\nATOM   1002  CD1 ILE A 129      -9.829  -7.409  -9.381  1.00 38.12           C  \\nATOM   1003  N   HIS A 130      -7.764 -11.175  -5.874  1.00 40.63           N  \\nATOM   1004  CA  HIS A 130      -7.859 -11.893  -4.608  1.00 40.63           C  \\nATOM   1005  C   HIS A 130      -6.489 -12.044  -3.955  1.00 40.63           C  \\nATOM   1006  O   HIS A 130      -6.357 -11.897  -2.738  1.00 40.63           O  \\nATOM   1007  CB  HIS A 130      -8.494 -13.269  -4.819  1.00 40.63           C  \\nATOM   1008  CG  HIS A 130      -9.952 -13.314  -4.487  1.00 40.63           C  \\nATOM   1009  ND1 HIS A 130     -10.422 -13.255  -3.193  1.00 40.63           N  \\nATOM   1010  CD2 HIS A 130     -11.042 -13.410  -5.283  1.00 40.63           C  \\nATOM   1011  CE1 HIS A 130     -11.743 -13.314  -3.208  1.00 40.63           C  \\nATOM   1012  NE2 HIS A 130     -12.145 -13.408  -4.464  1.00 40.63           N  \\nATOM   1013  N   HIS A 131      -5.572 -12.238  -4.816  1.00 40.79           N  \\nATOM   1014  CA  HIS A 131      -4.220 -12.450  -4.311  1.00 40.79           C  \\nATOM   1015  C   HIS A 131      -3.643 -11.166  -3.725  1.00 40.79           C  \\nATOM   1016  O   HIS A 131      -3.049 -11.184  -2.644  1.00 40.79           O  \\nATOM   1017  CB  HIS A 131      -3.310 -12.976  -5.422  1.00 40.79           C  \\nATOM   1018  CG  HIS A 131      -1.991 -13.480  -4.929  1.00 40.79           C  \\nATOM   1019  ND1 HIS A 131      -1.871 -14.609  -4.148  1.00 40.79           N  \\nATOM   1020  CD2 HIS A 131      -0.735 -13.008  -5.110  1.00 40.79           C  \\nATOM   1021  CE1 HIS A 131      -0.594 -14.810  -3.869  1.00 40.79           C  \\nATOM   1022  NE2 HIS A 131       0.116 -13.853  -4.440  1.00 40.79           N  \\nATOM   1023  N   ILE A 132      -3.853 -10.111  -4.470  1.00 38.66           N  \\nATOM   1024  CA  ILE A 132      -3.362  -8.823  -3.993  1.00 38.66           C  \\nATOM   1025  C   ILE A 132      -4.000  -8.494  -2.645  1.00 38.66           C  \\nATOM   1026  O   ILE A 132      -3.318  -8.047  -1.720  1.00 38.66           O  \\nATOM   1027  CB  ILE A 132      -3.648  -7.696  -5.011  1.00 38.66           C  \\nATOM   1028  CG1 ILE A 132      -2.803  -7.891  -6.275  1.00 38.66           C  \\nATOM   1029  CG2 ILE A 132      -3.388  -6.323  -4.385  1.00 38.66           C  \\nATOM   1030  CD1 ILE A 132      -3.151  -6.933  -7.406  1.00 38.66           C  \\nATOM   1031  N   LEU A 133      -5.274  -8.771  -2.550  1.00 39.66           N  \\nATOM   1032  CA  LEU A 133      -5.979  -8.494  -1.303  1.00 39.66           C  \\nATOM   1033  C   LEU A 133      -5.413  -9.331  -0.161  1.00 39.66           C  \\nATOM   1034  O   LEU A 133      -5.257  -8.837   0.958  1.00 39.66           O  \\nATOM   1035  CB  LEU A 133      -7.476  -8.770  -1.461  1.00 39.66           C  \\nATOM   1036  CG  LEU A 133      -8.275  -7.740  -2.262  1.00 39.66           C  \\nATOM   1037  CD1 LEU A 133      -9.675  -8.269  -2.557  1.00 39.66           C  \\nATOM   1038  CD2 LEU A 133      -8.345  -6.415  -1.511  1.00 39.66           C  \\nATOM   1039  N   GLU A 134      -5.138 -10.509  -0.474  1.00 41.35           N  \\nATOM   1040  CA  GLU A 134      -4.553 -11.396   0.527  1.00 41.35           C  \\nATOM   1041  C   GLU A 134      -3.221 -10.852   1.036  1.00 41.35           C  \\nATOM   1042  O   GLU A 134      -2.944 -10.898   2.236  1.00 41.35           O  \\nATOM   1043  CB  GLU A 134      -4.361 -12.802  -0.048  1.00 41.35           C  \\nATOM   1044  CG  GLU A 134      -5.012 -13.900   0.782  1.00 41.35           C  \\nATOM   1045  CD  GLU A 134      -4.913 -15.275   0.142  1.00 41.35           C  \\nATOM   1046  OE1 GLU A 134      -5.416 -16.257   0.732  1.00 41.35           O  \\nATOM   1047  OE2 GLU A 134      -4.325 -15.371  -0.959  1.00 41.35           O  \\nATOM   1048  N   LEU A 135      -2.496 -10.385   0.074  1.00 38.21           N  \\nATOM   1049  CA  LEU A 135      -1.176  -9.874   0.430  1.00 38.21           C  \\nATOM   1050  C   LEU A 135      -1.292  -8.642   1.320  1.00 38.21           C  \\nATOM   1051  O   LEU A 135      -0.541  -8.496   2.287  1.00 38.21           O  \\nATOM   1052  CB  LEU A 135      -0.376  -9.534  -0.830  1.00 38.21           C  \\nATOM   1053  CG  LEU A 135       0.274 -10.712  -1.559  1.00 38.21           C  \\nATOM   1054  CD1 LEU A 135       0.641 -10.315  -2.985  1.00 38.21           C  \\nATOM   1055  CD2 LEU A 135       1.504 -11.197  -0.800  1.00 38.21           C  \\nATOM   1056  N   ILE A 136      -2.218  -7.828   0.962  1.00 38.12           N  \\nATOM   1057  CA  ILE A 136      -2.485  -6.633   1.755  1.00 38.12           C  \\nATOM   1058  C   ILE A 136      -2.859  -7.033   3.181  1.00 38.12           C  \\nATOM   1059  O   ILE A 136      -2.366  -6.446   4.146  1.00 38.12           O  \\nATOM   1060  CB  ILE A 136      -3.607  -5.777   1.127  1.00 38.12           C  \\nATOM   1061  CG1 ILE A 136      -3.137  -5.170  -0.200  1.00 38.12           C  \\nATOM   1062  CG2 ILE A 136      -4.061  -4.683   2.099  1.00 38.12           C  \\nATOM   1063  CD1 ILE A 136      -4.238  -4.472  -0.986  1.00 38.12           C  \\nATOM   1064  N   GLU A 137      -3.697  -8.018   3.289  1.00 38.61           N  \\nATOM   1065  CA  GLU A 137      -4.160  -8.511   4.583  1.00 38.61           C  \\nATOM   1066  C   GLU A 137      -2.993  -8.989   5.441  1.00 38.61           C  \\nATOM   1067  O   GLU A 137      -2.934  -8.698   6.637  1.00 38.61           O  \\nATOM   1068  CB  GLU A 137      -5.174  -9.642   4.396  1.00 38.61           C  \\nATOM   1069  CG  GLU A 137      -6.624  -9.188   4.480  1.00 38.61           C  \\nATOM   1070  CD  GLU A 137      -7.620 -10.316   4.263  1.00 38.61           C  \\nATOM   1071  OE1 GLU A 137      -8.845 -10.060   4.296  1.00 38.61           O  \\nATOM   1072  OE2 GLU A 137      -7.171 -11.467   4.059  1.00 38.61           O  \\nATOM   1073  N   LEU A 138      -2.200  -9.788   4.817  1.00 37.67           N  \\nATOM   1074  CA  LEU A 138      -1.081 -10.387   5.535  1.00 37.67           C  \\nATOM   1075  C   LEU A 138      -0.132  -9.313   6.057  1.00 37.67           C  \\nATOM   1076  O   LEU A 138       0.335  -9.392   7.196  1.00 37.67           O  \\nATOM   1077  CB  LEU A 138      -0.321 -11.359   4.628  1.00 37.67           C  \\nATOM   1078  CG  LEU A 138      -0.925 -12.755   4.473  1.00 37.67           C  \\nATOM   1079  CD1 LEU A 138      -0.412 -13.416   3.197  1.00 37.67           C  \\nATOM   1080  CD2 LEU A 138      -0.605 -13.614   5.692  1.00 37.67           C  \\nATOM   1081  N   GLU A 139       0.026  -8.331   5.222  1.00 36.33           N  \\nATOM   1082  CA  GLU A 139       0.957  -7.274   5.606  1.00 36.33           C  \\nATOM   1083  C   GLU A 139       0.376  -6.404   6.717  1.00 36.33           C  \\nATOM   1084  O   GLU A 139       1.085  -6.027   7.652  1.00 36.33           O  \\nATOM   1085  CB  GLU A 139       1.319  -6.409   4.396  1.00 36.33           C  \\nATOM   1086  CG  GLU A 139       2.729  -6.641   3.874  1.00 36.33           C  \\nATOM   1087  CD  GLU A 139       3.081  -5.762   2.684  1.00 36.33           C  \\nATOM   1088  OE1 GLU A 139       4.226  -5.844   2.186  1.00 36.33           O  \\nATOM   1089  OE2 GLU A 139       2.203  -4.985   2.247  1.00 36.33           O  \\nATOM   1090  N   GLN A 140      -0.845  -6.023   6.441  1.00 41.00           N  \\nATOM   1091  CA  GLN A 140      -1.501  -5.212   7.461  1.00 41.00           C  \\nATOM   1092  C   GLN A 140      -1.412  -5.875   8.832  1.00 41.00           C  \\nATOM   1093  O   GLN A 140      -1.216  -5.197   9.844  1.00 41.00           O  \\nATOM   1094  CB  GLN A 140      -2.964  -4.964   7.091  1.00 41.00           C  \\nATOM   1095  CG  GLN A 140      -3.156  -3.904   6.015  1.00 41.00           C  \\nATOM   1096  CD  GLN A 140      -4.614  -3.547   5.796  1.00 41.00           C  \\nATOM   1097  OE1 GLN A 140      -5.496  -4.004   6.530  1.00 41.00           O  \\nATOM   1098  NE2 GLN A 140      -4.878  -2.730   4.782  1.00 41.00           N  \\nATOM   1099  N   HIS A 141      -1.576  -7.171   8.698  1.00 39.41           N  \\nATOM   1100  CA  HIS A 141      -1.461  -7.906   9.953  1.00 39.41           C  \\nATOM   1101  C   HIS A 141      -0.080  -7.721  10.574  1.00 39.41           C  \\nATOM   1102  O   HIS A 141       0.037  -7.497  11.781  1.00 39.41           O  \\nATOM   1103  CB  HIS A 141      -1.743  -9.393   9.732  1.00 39.41           C  \\nATOM   1104  CG  HIS A 141      -1.821 -10.184  10.999  1.00 39.41           C  \\nATOM   1105  ND1 HIS A 141      -2.840 -10.026  11.913  1.00 39.41           N  \\nATOM   1106  CD2 HIS A 141      -1.007 -11.142  11.500  1.00 39.41           C  \\nATOM   1107  CE1 HIS A 141      -2.647 -10.854  12.925  1.00 39.41           C  \\nATOM   1108  NE2 HIS A 141      -1.542 -11.543  12.699  1.00 39.41           N  \\nATOM   1109  N   GLU A 142       0.866  -7.749   9.676  1.00 41.86           N  \\nATOM   1110  CA  GLU A 142       2.227  -7.660  10.196  1.00 41.86           C  \\nATOM   1111  C   GLU A 142       2.539  -6.251  10.690  1.00 41.86           C  \\nATOM   1112  O   GLU A 142       3.156  -6.079  11.743  1.00 41.86           O  \\nATOM   1113  CB  GLU A 142       3.240  -8.079   9.127  1.00 41.86           C  \\nATOM   1114  CG  GLU A 142       3.638  -9.546   9.198  1.00 41.86           C  \\nATOM   1115  CD  GLU A 142       4.642  -9.948   8.129  1.00 41.86           C  \\nATOM   1116  OE1 GLU A 142       5.016 -11.141   8.066  1.00 41.86           O  \\nATOM   1117  OE2 GLU A 142       5.057  -9.062   7.349  1.00 41.86           O  \\nATOM   1118  N   GLU A 143       2.043  -5.311   9.880  1.00 39.82           N  \\nATOM   1119  CA  GLU A 143       2.288  -3.926  10.269  1.00 39.82           C  \\nATOM   1120  C   GLU A 143       1.577  -3.586  11.576  1.00 39.82           C  \\nATOM   1121  O   GLU A 143       2.147  -2.922  12.444  1.00 39.82           O  \\nATOM   1122  CB  GLU A 143       1.840  -2.969   9.162  1.00 39.82           C  \\nATOM   1123  CG  GLU A 143       2.706  -1.724   9.037  1.00 39.82           C  \\nATOM   1124  CD  GLU A 143       2.444  -0.936   7.763  1.00 39.82           C  \\nATOM   1125  OE1 GLU A 143       3.167   0.052   7.501  1.00 39.82           O  \\nATOM   1126  OE2 GLU A 143       1.508  -1.309   7.022  1.00 39.82           O  \\nATOM   1127  N   ARG A 144       0.343  -4.065  11.613  1.00 38.38           N  \\nATOM   1128  CA  ARG A 144      -0.425  -3.878  12.840  1.00 38.38           C  \\nATOM   1129  C   ARG A 144       0.338  -4.407  14.049  1.00 38.38           C  \\nATOM   1130  O   ARG A 144       0.376  -3.761  15.098  1.00 38.38           O  \\nATOM   1131  CB  ARG A 144      -1.785  -4.571  12.734  1.00 38.38           C  \\nATOM   1132  CG  ARG A 144      -2.841  -4.001  13.668  1.00 38.38           C  \\nATOM   1133  CD  ARG A 144      -4.141  -4.790  13.597  1.00 38.38           C  \\nATOM   1134  NE  ARG A 144      -5.304  -3.931  13.800  1.00 38.38           N  \\nATOM   1135  CZ  ARG A 144      -6.398  -4.282  14.471  1.00 38.38           C  \\nATOM   1136  NH1 ARG A 144      -6.499  -5.488  15.019  1.00 38.38           N  \\nATOM   1137  NH2 ARG A 144      -7.398  -3.422  14.596  1.00 38.38           N  \\nATOM   1138  N   THR A 145       0.823  -5.579  13.808  1.00 41.78           N  \\nATOM   1139  CA  THR A 145       1.515  -6.259  14.897  1.00 41.78           C  \\nATOM   1140  C   THR A 145       2.738  -5.463  15.342  1.00 41.78           C  \\nATOM   1141  O   THR A 145       2.978  -5.301  16.541  1.00 41.78           O  \\nATOM   1142  CB  THR A 145       1.947  -7.679  14.484  1.00 41.78           C  \\nATOM   1143  OG1 THR A 145       0.816  -8.379  13.949  1.00 41.78           O  \\nATOM   1144  CG2 THR A 145       2.494  -8.456  15.676  1.00 41.78           C  \\nATOM   1145  N   GLN A 146       3.351  -4.877  14.335  1.00 38.58           N  \\nATOM   1146  CA  GLN A 146       4.551  -4.114  14.664  1.00 38.58           C  \\nATOM   1147  C   GLN A 146       4.196  -2.810  15.372  1.00 38.58           C  \\nATOM   1148  O   GLN A 146       4.877  -2.405  16.316  1.00 38.58           O  \\nATOM   1149  CB  GLN A 146       5.366  -3.824  13.403  1.00 38.58           C  \\nATOM   1150  CG  GLN A 146       6.170  -5.016  12.903  1.00 38.58           C  \\nATOM   1151  CD  GLN A 146       7.046  -4.674  11.711  1.00 38.58           C  \\nATOM   1152  OE1 GLN A 146       6.906  -3.606  11.107  1.00 38.58           O  \\nATOM   1153  NE2 GLN A 146       7.955  -5.579  11.364  1.00 38.58           N  \\nATOM   1154  N   LEU A 147       3.188  -2.213  14.814  1.00 33.95           N  \\nATOM   1155  CA  LEU A 147       2.769  -0.953  15.420  1.00 33.95           C  \\nATOM   1156  C   LEU A 147       2.307  -1.168  16.857  1.00 33.95           C  \\nATOM   1157  O   LEU A 147       2.613  -0.362  17.739  1.00 33.95           O  \\nATOM   1158  CB  LEU A 147       1.645  -0.312  14.601  1.00 33.95           C  \\nATOM   1159  CG  LEU A 147       1.774   1.187  14.330  1.00 33.95           C  \\nATOM   1160  CD1 LEU A 147       2.271   1.428  12.908  1.00 33.95           C  \\nATOM   1161  CD2 LEU A 147       0.441   1.890  14.562  1.00 33.95           C  \\nATOM   1162  N   ILE A 148       1.640  -2.289  17.036  1.00 37.46           N  \\nATOM   1163  CA  ILE A 148       1.179  -2.701  18.357  1.00 37.46           C  \\nATOM   1164  C   ILE A 148       2.370  -2.827  19.304  1.00 37.46           C  \\nATOM   1165  O   ILE A 148       2.325  -2.339  20.436  1.00 37.46           O  \\nATOM   1166  CB  ILE A 148       0.401  -4.034  18.294  1.00 37.46           C  \\nATOM   1167  CG1 ILE A 148      -0.933  -3.840  17.563  1.00 37.46           C  \\nATOM   1168  CG2 ILE A 148       0.178  -4.597  19.701  1.00 37.46           C  \\nATOM   1169  CD1 ILE A 148      -1.672  -5.138  17.269  1.00 37.46           C  \\nATOM   1170  N   SER A 149       3.330  -3.576  18.761  1.00 40.09           N  \\nATOM   1171  CA  SER A 149       4.516  -3.829  19.574  1.00 40.09           C  \\nATOM   1172  C   SER A 149       5.244  -2.531  19.907  1.00 40.09           C  \\nATOM   1173  O   SER A 149       5.658  -2.321  21.049  1.00 40.09           O  \\nATOM   1174  CB  SER A 149       5.466  -4.787  18.854  1.00 40.09           C  \\nATOM   1175  OG  SER A 149       6.730  -4.817  19.494  1.00 40.09           O  \\nATOM   1176  N   TYR A 150       5.237  -1.720  18.876  1.00 34.15           N  \\nATOM   1177  CA  TYR A 150       5.913  -0.439  19.052  1.00 34.15           C  \\nATOM   1178  C   TYR A 150       5.167   0.439  20.050  1.00 34.15           C  \\nATOM   1179  O   TYR A 150       5.783   1.078  20.906  1.00 34.15           O  \\nATOM   1180  CB  TYR A 150       6.044   0.289  17.711  1.00 34.15           C  \\nATOM   1181  CG  TYR A 150       6.913   1.521  17.772  1.00 34.15           C  \\nATOM   1182  CD1 TYR A 150       6.349   2.793  17.849  1.00 34.15           C  \\nATOM   1183  CD2 TYR A 150       8.300   1.417  17.750  1.00 34.15           C  \\nATOM   1184  CE1 TYR A 150       7.146   3.931  17.903  1.00 34.15           C  \\nATOM   1185  CE2 TYR A 150       9.107   2.549  17.803  1.00 34.15           C  \\nATOM   1186  CZ  TYR A 150       8.522   3.799  17.880  1.00 34.15           C  \\nATOM   1187  OH  TYR A 150       9.316   4.923  17.933  1.00 34.15           O  \\nATOM   1188  N   GLY A 151       3.867   0.438  19.913  1.00 33.68           N  \\nATOM   1189  CA  GLY A 151       3.052   1.234  20.817  1.00 33.68           C  \\nATOM   1190  C   GLY A 151       3.113   0.755  22.255  1.00 33.68           C  \\nATOM   1191  O   GLY A 151       3.188   1.564  23.182  1.00 33.68           O  \\nATOM   1192  N   ARG A 152       3.137  -0.500  22.440  1.00 37.17           N  \\nATOM   1193  CA  ARG A 152       3.268  -1.075  23.774  1.00 37.17           C  \\nATOM   1194  C   ARG A 152       4.593  -0.674  24.415  1.00 37.17           C  \\nATOM   1195  O   ARG A 152       4.642  -0.357  25.606  1.00 37.17           O  \\nATOM   1196  CB  ARG A 152       3.154  -2.600  23.717  1.00 37.17           C  \\nATOM   1197  CG  ARG A 152       1.735  -3.118  23.887  1.00 37.17           C  \\nATOM   1198  CD  ARG A 152       1.690  -4.640  23.910  1.00 37.17           C  \\nATOM   1199  NE  ARG A 152       0.333  -5.135  24.125  1.00 37.17           N  \\nATOM   1200  CZ  ARG A 152      -0.036  -6.408  24.008  1.00 37.17           C  \\nATOM   1201  NH1 ARG A 152       0.846  -7.342  23.675  1.00 37.17           N  \\nATOM   1202  NH2 ARG A 152      -1.298  -6.749  24.227  1.00 37.17           N  \\nATOM   1203  N   ASN A 153       5.633  -0.656  23.538  1.00 38.01           N  \\nATOM   1204  CA  ASN A 153       6.950  -0.338  24.079  1.00 38.01           C  \\nATOM   1205  C   ASN A 153       7.037   1.119  24.524  1.00 38.01           C  \\nATOM   1206  O   ASN A 153       7.774   1.444  25.456  1.00 38.01           O  \\nATOM   1207  CB  ASN A 153       8.041  -0.645  23.050  1.00 38.01           C  \\nATOM   1208  CG  ASN A 153       8.313  -2.130  22.913  1.00 38.01           C  \\nATOM   1209  OD1 ASN A 153       8.002  -2.916  23.812  1.00 38.01           O  \\nATOM   1210  ND2 ASN A 153       8.895  -2.525  21.787  1.00 38.01           N  \\nATOM   1211  N   LEU A 154       6.149   1.874  23.966  1.00 37.24           N  \\nATOM   1212  CA  LEU A 154       6.212   3.287  24.326  1.00 37.24           C  \\nATOM   1213  C   LEU A 154       5.149   3.632  25.363  1.00 37.24           C  \\nATOM   1214  O   LEU A 154       5.073   4.773  25.825  1.00 37.24           O  \\nATOM   1215  CB  LEU A 154       6.035   4.164  23.084  1.00 37.24           C  \\nATOM   1216  CG  LEU A 154       7.210   4.201  22.106  1.00 37.24           C  \\nATOM   1217  CD1 LEU A 154       6.757   4.740  20.753  1.00 37.24           C  \\nATOM   1218  CD2 LEU A 154       8.349   5.044  22.670  1.00 37.24           C  \\nATOM   1219  N   GLY A 155       4.348   2.596  25.694  1.00 38.22           N  \\nATOM   1220  CA  GLY A 155       3.325   2.821  26.703  1.00 38.22           C  \\nATOM   1221  C   GLY A 155       2.077   3.483  26.150  1.00 38.22           C  \\nATOM   1222  O   GLY A 155       1.397   4.227  26.860  1.00 38.22           O  \\nATOM   1223  N   LEU A 156       1.975   3.544  24.804  1.00 32.14           N  \\nATOM   1224  CA  LEU A 156       0.886   4.243  24.130  1.00 32.14           C  \\nATOM   1225  C   LEU A 156      -0.312   3.321  23.931  1.00 32.14           C  \\nATOM   1226  O   LEU A 156      -1.090   3.500  22.991  1.00 32.14           O  \\nATOM   1227  CB  LEU A 156       1.354   4.789  22.778  1.00 32.14           C  \\nATOM   1228  CG  LEU A 156       2.471   5.834  22.819  1.00 32.14           C  \\nATOM   1229  CD1 LEU A 156       3.040   6.057  21.422  1.00 32.14           C  \\nATOM   1230  CD2 LEU A 156       1.957   7.143  23.408  1.00 32.14           C  \\nATOM   1231  N   ASP A 157      -0.456   2.373  24.849  1.00 29.26           N  \\nATOM   1232  CA  ASP A 157      -1.521   1.380  24.746  1.00 29.26           C  \\nATOM   1233  C   ASP A 157      -2.882   2.049  24.569  1.00 29.26           C  \\nATOM   1234  O   ASP A 157      -3.748   1.530  23.862  1.00 29.26           O  \\nATOM   1235  CB  ASP A 157      -1.535   0.477  25.981  1.00 29.26           C  \\nATOM   1236  CG  ASP A 157      -0.712  -0.787  25.802  1.00 29.26           C  \\nATOM   1237  OD1 ASP A 157      -0.294  -1.086  24.663  1.00 29.26           O  \\nATOM   1238  OD2 ASP A 157      -0.483  -1.492  26.808  1.00 29.26           O  \\nATOM   1239  N   ASP A 158      -2.997   3.126  25.254  1.00 29.77           N  \\nATOM   1240  CA  ASP A 158      -4.304   3.776  25.277  1.00 29.77           C  \\nATOM   1241  C   ASP A 158      -4.663   4.334  23.902  1.00 29.77           C  \\nATOM   1242  O   ASP A 158      -5.834   4.338  23.514  1.00 29.77           O  \\nATOM   1243  CB  ASP A 158      -4.329   4.894  26.322  1.00 29.77           C  \\nATOM   1244  CG  ASP A 158      -4.106   4.389  27.736  1.00 29.77           C  \\nATOM   1245  OD1 ASP A 158      -4.399   3.206  28.015  1.00 29.77           O  \\nATOM   1246  OD2 ASP A 158      -3.635   5.182  28.580  1.00 29.77           O  \\nATOM   1247  N   ILE A 159      -3.674   4.844  23.275  1.00 27.55           N  \\nATOM   1248  CA  ILE A 159      -3.891   5.431  21.957  1.00 27.55           C  \\nATOM   1249  C   ILE A 159      -4.218   4.330  20.950  1.00 27.55           C  \\nATOM   1250  O   ILE A 159      -5.102   4.495  20.106  1.00 27.55           O  \\nATOM   1251  CB  ILE A 159      -2.661   6.238  21.487  1.00 27.55           C  \\nATOM   1252  CG1 ILE A 159      -2.368   7.383  22.465  1.00 27.55           C  \\nATOM   1253  CG2 ILE A 159      -2.874   6.772  20.068  1.00 27.55           C  \\nATOM   1254  CD1 ILE A 159      -1.049   8.097  22.205  1.00 27.55           C  \\nATOM   1255  N   ILE A 160      -3.443   3.233  21.173  1.00 27.13           N  \\nATOM   1256  CA  ILE A 160      -3.547   2.154  20.197  1.00 27.13           C  \\nATOM   1257  C   ILE A 160      -4.890   1.444  20.354  1.00 27.13           C  \\nATOM   1258  O   ILE A 160      -5.567   1.158  19.364  1.00 27.13           O  \\nATOM   1259  CB  ILE A 160      -2.387   1.145  20.344  1.00 27.13           C  \\nATOM   1260  CG1 ILE A 160      -1.039   1.846  20.136  1.00 27.13           C  \\nATOM   1261  CG2 ILE A 160      -2.555  -0.019  19.362  1.00 27.13           C  \\nATOM   1262  CD1 ILE A 160       0.159   0.907  20.168  1.00 27.13           C  \\nATOM   1263  N   TYR A 161      -5.308   1.248  21.616  1.00 30.81           N  \\nATOM   1264  CA  TYR A 161      -6.567   0.531  21.784  1.00 30.81           C  \\nATOM   1265  C   TYR A 161      -7.722   1.501  22.005  1.00 30.81           C  \\nATOM   1266  O   TYR A 161      -8.883   1.159  21.769  1.00 30.81           O  \\nATOM   1267  CB  TYR A 161      -6.476  -0.448  22.958  1.00 30.81           C  \\nATOM   1268  CG  TYR A 161      -5.652  -1.677  22.661  1.00 30.81           C  \\nATOM   1269  CD1 TYR A 161      -6.192  -2.745  21.946  1.00 30.81           C  \\nATOM   1270  CD2 TYR A 161      -4.335  -1.775  23.094  1.00 30.81           C  \\nATOM   1271  CE1 TYR A 161      -5.437  -3.881  21.671  1.00 30.81           C  \\nATOM   1272  CE2 TYR A 161      -3.571  -2.906  22.825  1.00 30.81           C  \\nATOM   1273  CZ  TYR A 161      -4.130  -3.952  22.114  1.00 30.81           C  \\nATOM   1274  OH  TYR A 161      -3.379  -5.074  21.844  1.00 30.81           O  \\nATOM   1275  N   GLY A 162      -7.552   2.791  21.777  1.00 26.08           N  \\nATOM   1276  CA  GLY A 162      -8.743   3.522  22.178  1.00 26.08           C  \\nATOM   1277  C   GLY A 162      -9.328   3.036  23.490  1.00 26.08           C  \\nATOM   1278  O   GLY A 162      -9.262   1.846  23.803  1.00 26.08           O  \\nATOM   1279  N   SER A 163      -8.631   3.341  24.599  1.00 23.13           N  \\nATOM   1280  CA  SER A 163      -9.186   3.141  25.934  1.00 23.13           C  \\nATOM   1281  C   SER A 163     -10.555   2.472  25.869  1.00 23.13           C  \\nATOM   1282  O   SER A 163     -11.456   2.955  25.181  1.00 23.13           O  \\nATOM   1283  CB  SER A 163      -9.296   4.474  26.675  1.00 23.13           C  \\nATOM   1284  OG  SER A 163      -9.078   5.560  25.790  1.00 23.13           O  \\nATOM   1285  N   ASP A 164     -10.637   1.351  25.467  1.00 25.30           N  \\nATOM   1286  CA  ASP A 164     -11.866   0.610  25.738  1.00 25.30           C  \\nATOM   1287  C   ASP A 164     -12.616   1.207  26.927  1.00 25.30           C  \\nATOM   1288  O   ASP A 164     -13.592   0.626  27.406  1.00 25.30           O  \\nATOM   1289  CB  ASP A 164     -11.556  -0.865  25.999  1.00 25.30           C  \\nATOM   1290  CG  ASP A 164     -11.423  -1.678  24.723  1.00 25.30           C  \\nATOM   1291  OD1 ASP A 164     -11.766  -1.166  23.636  1.00 25.30           O  \\nATOM   1292  OD2 ASP A 164     -10.974  -2.842  24.807  1.00 25.30           O  \\nATOM   1293  N   SER A 165     -12.095   2.268  27.518  1.00 26.30           N  \\nATOM   1294  CA  SER A 165     -12.850   2.618  28.717  1.00 26.30           C  \\nATOM   1295  C   SER A 165     -14.002   3.561  28.389  1.00 26.30           C  \\nATOM   1296  O   SER A 165     -14.710   4.021  29.288  1.00 26.30           O  \\nATOM   1297  CB  SER A 165     -11.933   3.263  29.758  1.00 26.30           C  \\nATOM   1298  OG  SER A 165     -10.742   2.511  29.914  1.00 26.30           O  \\nATOM   1299  N   GLU A 166     -14.498   3.611  27.149  1.00 27.27           N  \\nATOM   1300  CA  GLU A 166     -15.722   4.406  27.180  1.00 27.27           C  \\nATOM   1301  C   GLU A 166     -16.956   3.527  27.002  1.00 27.27           C  \\nATOM   1302  O   GLU A 166     -18.075   4.033  26.894  1.00 27.27           O  \\nATOM   1303  CB  GLU A 166     -15.687   5.489  26.099  1.00 27.27           C  \\nATOM   1304  CG  GLU A 166     -15.158   6.830  26.587  1.00 27.27           C  \\nATOM   1305  CD  GLU A 166     -14.053   7.395  25.709  1.00 27.27           C  \\nATOM   1306  OE1 GLU A 166     -13.591   8.529  25.969  1.00 27.27           O  \\nATOM   1307  OE2 GLU A 166     -13.647   6.699  24.752  1.00 27.27           O  \\nATOM   1308  N   GLU A 167     -16.797   2.224  26.758  1.00 25.69           N  \\nATOM   1309  CA  GLU A 167     -18.060   1.496  26.666  1.00 25.69           C  \\nATOM   1310  C   GLU A 167     -18.579   1.115  28.049  1.00 25.69           C  \\nATOM   1311  O   GLU A 167     -19.786   0.958  28.244  1.00 25.69           O  \\nATOM   1312  CB  GLU A 167     -17.896   0.243  25.802  1.00 25.69           C  \\nATOM   1313  CG  GLU A 167     -18.190   0.471  24.326  1.00 25.69           C  \\nATOM   1314  CD  GLU A 167     -17.979  -0.770  23.474  1.00 25.69           C  \\nATOM   1315  OE1 GLU A 167     -18.175  -0.699  22.239  1.00 25.69           O  \\nATOM   1316  OE2 GLU A 167     -17.617  -1.823  24.045  1.00 25.69           O  \\nATOM   1317  N   GLU A 168     -17.795   1.360  29.107  1.00 24.06           N  \\nATOM   1318  CA  GLU A 168     -18.479   0.985  30.341  1.00 24.06           C  \\nATOM   1319  C   GLU A 168     -19.124   2.199  31.004  1.00 24.06           C  \\nATOM   1320  O   GLU A 168     -19.728   2.081  32.072  1.00 24.06           O  \\nATOM   1321  CB  GLU A 168     -17.507   0.310  31.311  1.00 24.06           C  \\nATOM   1322  CG  GLU A 168     -17.579  -1.210  31.297  1.00 24.06           C  \\nATOM   1323  CD  GLU A 168     -16.537  -1.869  32.186  1.00 24.06           C  \\nATOM   1324  OE1 GLU A 168     -16.478  -3.119  32.228  1.00 24.06           O  \\nATOM   1325  OE2 GLU A 168     -15.774  -1.129  32.847  1.00 24.06           O  \\nATOM   1326  N   LYS A 169     -19.215   3.328  30.322  1.00 27.60           N  \\nATOM   1327  CA  LYS A 169     -20.064   4.316  30.981  1.00 27.60           C  \\nATOM   1328  C   LYS A 169     -21.482   4.285  30.417  1.00 27.60           C  \\nATOM   1329  O   LYS A 169     -22.356   5.019  30.882  1.00 27.60           O  \\nATOM   1330  CB  LYS A 169     -19.471   5.718  30.832  1.00 27.60           C  \\nATOM   1331  CG  LYS A 169     -18.528   6.114  31.958  1.00 27.60           C  \\nATOM   1332  CD  LYS A 169     -18.061   7.557  31.815  1.00 27.60           C  \\nATOM   1333  CE  LYS A 169     -17.048   7.927  32.890  1.00 27.60           C  \\nATOM   1334  NZ  LYS A 169     -16.505   9.304  32.689  1.00 27.60           N  \\nATOM   1335  N   ALA A 170     -21.738   3.367  29.521  1.00 26.90           N  \\nATOM   1336  CA  ALA A 170     -23.161   3.451  29.203  1.00 26.90           C  \\nATOM   1337  C   ALA A 170     -23.936   2.309  29.854  1.00 26.90           C  \\nATOM   1338  O   ALA A 170     -25.169   2.295  29.831  1.00 26.90           O  \\nATOM   1339  CB  ALA A 170     -23.371   3.438  27.691  1.00 26.90           C  \\nATOM   1340  N   GLU A 171     -23.315   1.665  30.806  1.00 25.00           N  \\nATOM   1341  CA  GLU A 171     -24.312   0.884  31.532  1.00 25.00           C  \\nATOM   1342  C   GLU A 171     -24.296   1.217  33.022  1.00 25.00           C  \\nATOM   1343  O   GLU A 171     -25.202   0.827  33.760  1.00 25.00           O  \\nATOM   1344  CB  GLU A 171     -24.076  -0.615  31.326  1.00 25.00           C  \\nATOM   1345  CG  GLU A 171     -24.866  -1.208  30.169  1.00 25.00           C  \\nATOM   1346  CD  GLU A 171     -24.586  -2.686  29.946  1.00 25.00           C  \\nATOM   1347  OE1 GLU A 171     -25.155  -3.275  28.999  1.00 25.00           O  \\nATOM   1348  OE2 GLU A 171     -23.793  -3.259  30.725  1.00 25.00           O  \\nATOM   1349  N   GLU A 172     -24.195   2.525  33.374  1.00 16.39           N  \\nATOM   1350  CA  GLU A 172     -24.758   2.905  34.667  1.00 16.39           C  \\nATOM   1351  C   GLU A 172     -25.470   4.252  34.584  1.00 16.39           C  \\nATOM   1352  O   GLU A 172     -26.483   4.466  35.252  1.00 16.39           O  \\nATOM   1353  CB  GLU A 172     -23.664   2.953  35.736  1.00 16.39           C  \\nATOM   1354  CG  GLU A 172     -23.370   1.604  36.376  1.00 16.39           C  \\nATOM   1355  CD  GLU A 172     -22.367   1.687  37.517  1.00 16.39           C  \\nATOM   1356  OE1 GLU A 172     -22.064   0.641  38.135  1.00 16.39           O  \\nATOM   1357  OE2 GLU A 172     -21.881   2.806  37.794  1.00 16.39           O  \",\"pdb\",{\"hbondCutoff\": 4.0});\n\tviewer_17225171870958037.setStyle({\"cartoon\": {\"colorscheme\": {\"prop\": \"b\", \"gradient\": \"roygb\", \"min\": 50, \"max\": 90}}});\n\tviewer_17225171870958037.zoomTo();\nviewer_17225171870958037.render();\n});\n</script>",
            "text/html": [
              "<div id=\"3dmolviewer_17225171870958037\"  style=\"position: relative; width: 800px; height: 480px;\">\n",
              "        <p id=\"3dmolwarning_17225171870958037\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n",
              "        </div>\n",
              "<script>\n",
              "\n",
              "var loadScriptAsync = function(uri){\n",
              "  return new Promise((resolve, reject) => {\n",
              "    //this is to ignore the existence of requirejs amd\n",
              "    var savedexports, savedmodule;\n",
              "    if (typeof exports !== 'undefined') savedexports = exports;\n",
              "    else exports = {}\n",
              "    if (typeof module !== 'undefined') savedmodule = module;\n",
              "    else module = {}\n",
              "\n",
              "    var tag = document.createElement('script');\n",
              "    tag.src = uri;\n",
              "    tag.async = true;\n",
              "    tag.onload = () => {\n",
              "        exports = savedexports;\n",
              "        module = savedmodule;\n",
              "        resolve();\n",
              "    };\n",
              "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
              "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
              "});\n",
              "};\n",
              "\n",
              "if(typeof $3Dmolpromise === 'undefined') {\n",
              "$3Dmolpromise = null;\n",
              "  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n",
              "}\n",
              "\n",
              "var viewer_17225171870958037 = null;\n",
              "var warn = document.getElementById(\"3dmolwarning_17225171870958037\");\n",
              "if(warn) {\n",
              "    warn.parentNode.removeChild(warn);\n",
              "}\n",
              "$3Dmolpromise.then(function() {\n",
              "viewer_17225171870958037 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_17225171870958037\"),{backgroundColor:\"white\"});\n",
              "viewer_17225171870958037.zoomTo();\n",
              "\tviewer_17225171870958037.addModel(\"ATOM      1  N   MET A   1      20.342  -5.221  -3.009  1.00 19.26           N  \\nATOM      2  CA  MET A   1      20.535  -3.913  -2.390  1.00 19.26           C  \\nATOM      3  C   MET A   1      19.208  -3.338  -1.907  1.00 19.26           C  \\nATOM      4  O   MET A   1      18.222  -3.334  -2.647  1.00 19.26           O  \\nATOM      5  CB  MET A   1      21.198  -2.945  -3.373  1.00 19.26           C  \\nATOM      6  CG  MET A   1      22.687  -2.756  -3.137  1.00 19.26           C  \\nATOM      7  SD  MET A   1      23.414  -1.470  -4.227  1.00 19.26           S  \\nATOM      8  CE  MET A   1      25.093  -2.130  -4.425  1.00 19.26           C  \\nATOM      9  N   ASN A   2      18.936  -3.437  -0.635  1.00 24.82           N  \\nATOM     10  CA  ASN A   2      18.253  -2.636   0.376  1.00 24.82           C  \\nATOM     11  C   ASN A   2      17.861  -1.265  -0.167  1.00 24.82           C  \\nATOM     12  O   ASN A   2      18.621  -0.648  -0.915  1.00 24.82           O  \\nATOM     13  CB  ASN A   2      19.128  -2.484   1.622  1.00 24.82           C  \\nATOM     14  CG  ASN A   2      18.782  -3.488   2.704  1.00 24.82           C  \\nATOM     15  OD1 ASN A   2      17.699  -4.080   2.695  1.00 24.82           O  \\nATOM     16  ND2 ASN A   2      19.698  -3.686   3.644  1.00 24.82           N  \\nATOM     17  N   PHE A   3      16.733  -1.136  -0.851  1.00 26.73           N  \\nATOM     18  CA  PHE A   3      16.253   0.239  -0.920  1.00 26.73           C  \\nATOM     19  C   PHE A   3      17.150   1.167  -0.111  1.00 26.73           C  \\nATOM     20  O   PHE A   3      17.429   0.904   1.061  1.00 26.73           O  \\nATOM     21  CB  PHE A   3      14.810   0.332  -0.416  1.00 26.73           C  \\nATOM     22  CG  PHE A   3      13.794   0.515  -1.511  1.00 26.73           C  \\nATOM     23  CD1 PHE A   3      13.466   1.786  -1.967  1.00 26.73           C  \\nATOM     24  CD2 PHE A   3      13.168  -0.584  -2.084  1.00 26.73           C  \\nATOM     25  CE1 PHE A   3      12.526   1.959  -2.981  1.00 26.73           C  \\nATOM     26  CE2 PHE A   3      12.228  -0.419  -3.097  1.00 26.73           C  \\nATOM     27  CZ  PHE A   3      11.908   0.854  -3.544  1.00 26.73           C  \\nATOM     28  N   SER A   4      18.058   1.794  -0.677  1.00 31.72           N  \\nATOM     29  CA  SER A   4      18.738   2.788   0.146  1.00 31.72           C  \\nATOM     30  C   SER A   4      17.741   3.719   0.827  1.00 31.72           C  \\nATOM     31  O   SER A   4      16.580   3.798   0.421  1.00 31.72           O  \\nATOM     32  CB  SER A   4      19.715   3.607  -0.700  1.00 31.72           C  \\nATOM     33  OG  SER A   4      19.059   4.710  -1.300  1.00 31.72           O  \\nATOM     34  N   SER A   5      17.853   3.851   2.111  1.00 33.25           N  \\nATOM     35  CA  SER A   5      17.126   4.857   2.879  1.00 33.25           C  \\nATOM     36  C   SER A   5      16.843   6.097   2.037  1.00 33.25           C  \\nATOM     37  O   SER A   5      15.799   6.734   2.191  1.00 33.25           O  \\nATOM     38  CB  SER A   5      17.913   5.249   4.130  1.00 33.25           C  \\nATOM     39  OG  SER A   5      18.026   4.150   5.018  1.00 33.25           O  \\nATOM     40  N   LYS A   6      17.716   6.326   1.037  1.00 36.79           N  \\nATOM     41  CA  LYS A   6      17.521   7.550   0.266  1.00 36.79           C  \\nATOM     42  C   LYS A   6      16.348   7.411  -0.699  1.00 36.79           C  \\nATOM     43  O   LYS A   6      15.500   8.302  -0.787  1.00 36.79           O  \\nATOM     44  CB  LYS A   6      18.793   7.909  -0.503  1.00 36.79           C  \\nATOM     45  CG  LYS A   6      19.007   9.404  -0.684  1.00 36.79           C  \\nATOM     46  CD  LYS A   6      20.292   9.697  -1.447  1.00 36.79           C  \\nATOM     47  CE  LYS A   6      20.458  11.188  -1.713  1.00 36.79           C  \\nATOM     48  NZ  LYS A   6      21.741  11.485  -2.418  1.00 36.79           N  \\nATOM     49  N   ASP A   7      16.314   6.310  -1.417  1.00 35.84           N  \\nATOM     50  CA  ASP A   7      15.257   6.105  -2.403  1.00 35.84           C  \\nATOM     51  C   ASP A   7      13.889   6.015  -1.730  1.00 35.84           C  \\nATOM     52  O   ASP A   7      12.904   6.553  -2.240  1.00 35.84           O  \\nATOM     53  CB  ASP A   7      15.525   4.841  -3.222  1.00 35.84           C  \\nATOM     54  CG  ASP A   7      16.715   4.979  -4.155  1.00 35.84           C  \\nATOM     55  OD1 ASP A   7      17.032   6.113  -4.575  1.00 35.84           O  \\nATOM     56  OD2 ASP A   7      17.340   3.945  -4.475  1.00 35.84           O  \\nATOM     57  N   ARG A   8      13.909   5.362  -0.564  1.00 35.59           N  \\nATOM     58  CA  ARG A   8      12.718   5.338   0.278  1.00 35.59           C  \\nATOM     59  C   ARG A   8      12.256   6.753   0.613  1.00 35.59           C  \\nATOM     60  O   ARG A   8      11.061   7.051   0.558  1.00 35.59           O  \\nATOM     61  CB  ARG A   8      12.984   4.555   1.566  1.00 35.59           C  \\nATOM     62  CG  ARG A   8      12.770   3.056   1.430  1.00 35.59           C  \\nATOM     63  CD  ARG A   8      13.043   2.327   2.739  1.00 35.59           C  \\nATOM     64  NE  ARG A   8      13.268   0.901   2.524  1.00 35.59           N  \\nATOM     65  CZ  ARG A   8      14.007   0.123   3.310  1.00 35.59           C  \\nATOM     66  NH1 ARG A   8      14.609   0.621   4.384  1.00 35.59           N  \\nATOM     67  NH2 ARG A   8      14.145  -1.163   3.022  1.00 35.59           N  \\nATOM     68  N   GLY A   9      13.346   7.498   0.933  1.00 33.98           N  \\nATOM     69  CA  GLY A   9      13.037   8.875   1.285  1.00 33.98           C  \\nATOM     70  C   GLY A   9      12.453   9.669   0.132  1.00 33.98           C  \\nATOM     71  O   GLY A   9      11.467  10.389   0.303  1.00 33.98           O  \\nATOM     72  N   GLU A  10      13.106   9.485  -1.017  1.00 35.96           N  \\nATOM     73  CA  GLU A  10      12.625  10.251  -2.163  1.00 35.96           C  \\nATOM     74  C   GLU A  10      11.196   9.861  -2.527  1.00 35.96           C  \\nATOM     75  O   GLU A  10      10.372  10.724  -2.839  1.00 35.96           O  \\nATOM     76  CB  GLU A  10      13.548  10.052  -3.368  1.00 35.96           C  \\nATOM     77  CG  GLU A  10      14.817  10.891  -3.317  1.00 35.96           C  \\nATOM     78  CD  GLU A  10      15.622  10.841  -4.605  1.00 35.96           C  \\nATOM     79  OE1 GLU A  10      16.582  11.630  -4.752  1.00 35.96           O  \\nATOM     80  OE2 GLU A  10      15.289  10.005  -5.475  1.00 35.96           O  \\nATOM     81  N   PHE A  11      10.908   8.518  -2.432  1.00 33.60           N  \\nATOM     82  CA  PHE A  11       9.574   8.015  -2.736  1.00 33.60           C  \\nATOM     83  C   PHE A  11       8.558   8.527  -1.722  1.00 33.60           C  \\nATOM     84  O   PHE A  11       7.482   8.999  -2.097  1.00 33.60           O  \\nATOM     85  CB  PHE A  11       9.567   6.484  -2.759  1.00 33.60           C  \\nATOM     86  CG  PHE A  11       8.250   5.888  -3.176  1.00 33.60           C  \\nATOM     87  CD1 PHE A  11       7.377   5.364  -2.230  1.00 33.60           C  \\nATOM     88  CD2 PHE A  11       7.883   5.853  -4.515  1.00 33.60           C  \\nATOM     89  CE1 PHE A  11       6.157   4.812  -2.613  1.00 33.60           C  \\nATOM     90  CE2 PHE A  11       6.666   5.303  -4.906  1.00 33.60           C  \\nATOM     91  CZ  PHE A  11       5.804   4.783  -3.953  1.00 33.60           C  \\nATOM     92  N   LEU A  12       8.915   8.417  -0.481  1.00 35.05           N  \\nATOM     93  CA  LEU A  12       8.045   8.927   0.573  1.00 35.05           C  \\nATOM     94  C   LEU A  12       7.765  10.413   0.377  1.00 35.05           C  \\nATOM     95  O   LEU A  12       6.638  10.869   0.581  1.00 35.05           O  \\nATOM     96  CB  LEU A  12       8.675   8.692   1.949  1.00 35.05           C  \\nATOM     97  CG  LEU A  12       8.477   7.303   2.557  1.00 35.05           C  \\nATOM     98  CD1 LEU A  12       9.563   7.016   3.589  1.00 35.05           C  \\nATOM     99  CD2 LEU A  12       7.092   7.186   3.184  1.00 35.05           C  \\nATOM    100  N   THR A  13       8.824  11.120  -0.076  1.00 37.91           N  \\nATOM    101  CA  THR A  13       8.690  12.553  -0.316  1.00 37.91           C  \\nATOM    102  C   THR A  13       7.724  12.821  -1.466  1.00 37.91           C  \\nATOM    103  O   THR A  13       6.872  13.707  -1.374  1.00 37.91           O  \\nATOM    104  CB  THR A  13      10.054  13.197  -0.627  1.00 37.91           C  \\nATOM    105  OG1 THR A  13      10.983  12.858   0.410  1.00 37.91           O  \\nATOM    106  CG2 THR A  13       9.938  14.715  -0.713  1.00 37.91           C  \\nATOM    107  N   PHE A  14       7.950  12.117  -2.587  1.00 34.13           N  \\nATOM    108  CA  PHE A  14       7.063  12.261  -3.735  1.00 34.13           C  \\nATOM    109  C   PHE A  14       5.612  12.026  -3.331  1.00 34.13           C  \\nATOM    110  O   PHE A  14       4.732  12.818  -3.672  1.00 34.13           O  \\nATOM    111  CB  PHE A  14       7.461  11.287  -4.849  1.00 34.13           C  \\nATOM    112  CG  PHE A  14       6.545  11.322  -6.042  1.00 34.13           C  \\nATOM    113  CD1 PHE A  14       5.556  10.359  -6.205  1.00 34.13           C  \\nATOM    114  CD2 PHE A  14       6.671  12.319  -7.001  1.00 34.13           C  \\nATOM    115  CE1 PHE A  14       4.706  10.389  -7.309  1.00 34.13           C  \\nATOM    116  CE2 PHE A  14       5.826  12.356  -8.106  1.00 34.13           C  \\nATOM    117  CZ  PHE A  14       4.845  11.389  -8.258  1.00 34.13           C  \\nATOM    118  N   ALA A  15       5.321  11.009  -2.556  1.00 35.05           N  \\nATOM    119  CA  ALA A  15       3.966  10.659  -2.137  1.00 35.05           C  \\nATOM    120  C   ALA A  15       3.388  11.723  -1.208  1.00 35.05           C  \\nATOM    121  O   ALA A  15       2.221  12.101  -1.337  1.00 35.05           O  \\nATOM    122  CB  ALA A  15       3.957   9.295  -1.451  1.00 35.05           C  \\nATOM    123  N   ALA A  16       4.214  12.010  -0.186  1.00 36.31           N  \\nATOM    124  CA  ALA A  16       3.776  13.072   0.716  1.00 36.31           C  \\nATOM    125  C   ALA A  16       3.390  14.327  -0.060  1.00 36.31           C  \\nATOM    126  O   ALA A  16       2.389  14.975   0.252  1.00 36.31           O  \\nATOM    127  CB  ALA A  16       4.871  13.393   1.731  1.00 36.31           C  \\nATOM    128  N   LYS A  17       4.177  14.552  -1.125  1.00 37.13           N  \\nATOM    129  CA  LYS A  17       3.895  15.732  -1.936  1.00 37.13           C  \\nATOM    130  C   LYS A  17       2.565  15.589  -2.671  1.00 37.13           C  \\nATOM    131  O   LYS A  17       1.791  16.545  -2.757  1.00 37.13           O  \\nATOM    132  CB  LYS A  17       5.024  15.977  -2.939  1.00 37.13           C  \\nATOM    133  CG  LYS A  17       6.106  16.919  -2.434  1.00 37.13           C  \\nATOM    134  CD  LYS A  17       7.173  17.164  -3.493  1.00 37.13           C  \\nATOM    135  CE  LYS A  17       8.324  17.999  -2.947  1.00 37.13           C  \\nATOM    136  NZ  LYS A  17       9.372  18.243  -3.983  1.00 37.13           N  \\nATOM    137  N   CYS A  18       2.370  14.413  -3.193  1.00 33.27           N  \\nATOM    138  CA  CYS A  18       1.122  14.193  -3.916  1.00 33.27           C  \\nATOM    139  C   CYS A  18      -0.074  14.272  -2.976  1.00 33.27           C  \\nATOM    140  O   CYS A  18      -1.113  14.831  -3.332  1.00 33.27           O  \\nATOM    141  CB  CYS A  18       1.142  12.837  -4.620  1.00 33.27           C  \\nATOM    142  SG  CYS A  18       2.165  12.803  -6.108  1.00 33.27           S  \\nATOM    143  N   GLY A  19       0.168  13.624  -1.777  1.00 33.08           N  \\nATOM    144  CA  GLY A  19      -0.874  13.715  -0.767  1.00 33.08           C  \\nATOM    145  C   GLY A  19      -1.165  15.141  -0.337  1.00 33.08           C  \\nATOM    146  O   GLY A  19      -2.327  15.522  -0.177  1.00 33.08           O  \\nATOM    147  N   LYS A  20      -0.084  15.861  -0.183  1.00 36.99           N  \\nATOM    148  CA  LYS A  20      -0.295  17.256   0.192  1.00 36.99           C  \\nATOM    149  C   LYS A  20      -1.025  18.018  -0.911  1.00 36.99           C  \\nATOM    150  O   LYS A  20      -1.909  18.830  -0.631  1.00 36.99           O  \\nATOM    151  CB  LYS A  20       1.039  17.936   0.504  1.00 36.99           C  \\nATOM    152  CG  LYS A  20       1.473  17.811   1.957  1.00 36.99           C  \\nATOM    153  CD  LYS A  20       2.714  18.647   2.243  1.00 36.99           C  \\nATOM    154  CE  LYS A  20       3.215  18.436   3.666  1.00 36.99           C  \\nATOM    155  NZ  LYS A  20       4.449  19.230   3.942  1.00 36.99           N  \\nATOM    156  N   LYS A  21      -0.487  17.855  -2.114  1.00 36.37           N  \\nATOM    157  CA  LYS A  21      -1.130  18.551  -3.225  1.00 36.37           C  \\nATOM    158  C   LYS A  21      -2.621  18.232  -3.284  1.00 36.37           C  \\nATOM    159  O   LYS A  21      -3.441  19.114  -3.545  1.00 36.37           O  \\nATOM    160  CB  LYS A  21      -0.462  18.182  -4.550  1.00 36.37           C  \\nATOM    161  CG  LYS A  21      -0.677  19.203  -5.657  1.00 36.37           C  \\nATOM    162  CD  LYS A  21       0.053  18.805  -6.934  1.00 36.37           C  \\nATOM    163  CE  LYS A  21      -0.241  19.774  -8.072  1.00 36.37           C  \\nATOM    164  NZ  LYS A  21       0.485  19.397  -9.321  1.00 36.37           N  \\nATOM    165  N   HIS A  22      -2.923  16.976  -2.857  1.00 31.47           N  \\nATOM    166  CA  HIS A  22      -4.329  16.624  -3.016  1.00 31.47           C  \\nATOM    167  C   HIS A  22      -5.024  16.506  -1.664  1.00 31.47           C  \\nATOM    168  O   HIS A  22      -6.209  16.169  -1.597  1.00 31.47           O  \\nATOM    169  CB  HIS A  22      -4.469  15.315  -3.795  1.00 31.47           C  \\nATOM    170  CG  HIS A  22      -4.007  15.408  -5.214  1.00 31.47           C  \\nATOM    171  ND1 HIS A  22      -4.714  16.088  -6.183  1.00 31.47           N  \\nATOM    172  CD2 HIS A  22      -2.908  14.909  -5.826  1.00 31.47           C  \\nATOM    173  CE1 HIS A  22      -4.066  16.002  -7.333  1.00 31.47           C  \\nATOM    174  NE2 HIS A  22      -2.967  15.292  -7.144  1.00 31.47           N  \\nATOM    175  N   LYS A  23      -4.304  16.943  -0.647  1.00 33.11           N  \\nATOM    176  CA  LYS A  23      -4.820  16.880   0.717  1.00 33.11           C  \\nATOM    177  C   LYS A  23      -5.295  15.472   1.061  1.00 33.11           C  \\nATOM    178  O   LYS A  23      -6.340  15.300   1.692  1.00 33.11           O  \\nATOM    179  CB  LYS A  23      -5.963  17.879   0.905  1.00 33.11           C  \\nATOM    180  CG  LYS A  23      -5.518  19.335   0.907  1.00 33.11           C  \\nATOM    181  CD  LYS A  23      -6.662  20.268   1.282  1.00 33.11           C  \\nATOM    182  CE  LYS A  23      -6.234  21.728   1.230  1.00 33.11           C  \\nATOM    183  NZ  LYS A  23      -7.350  22.647   1.605  1.00 33.11           N  \\nATOM    184  N   ARG A  24      -4.766  14.528   0.459  1.00 32.11           N  \\nATOM    185  CA  ARG A  24      -5.162  13.129   0.584  1.00 32.11           C  \\nATOM    186  C   ARG A  24      -4.118  12.332   1.358  1.00 32.11           C  \\nATOM    187  O   ARG A  24      -2.919  12.599   1.249  1.00 32.11           O  \\nATOM    188  CB  ARG A  24      -5.379  12.506  -0.797  1.00 32.11           C  \\nATOM    189  CG  ARG A  24      -6.703  12.885  -1.442  1.00 32.11           C  \\nATOM    190  CD  ARG A  24      -6.875  12.226  -2.804  1.00 32.11           C  \\nATOM    191  NE  ARG A  24      -7.613  13.083  -3.728  1.00 32.11           N  \\nATOM    192  CZ  ARG A  24      -7.873  12.783  -4.997  1.00 32.11           C  \\nATOM    193  NH1 ARG A  24      -7.458  11.634  -5.520  1.00 32.11           N  \\nATOM    194  NH2 ARG A  24      -8.552  13.636  -5.749  1.00 32.11           N  \\nATOM    195  N   LYS A  25      -4.513  11.775   2.477  1.00 28.54           N  \\nATOM    196  CA  LYS A  25      -3.564  10.892   3.148  1.00 28.54           C  \\nATOM    197  C   LYS A  25      -3.399   9.580   2.386  1.00 28.54           C  \\nATOM    198  O   LYS A  25      -4.378   9.015   1.893  1.00 28.54           O  \\nATOM    199  CB  LYS A  25      -4.013  10.611   4.583  1.00 28.54           C  \\nATOM    200  CG  LYS A  25      -3.541  11.647   5.593  1.00 28.54           C  \\nATOM    201  CD  LYS A  25      -3.863  11.222   7.020  1.00 28.54           C  \\nATOM    202  CE  LYS A  25      -3.480  12.301   8.025  1.00 28.54           C  \\nATOM    203  NZ  LYS A  25      -3.807  11.895   9.425  1.00 28.54           N  \\nATOM    204  N   ALA A  26      -2.226   9.407   1.842  1.00 28.49           N  \\nATOM    205  CA  ALA A  26      -1.820   8.259   1.034  1.00 28.49           C  \\nATOM    206  C   ALA A  26      -1.350   7.106   1.916  1.00 28.49           C  \\nATOM    207  O   ALA A  26      -0.676   7.324   2.926  1.00 28.49           O  \\nATOM    208  CB  ALA A  26      -0.719   8.660   0.055  1.00 28.49           C  \\nATOM    209  N   ILE A  27      -1.995   6.070   1.927  1.00 28.34           N  \\nATOM    210  CA  ILE A  27      -1.384   4.929   2.601  1.00 28.34           C  \\nATOM    211  C   ILE A  27      -0.755   3.996   1.569  1.00 28.34           C  \\nATOM    212  O   ILE A  27      -1.353   3.721   0.526  1.00 28.34           O  \\nATOM    213  CB  ILE A  27      -2.414   4.162   3.460  1.00 28.34           C  \\nATOM    214  CG1 ILE A  27      -3.061   5.101   4.484  1.00 28.34           C  \\nATOM    215  CG2 ILE A  27      -1.755   2.966   4.153  1.00 28.34           C  \\nATOM    216  CD1 ILE A  27      -4.155   4.448   5.319  1.00 28.34           C  \\nATOM    217  N   PHE A  28       0.214   3.179   2.000  1.00 26.57           N  \\nATOM    218  CA  PHE A  28       1.266   2.559   1.202  1.00 26.57           C  \\nATOM    219  C   PHE A  28       1.099   1.044   1.170  1.00 26.57           C  \\nATOM    220  O   PHE A  28       0.980   0.405   2.218  1.00 26.57           O  \\nATOM    221  CB  PHE A  28       2.647   2.924   1.755  1.00 26.57           C  \\nATOM    222  CG  PHE A  28       3.103   4.309   1.384  1.00 26.57           C  \\nATOM    223  CD1 PHE A  28       3.851   4.526   0.234  1.00 26.57           C  \\nATOM    224  CD2 PHE A  28       2.782   5.396   2.187  1.00 26.57           C  \\nATOM    225  CE1 PHE A  28       4.274   5.807  -0.112  1.00 26.57           C  \\nATOM    226  CE2 PHE A  28       3.201   6.680   1.849  1.00 26.57           C  \\nATOM    227  CZ  PHE A  28       3.948   6.883   0.699  1.00 26.57           C  \\nATOM    228  N   VAL A  29       1.193   0.417   0.120  1.00 26.65           N  \\nATOM    229  CA  VAL A  29       1.338  -1.030   0.242  1.00 26.65           C  \\nATOM    230  C   VAL A  29       2.568  -1.493  -0.536  1.00 26.65           C  \\nATOM    231  O   VAL A  29       2.718  -1.176  -1.718  1.00 26.65           O  \\nATOM    232  CB  VAL A  29       0.079  -1.771  -0.260  1.00 26.65           C  \\nATOM    233  CG1 VAL A  29       0.140  -3.251   0.112  1.00 26.65           C  \\nATOM    234  CG2 VAL A  29      -1.182  -1.125   0.310  1.00 26.65           C  \\nATOM    235  N   THR A  30       3.658  -1.545   0.037  1.00 27.23           N  \\nATOM    236  CA  THR A  30       4.823  -2.065  -0.671  1.00 27.23           C  \\nATOM    237  C   THR A  30       4.768  -3.587  -0.758  1.00 27.23           C  \\nATOM    238  O   THR A  30       4.133  -4.239   0.073  1.00 27.23           O  \\nATOM    239  CB  THR A  30       6.132  -1.634   0.017  1.00 27.23           C  \\nATOM    240  OG1 THR A  30       6.681  -2.751   0.729  1.00 27.23           O  \\nATOM    241  CG2 THR A  30       5.888  -0.492   0.997  1.00 27.23           C  \\nATOM    242  N   GLY A  31       4.924  -4.134  -1.893  1.00 26.19           N  \\nATOM    243  CA  GLY A  31       5.068  -5.560  -2.142  1.00 26.19           C  \\nATOM    244  C   GLY A  31       6.496  -6.050  -1.998  1.00 26.19           C  \\nATOM    245  O   GLY A  31       7.414  -5.489  -2.600  1.00 26.19           O  \\nATOM    246  N   SER A  32       7.047  -6.276  -0.823  1.00 24.92           N  \\nATOM    247  CA  SER A  32       8.323  -6.969  -0.970  1.00 24.92           C  \\nATOM    248  C   SER A  32       8.182  -8.458  -0.670  1.00 24.92           C  \\nATOM    249  O   SER A  32       7.392  -8.851   0.191  1.00 24.92           O  \\nATOM    250  CB  SER A  32       9.377  -6.353  -0.049  1.00 24.92           C  \\nATOM    251  OG  SER A  32       9.065  -6.604   1.311  1.00 24.92           O  \\nATOM    252  N   ARG A  33       8.693  -9.399  -1.484  1.00 24.59           N  \\nATOM    253  CA  ARG A  33       9.346 -10.608  -0.992  1.00 24.59           C  \\nATOM    254  C   ARG A  33      10.065 -11.340  -2.121  1.00 24.59           C  \\nATOM    255  O   ARG A  33       9.440 -11.738  -3.106  1.00 24.59           O  \\nATOM    256  CB  ARG A  33       8.328 -11.538  -0.329  1.00 24.59           C  \\nATOM    257  CG  ARG A  33       8.186 -11.327   1.170  1.00 24.59           C  \\nATOM    258  CD  ARG A  33       7.311 -12.396   1.810  1.00 24.59           C  \\nATOM    259  NE  ARG A  33       7.206 -12.213   3.255  1.00 24.59           N  \\nATOM    260  CZ  ARG A  33       6.553 -13.030   4.076  1.00 24.59           C  \\nATOM    261  NH1 ARG A  33       5.930 -14.106   3.610  1.00 24.59           N  \\nATOM    262  NH2 ARG A  33       6.521 -12.769   5.375  1.00 24.59           N  \\nATOM    263  N   GLY A  34      11.362 -11.122  -2.264  1.00 22.55           N  \\nATOM    264  CA  GLY A  34      12.088 -12.185  -2.940  1.00 22.55           C  \\nATOM    265  C   GLY A  34      13.337 -12.620  -2.198  1.00 22.55           C  \\nATOM    266  O   GLY A  34      13.778 -11.945  -1.266  1.00 22.55           O  \\nATOM    267  N   THR A  35      13.476 -13.813  -1.827  1.00 22.69           N  \\nATOM    268  CA  THR A  35      14.695 -14.564  -1.547  1.00 22.69           C  \\nATOM    269  C   THR A  35      15.899 -13.913  -2.222  1.00 22.69           C  \\nATOM    270  O   THR A  35      15.775 -13.346  -3.310  1.00 22.69           O  \\nATOM    271  CB  THR A  35      14.570 -16.026  -2.013  1.00 22.69           C  \\nATOM    272  OG1 THR A  35      14.055 -16.053  -3.350  1.00 22.69           O  \\nATOM    273  CG2 THR A  35      13.634 -16.815  -1.104  1.00 22.69           C  \\nATOM    274  N   ARG A  36      16.632 -13.027  -1.470  1.00 23.57           N  \\nATOM    275  CA  ARG A  36      17.886 -12.281  -1.524  1.00 23.57           C  \\nATOM    276  C   ARG A  36      18.638 -12.568  -2.819  1.00 23.57           C  \\nATOM    277  O   ARG A  36      19.173 -11.652  -3.448  1.00 23.57           O  \\nATOM    278  CB  ARG A  36      18.766 -12.621  -0.320  1.00 23.57           C  \\nATOM    279  CG  ARG A  36      18.402 -11.860   0.945  1.00 23.57           C  \\nATOM    280  CD  ARG A  36      19.277 -12.269   2.122  1.00 23.57           C  \\nATOM    281  NE  ARG A  36      19.369 -11.208   3.120  1.00 23.57           N  \\nATOM    282  CZ  ARG A  36      20.028 -11.305   4.271  1.00 23.57           C  \\nATOM    283  NH1 ARG A  36      20.670 -12.422   4.593  1.00 23.57           N  \\nATOM    284  NH2 ARG A  36      20.047 -10.277   5.107  1.00 23.57           N  \\nATOM    285  N   SER A  37      18.651 -13.793  -3.376  1.00 23.37           N  \\nATOM    286  CA  SER A  37      19.697 -14.026  -4.366  1.00 23.37           C  \\nATOM    287  C   SER A  37      19.174 -13.818  -5.783  1.00 23.37           C  \\nATOM    288  O   SER A  37      19.904 -13.348  -6.658  1.00 23.37           O  \\nATOM    289  CB  SER A  37      20.263 -15.440  -4.226  1.00 23.37           C  \\nATOM    290  OG  SER A  37      19.434 -16.381  -4.885  1.00 23.37           O  \\nATOM    291  N   ASP A  38      17.924 -14.032  -5.972  1.00 26.30           N  \\nATOM    292  CA  ASP A  38      17.442 -13.880  -7.341  1.00 26.30           C  \\nATOM    293  C   ASP A  38      17.088 -12.425  -7.641  1.00 26.30           C  \\nATOM    294  O   ASP A  38      16.953 -12.041  -8.804  1.00 26.30           O  \\nATOM    295  CB  ASP A  38      16.227 -14.778  -7.585  1.00 26.30           C  \\nATOM    296  CG  ASP A  38      16.592 -16.247  -7.714  1.00 26.30           C  \\nATOM    297  OD1 ASP A  38      17.761 -16.560  -8.028  1.00 26.30           O  \\nATOM    298  OD2 ASP A  38      15.702 -17.098  -7.503  1.00 26.30           O  \\nATOM    299  N   GLU A  39      17.116 -11.671  -6.538  1.00 25.48           N  \\nATOM    300  CA  GLU A  39      16.894 -10.230  -6.609  1.00 25.48           C  \\nATOM    301  C   GLU A  39      17.946  -9.553  -7.482  1.00 25.48           C  \\nATOM    302  O   GLU A  39      17.635  -8.622  -8.229  1.00 25.48           O  \\nATOM    303  CB  GLU A  39      16.899  -9.616  -5.207  1.00 25.48           C  \\nATOM    304  CG  GLU A  39      15.528  -9.576  -4.548  1.00 25.48           C  \\nATOM    305  CD  GLU A  39      15.552  -8.984  -3.148  1.00 25.48           C  \\nATOM    306  OE1 GLU A  39      14.472  -8.839  -2.532  1.00 25.48           O  \\nATOM    307  OE2 GLU A  39      16.660  -8.662  -2.664  1.00 25.48           O  \\nATOM    308  N   MET A  40      19.098 -10.098  -7.437  1.00 22.65           N  \\nATOM    309  CA  MET A  40      20.196  -9.370  -8.067  1.00 22.65           C  \\nATOM    310  C   MET A  40      20.121  -9.480  -9.586  1.00 22.65           C  \\nATOM    311  O   MET A  40      20.442  -8.527 -10.298  1.00 22.65           O  \\nATOM    312  CB  MET A  40      21.544  -9.894  -7.569  1.00 22.65           C  \\nATOM    313  CG  MET A  40      22.393  -8.840  -6.877  1.00 22.65           C  \\nATOM    314  SD  MET A  40      23.746  -9.573  -5.877  1.00 22.65           S  \\nATOM    315  CE  MET A  40      25.033  -9.732  -7.147  1.00 22.65           C  \\nATOM    316  N   ALA A  41      19.631 -10.621  -9.985  1.00 25.44           N  \\nATOM    317  CA  ALA A  41      19.750 -10.800 -11.430  1.00 25.44           C  \\nATOM    318  C   ALA A  41      18.748  -9.922 -12.175  1.00 25.44           C  \\nATOM    319  O   ALA A  41      19.074  -9.344 -13.215  1.00 25.44           O  \\nATOM    320  CB  ALA A  41      19.548 -12.266 -11.802  1.00 25.44           C  \\nATOM    321  N   GLY A  42      17.666  -9.760 -11.538  1.00 26.60           N  \\nATOM    322  CA  GLY A  42      16.696  -9.030 -12.339  1.00 26.60           C  \\nATOM    323  C   GLY A  42      16.845  -7.524 -12.230  1.00 26.60           C  \\nATOM    324  O   GLY A  42      16.531  -6.794 -13.172  1.00 26.60           O  \\nATOM    325  N   TYR A  43      17.698  -7.174 -11.288  1.00 24.07           N  \\nATOM    326  CA  TYR A  43      17.898  -5.762 -10.983  1.00 24.07           C  \\nATOM    327  C   TYR A  43      18.847  -5.116 -11.984  1.00 24.07           C  \\nATOM    328  O   TYR A  43      18.656  -3.963 -12.376  1.00 24.07           O  \\nATOM    329  CB  TYR A  43      18.442  -5.591  -9.561  1.00 24.07           C  \\nATOM    330  CG  TYR A  43      17.852  -4.414  -8.823  1.00 24.07           C  \\nATOM    331  CD1 TYR A  43      18.441  -3.154  -8.897  1.00 24.07           C  \\nATOM    332  CD2 TYR A  43      16.706  -4.561  -8.049  1.00 24.07           C  \\nATOM    333  CE1 TYR A  43      17.901  -2.067  -8.216  1.00 24.07           C  \\nATOM    334  CE2 TYR A  43      16.158  -3.481  -7.364  1.00 24.07           C  \\nATOM    335  CZ  TYR A  43      16.761  -2.240  -7.454  1.00 24.07           C  \\nATOM    336  OH  TYR A  43      16.223  -1.168  -6.778  1.00 24.07           O  \\nATOM    337  N   GLU A  44      20.053  -5.666 -12.051  1.00 26.05           N  \\nATOM    338  CA  GLU A  44      20.923  -4.988 -13.008  1.00 26.05           C  \\nATOM    339  C   GLU A  44      20.165  -4.627 -14.282  1.00 26.05           C  \\nATOM    340  O   GLU A  44      20.485  -3.636 -14.941  1.00 26.05           O  \\nATOM    341  CB  GLU A  44      22.135  -5.860 -13.345  1.00 26.05           C  \\nATOM    342  CG  GLU A  44      23.270  -5.753 -12.337  1.00 26.05           C  \\nATOM    343  CD  GLU A  44      24.624  -6.131 -12.916  1.00 26.05           C  \\nATOM    344  OE1 GLU A  44      25.653  -5.930 -12.232  1.00 26.05           O  \\nATOM    345  OE2 GLU A  44      24.656  -6.631 -14.062  1.00 26.05           O  \\nATOM    346  N   GLN A  45      19.059  -5.270 -14.483  1.00 26.60           N  \\nATOM    347  CA  GLN A  45      18.412  -4.773 -15.693  1.00 26.60           C  \\nATOM    348  C   GLN A  45      17.568  -3.537 -15.397  1.00 26.60           C  \\nATOM    349  O   GLN A  45      17.360  -2.696 -16.274  1.00 26.60           O  \\nATOM    350  CB  GLN A  45      17.544  -5.862 -16.325  1.00 26.60           C  \\nATOM    351  CG  GLN A  45      18.321  -6.839 -17.197  1.00 26.60           C  \\nATOM    352  CD  GLN A  45      17.446  -7.940 -17.767  1.00 26.60           C  \\nATOM    353  OE1 GLN A  45      16.285  -8.094 -17.376  1.00 26.60           O  \\nATOM    354  NE2 GLN A  45      17.998  -8.714 -18.696  1.00 26.60           N  \\nATOM    355  N   PHE A  46      17.314  -3.170 -14.142  1.00 22.47           N  \\nATOM    356  CA  PHE A  46      16.252  -2.178 -14.265  1.00 22.47           C  \\nATOM    357  C   PHE A  46      16.679  -0.849 -13.652  1.00 22.47           C  \\nATOM    358  O   PHE A  46      15.841  -0.083 -13.172  1.00 22.47           O  \\nATOM    359  CB  PHE A  46      14.968  -2.676 -13.595  1.00 22.47           C  \\nATOM    360  CG  PHE A  46      14.238  -3.726 -14.388  1.00 22.47           C  \\nATOM    361  CD1 PHE A  46      13.443  -3.372 -15.471  1.00 22.47           C  \\nATOM    362  CD2 PHE A  46      14.347  -5.069 -14.051  1.00 22.47           C  \\nATOM    363  CE1 PHE A  46      12.766  -4.342 -16.207  1.00 22.47           C  \\nATOM    364  CE2 PHE A  46      13.674  -6.044 -14.781  1.00 22.47           C  \\nATOM    365  CZ  PHE A  46      12.883  -5.678 -15.859  1.00 22.47           C  \\nATOM    366  N   ARG A  47      18.041  -0.763 -13.387  1.00 26.21           N  \\nATOM    367  CA  ARG A  47      18.601   0.573 -13.210  1.00 26.21           C  \\nATOM    368  C   ARG A  47      17.892   1.587 -14.101  1.00 26.21           C  \\nATOM    369  O   ARG A  47      17.663   1.328 -15.285  1.00 26.21           O  \\nATOM    370  CB  ARG A  47      20.102   0.570 -13.510  1.00 26.21           C  \\nATOM    371  CG  ARG A  47      20.978   0.544 -12.268  1.00 26.21           C  \\nATOM    372  CD  ARG A  47      22.447   0.753 -12.610  1.00 26.21           C  \\nATOM    373  NE  ARG A  47      23.273   0.834 -11.409  1.00 26.21           N  \\nATOM    374  CZ  ARG A  47      24.589   1.026 -11.406  1.00 26.21           C  \\nATOM    375  NH1 ARG A  47      25.257   1.163 -12.546  1.00 26.21           N  \\nATOM    376  NH2 ARG A  47      25.243   1.083 -10.255  1.00 26.21           N  \\nATOM    377  N   GLY A  48      17.188   2.564 -13.588  1.00 26.84           N  \\nATOM    378  CA  GLY A  48      16.745   3.762 -14.283  1.00 26.84           C  \\nATOM    379  C   GLY A  48      15.394   3.596 -14.955  1.00 26.84           C  \\nATOM    380  O   GLY A  48      14.969   4.459 -15.726  1.00 26.84           O  \\nATOM    381  N   LYS A  49      14.758   2.475 -14.847  1.00 24.35           N  \\nATOM    382  CA  LYS A  49      13.395   2.418 -15.368  1.00 24.35           C  \\nATOM    383  C   LYS A  49      12.379   2.785 -14.291  1.00 24.35           C  \\nATOM    384  O   LYS A  49      12.368   2.190 -13.211  1.00 24.35           O  \\nATOM    385  CB  LYS A  49      13.091   1.026 -15.923  1.00 24.35           C  \\nATOM    386  CG  LYS A  49      13.787   0.719 -17.241  1.00 24.35           C  \\nATOM    387  CD  LYS A  49      13.298  -0.594 -17.838  1.00 24.35           C  \\nATOM    388  CE  LYS A  49      14.020  -0.921 -19.139  1.00 24.35           C  \\nATOM    389  NZ  LYS A  49      13.574  -2.228 -19.707  1.00 24.35           N  \\nATOM    390  N   PRO A  50      12.038   4.055 -14.194  1.00 23.93           N  \\nATOM    391  CA  PRO A  50      11.130   4.564 -13.164  1.00 23.93           C  \\nATOM    392  C   PRO A  50       9.960   3.620 -12.890  1.00 23.93           C  \\nATOM    393  O   PRO A  50       9.214   3.819 -11.928  1.00 23.93           O  \\nATOM    394  CB  PRO A  50      10.637   5.888 -13.752  1.00 23.93           C  \\nATOM    395  CG  PRO A  50      11.203   5.927 -15.134  1.00 23.93           C  \\nATOM    396  CD  PRO A  50      12.106   4.740 -15.314  1.00 23.93           C  \\nATOM    397  N   ASN A  51       9.856   2.506 -13.542  1.00 23.66           N  \\nATOM    398  CA  ASN A  51       8.621   1.745 -13.388  1.00 23.66           C  \\nATOM    399  C   ASN A  51       8.659   0.860 -12.144  1.00 23.66           C  \\nATOM    400  O   ASN A  51       7.688   0.164 -11.843  1.00 23.66           O  \\nATOM    401  CB  ASN A  51       8.352   0.898 -14.633  1.00 23.66           C  \\nATOM    402  CG  ASN A  51       7.750   1.703 -15.768  1.00 23.66           C  \\nATOM    403  OD1 ASN A  51       7.207   2.790 -15.554  1.00 23.66           O  \\nATOM    404  ND2 ASN A  51       7.842   1.176 -16.983  1.00 23.66           N  \\nATOM    405  N   ALA A  52       9.828   0.659 -11.617  1.00 23.02           N  \\nATOM    406  CA  ALA A  52       9.854  -0.401 -10.613  1.00 23.02           C  \\nATOM    407  C   ALA A  52       9.137   0.033  -9.339  1.00 23.02           C  \\nATOM    408  O   ALA A  52       8.710  -0.806  -8.542  1.00 23.02           O  \\nATOM    409  CB  ALA A  52      11.294  -0.802 -10.300  1.00 23.02           C  \\nATOM    410  N   ILE A  53       8.752   1.254  -9.265  1.00 22.86           N  \\nATOM    411  CA  ILE A  53       8.136   1.761  -8.044  1.00 22.86           C  \\nATOM    412  C   ILE A  53       6.658   1.378  -8.014  1.00 22.86           C  \\nATOM    413  O   ILE A  53       5.992   1.525  -6.986  1.00 22.86           O  \\nATOM    414  CB  ILE A  53       8.295   3.293  -7.925  1.00 22.86           C  \\nATOM    415  CG1 ILE A  53       9.779   3.677  -7.923  1.00 22.86           C  \\nATOM    416  CG2 ILE A  53       7.593   3.813  -6.666  1.00 22.86           C  \\nATOM    417  CD1 ILE A  53      10.031   5.176  -8.021  1.00 22.86           C  \\nATOM    418  N   ILE A  54       6.226   0.544  -8.942  1.00 24.11           N  \\nATOM    419  CA  ILE A  54       4.776   0.549  -9.101  1.00 24.11           C  \\nATOM    420  C   ILE A  54       4.160  -0.561  -8.253  1.00 24.11           C  \\nATOM    421  O   ILE A  54       2.936  -0.694  -8.188  1.00 24.11           O  \\nATOM    422  CB  ILE A  54       4.370   0.380 -10.583  1.00 24.11           C  \\nATOM    423  CG1 ILE A  54       4.980   1.498 -11.435  1.00 24.11           C  \\nATOM    424  CG2 ILE A  54       2.845   0.353 -10.725  1.00 24.11           C  \\nATOM    425  CD1 ILE A  54       4.699   1.364 -12.925  1.00 24.11           C  \\nATOM    426  N   GLY A  55       4.935  -1.285  -7.507  1.00 25.57           N  \\nATOM    427  CA  GLY A  55       4.109  -2.193  -6.729  1.00 25.57           C  \\nATOM    428  C   GLY A  55       3.530  -1.553  -5.481  1.00 25.57           C  \\nATOM    429  O   GLY A  55       2.831  -2.210  -4.707  1.00 25.57           O  \\nATOM    430  N   ALA A  56       3.666  -0.269  -5.338  1.00 25.26           N  \\nATOM    431  CA  ALA A  56       3.179   0.402  -4.136  1.00 25.26           C  \\nATOM    432  C   ALA A  56       1.816   1.044  -4.379  1.00 25.26           C  \\nATOM    433  O   ALA A  56       1.599   1.681  -5.413  1.00 25.26           O  \\nATOM    434  CB  ALA A  56       4.183   1.454  -3.670  1.00 25.26           C  \\nATOM    435  N   GLY A  57       0.778   0.228  -4.369  1.00 27.90           N  \\nATOM    436  CA  GLY A  57      -0.533   0.851  -4.456  1.00 27.90           C  \\nATOM    437  C   GLY A  57      -0.731   1.969  -3.451  1.00 27.90           C  \\nATOM    438  O   GLY A  57      -0.176   1.928  -2.351  1.00 27.90           O  \\nATOM    439  N   ILE A  58      -0.900   3.122  -3.945  1.00 27.63           N  \\nATOM    440  CA  ILE A  58      -1.239   4.328  -3.198  1.00 27.63           C  \\nATOM    441  C   ILE A  58      -2.740   4.356  -2.917  1.00 27.63           C  \\nATOM    442  O   ILE A  58      -3.552   4.221  -3.835  1.00 27.63           O  \\nATOM    443  CB  ILE A  58      -0.812   5.603  -3.959  1.00 27.63           C  \\nATOM    444  CG1 ILE A  58       0.687   5.558  -4.276  1.00 27.63           C  \\nATOM    445  CG2 ILE A  58      -1.164   6.857  -3.152  1.00 27.63           C  \\nATOM    446  CD1 ILE A  58       1.163   6.690  -5.177  1.00 27.63           C  \\nATOM    447  N   ILE A  59      -3.117   4.183  -1.694  1.00 30.18           N  \\nATOM    448  CA  ILE A  59      -4.528   4.395  -1.389  1.00 30.18           C  \\nATOM    449  C   ILE A  59      -4.734   5.815  -0.866  1.00 30.18           C  \\nATOM    450  O   ILE A  59      -4.039   6.251   0.055  1.00 30.18           O  \\nATOM    451  CB  ILE A  59      -5.045   3.364  -0.362  1.00 30.18           C  \\nATOM    452  CG1 ILE A  59      -4.815   1.938  -0.874  1.00 30.18           C  \\nATOM    453  CG2 ILE A  59      -6.527   3.604  -0.056  1.00 30.18           C  \\nATOM    454  CD1 ILE A  59      -5.166   0.854   0.136  1.00 30.18           C  \\nATOM    455  N   PHE A  60      -5.431   6.566  -1.626  1.00 27.15           N  \\nATOM    456  CA  PHE A  60      -5.690   7.944  -1.226  1.00 27.15           C  \\nATOM    457  C   PHE A  60      -6.935   8.030  -0.352  1.00 27.15           C  \\nATOM    458  O   PHE A  60      -7.978   7.466  -0.690  1.00 27.15           O  \\nATOM    459  CB  PHE A  60      -5.850   8.842  -2.457  1.00 27.15           C  \\nATOM    460  CG  PHE A  60      -4.610   8.935  -3.306  1.00 27.15           C  \\nATOM    461  CD1 PHE A  60      -3.615   9.858  -3.008  1.00 27.15           C  \\nATOM    462  CD2 PHE A  60      -4.440   8.099  -4.401  1.00 27.15           C  \\nATOM    463  CE1 PHE A  60      -2.467   9.947  -3.792  1.00 27.15           C  \\nATOM    464  CE2 PHE A  60      -3.296   8.181  -5.189  1.00 27.15           C  \\nATOM    465  CZ  PHE A  60      -2.311   9.107  -4.883  1.00 27.15           C  \\nATOM    466  N   LEU A  61      -6.792   8.558   0.800  1.00 28.66           N  \\nATOM    467  CA  LEU A  61      -7.975   8.703   1.642  1.00 28.66           C  \\nATOM    468  C   LEU A  61      -8.442  10.154   1.679  1.00 28.66           C  \\nATOM    469  O   LEU A  61      -7.623  11.075   1.639  1.00 28.66           O  \\nATOM    470  CB  LEU A  61      -7.685   8.211   3.063  1.00 28.66           C  \\nATOM    471  CG  LEU A  61      -7.315   6.734   3.207  1.00 28.66           C  \\nATOM    472  CD1 LEU A  61      -6.608   6.492   4.536  1.00 28.66           C  \\nATOM    473  CD2 LEU A  61      -8.557   5.858   3.089  1.00 28.66           C  \\nATOM    474  N   ASP A  62      -9.616  10.389   1.187  1.00 28.79           N  \\nATOM    475  CA  ASP A  62     -10.344  11.652   1.256  1.00 28.79           C  \\nATOM    476  C   ASP A  62     -10.496  12.121   2.701  1.00 28.79           C  \\nATOM    477  O   ASP A  62     -10.486  11.307   3.628  1.00 28.79           O  \\nATOM    478  CB  ASP A  62     -11.720  11.515   0.601  1.00 28.79           C  \\nATOM    479  CG  ASP A  62     -12.425  12.847   0.417  1.00 28.79           C  \\nATOM    480  OD1 ASP A  62     -11.890  13.886   0.861  1.00 28.79           O  \\nATOM    481  OD2 ASP A  62     -13.526  12.858  -0.175  1.00 28.79           O  \\nATOM    482  N   GLU A  63      -9.959  13.193   3.025  1.00 28.13           N  \\nATOM    483  CA  GLU A  63     -10.331  13.866   4.265  1.00 28.13           C  \\nATOM    484  C   GLU A  63     -11.779  13.564   4.644  1.00 28.13           C  \\nATOM    485  O   GLU A  63     -12.118  13.513   5.828  1.00 28.13           O  \\nATOM    486  CB  GLU A  63     -10.125  15.378   4.140  1.00 28.13           C  \\nATOM    487  CG  GLU A  63      -8.961  15.910   4.964  1.00 28.13           C  \\nATOM    488  CD  GLU A  63      -8.883  17.428   4.981  1.00 28.13           C  \\nATOM    489  OE1 GLU A  63      -8.025  17.984   5.703  1.00 28.13           O  \\nATOM    490  OE2 GLU A  63      -9.688  18.067   4.267  1.00 28.13           O  \\nATOM    491  N   ALA A  64     -12.668  13.684   3.656  1.00 26.80           N  \\nATOM    492  CA  ALA A  64     -14.004  13.353   4.145  1.00 26.80           C  \\nATOM    493  C   ALA A  64     -13.989  12.055   4.946  1.00 26.80           C  \\nATOM    494  O   ALA A  64     -14.824  11.855   5.832  1.00 26.80           O  \\nATOM    495  CB  ALA A  64     -14.986  13.245   2.980  1.00 26.80           C  \\nATOM    496  N   SER A  65     -12.929  11.339   4.776  1.00 27.10           N  \\nATOM    497  CA  SER A  65     -12.904  10.114   5.568  1.00 27.10           C  \\nATOM    498  C   SER A  65     -12.344  10.370   6.964  1.00 27.10           C  \\nATOM    499  O   SER A  65     -12.158   9.434   7.745  1.00 27.10           O  \\nATOM    500  CB  SER A  65     -12.075   9.038   4.867  1.00 27.10           C  \\nATOM    501  OG  SER A  65     -10.863   9.580   4.370  1.00 27.10           O  \\nATOM    502  N   LYS A  66     -11.629  11.465   7.109  1.00 27.42           N  \\nATOM    503  CA  LYS A  66     -11.222  11.763   8.479  1.00 27.42           C  \\nATOM    504  C   LYS A  66     -12.287  11.319   9.477  1.00 27.42           C  \\nATOM    505  O   LYS A  66     -11.964  10.866  10.577  1.00 27.42           O  \\nATOM    506  CB  LYS A  66     -10.941  13.257   8.643  1.00 27.42           C  \\nATOM    507  CG  LYS A  66      -9.531  13.669   8.245  1.00 27.42           C  \\nATOM    508  CD  LYS A  66      -9.274  15.141   8.541  1.00 27.42           C  \\nATOM    509  CE  LYS A  66      -7.876  15.563   8.110  1.00 27.42           C  \\nATOM    510  NZ  LYS A  66      -7.628  17.012   8.375  1.00 27.42           N  \\nATOM    511  N   HIS A  67     -13.413  11.807   9.159  1.00 24.43           N  \\nATOM    512  CA  HIS A  67     -14.326  11.473  10.246  1.00 24.43           C  \\nATOM    513  C   HIS A  67     -14.614   9.976  10.283  1.00 24.43           C  \\nATOM    514  O   HIS A  67     -15.337   9.500  11.162  1.00 24.43           O  \\nATOM    515  CB  HIS A  67     -15.633  12.256  10.107  1.00 24.43           C  \\nATOM    516  CG  HIS A  67     -15.436  13.731   9.954  1.00 24.43           C  \\nATOM    517  ND1 HIS A  67     -14.918  14.520  10.958  1.00 24.43           N  \\nATOM    518  CD2 HIS A  67     -15.687  14.558   8.912  1.00 24.43           C  \\nATOM    519  CE1 HIS A  67     -14.858  15.773  10.538  1.00 24.43           C  \\nATOM    520  NE2 HIS A  67     -15.319  15.822   9.300  1.00 24.43           N  \\nATOM    521  N   MET A  68     -13.890   9.296   9.509  1.00 22.35           N  \\nATOM    522  CA  MET A  68     -14.343   7.911   9.600  1.00 22.35           C  \\nATOM    523  C   MET A  68     -13.209   6.996  10.049  1.00 22.35           C  \\nATOM    524  O   MET A  68     -13.047   5.895   9.520  1.00 22.35           O  \\nATOM    525  CB  MET A  68     -14.897   7.436   8.255  1.00 22.35           C  \\nATOM    526  CG  MET A  68     -16.278   7.984   7.934  1.00 22.35           C  \\nATOM    527  SD  MET A  68     -16.839   7.528   6.247  1.00 22.35           S  \\nATOM    528  CE  MET A  68     -18.202   6.402   6.657  1.00 22.35           C  \\nATOM    529  N   TRP A  69     -12.254   7.629  10.737  1.00 23.82           N  \\nATOM    530  CA  TRP A  69     -11.306   6.713  11.363  1.00 23.82           C  \\nATOM    531  C   TRP A  69     -12.016   5.777  12.335  1.00 23.82           C  \\nATOM    532  O   TRP A  69     -12.974   6.174  13.002  1.00 23.82           O  \\nATOM    533  CB  TRP A  69     -10.208   7.491  12.092  1.00 23.82           C  \\nATOM    534  CG  TRP A  69      -8.818   7.142  11.650  1.00 23.82           C  \\nATOM    535  CD1 TRP A  69      -8.201   5.927  11.756  1.00 23.82           C  \\nATOM    536  CD2 TRP A  69      -7.871   8.021  11.036  1.00 23.82           C  \\nATOM    537  NE1 TRP A  69      -6.926   5.997  11.243  1.00 23.82           N  \\nATOM    538  CE2 TRP A  69      -6.699   7.271  10.795  1.00 23.82           C  \\nATOM    539  CE3 TRP A  69      -7.901   9.372  10.664  1.00 23.82           C  \\nATOM    540  CZ2 TRP A  69      -5.565   7.828  10.199  1.00 23.82           C  \\nATOM    541  CZ3 TRP A  69      -6.771   9.924  10.071  1.00 23.82           C  \\nATOM    542  CH2 TRP A  69      -5.621   9.151   9.845  1.00 23.82           C  \\nATOM    543  N   PRO A  70     -11.435   4.711  12.760  1.00 24.68           N  \\nATOM    544  CA  PRO A  70     -11.103   3.418  12.157  1.00 24.68           C  \\nATOM    545  C   PRO A  70     -12.289   2.457  12.130  1.00 24.68           C  \\nATOM    546  O   PRO A  70     -12.106   1.249  11.953  1.00 24.68           O  \\nATOM    547  CB  PRO A  70      -9.988   2.884  13.059  1.00 24.68           C  \\nATOM    548  CG  PRO A  70     -10.227   3.531  14.385  1.00 24.68           C  \\nATOM    549  CD  PRO A  70     -11.484   4.348  14.297  1.00 24.68           C  \\nATOM    550  N   GLY A  71     -13.464   2.935  12.256  1.00 26.84           N  \\nATOM    551  CA  GLY A  71     -14.462   1.923  11.947  1.00 26.84           C  \\nATOM    552  C   GLY A  71     -14.660   1.715  10.458  1.00 26.84           C  \\nATOM    553  O   GLY A  71     -15.062   0.633  10.025  1.00 26.84           O  \\nATOM    554  N   ASP A  72     -14.191   2.627   9.693  1.00 30.30           N  \\nATOM    555  CA  ASP A  72     -14.499   2.522   8.270  1.00 30.30           C  \\nATOM    556  C   ASP A  72     -13.496   1.618   7.556  1.00 30.30           C  \\nATOM    557  O   ASP A  72     -13.811   1.032   6.518  1.00 30.30           O  \\nATOM    558  CB  ASP A  72     -14.513   3.908   7.620  1.00 30.30           C  \\nATOM    559  CG  ASP A  72     -15.903   4.351   7.199  1.00 30.30           C  \\nATOM    560  OD1 ASP A  72     -16.826   3.510   7.163  1.00 30.30           O  \\nATOM    561  OD2 ASP A  72     -16.075   5.552   6.898  1.00 30.30           O  \\nATOM    562  N   ILE A  73     -12.272   1.560   8.064  1.00 28.40           N  \\nATOM    563  CA  ILE A  73     -11.586   0.493   7.344  1.00 28.40           C  \\nATOM    564  C   ILE A  73     -12.305  -0.833   7.581  1.00 28.40           C  \\nATOM    565  O   ILE A  73     -12.470  -1.631   6.655  1.00 28.40           O  \\nATOM    566  CB  ILE A  73     -10.105   0.384   7.771  1.00 28.40           C  \\nATOM    567  CG1 ILE A  73      -9.326   1.624   7.318  1.00 28.40           C  \\nATOM    568  CG2 ILE A  73      -9.474  -0.894   7.211  1.00 28.40           C  \\nATOM    569  CD1 ILE A  73      -7.813   1.462   7.380  1.00 28.40           C  \\nATOM    570  N   LYS A  74     -12.518  -1.142   8.818  1.00 32.10           N  \\nATOM    571  CA  LYS A  74     -13.345  -2.337   8.961  1.00 32.10           C  \\nATOM    572  C   LYS A  74     -14.589  -2.253   8.081  1.00 32.10           C  \\nATOM    573  O   LYS A  74     -14.941  -3.220   7.402  1.00 32.10           O  \\nATOM    574  CB  LYS A  74     -13.750  -2.537  10.423  1.00 32.10           C  \\nATOM    575  CG  LYS A  74     -14.260  -3.936  10.738  1.00 32.10           C  \\nATOM    576  CD  LYS A  74     -14.560  -4.100  12.222  1.00 32.10           C  \\nATOM    577  CE  LYS A  74     -15.009  -5.518  12.549  1.00 32.10           C  \\nATOM    578  NZ  LYS A  74     -15.464  -5.642  13.966  1.00 32.10           N  \\nATOM    579  N   ALA A  75     -15.297  -1.175   8.192  1.00 34.65           N  \\nATOM    580  CA  ALA A  75     -16.462  -1.043   7.321  1.00 34.65           C  \\nATOM    581  C   ALA A  75     -16.061  -1.128   5.852  1.00 34.65           C  \\nATOM    582  O   ALA A  75     -16.712  -1.817   5.063  1.00 34.65           O  \\nATOM    583  CB  ALA A  75     -17.186   0.273   7.597  1.00 34.65           C  \\nATOM    584  N   ILE A  76     -15.079  -0.313   5.469  1.00 33.14           N  \\nATOM    585  CA  ILE A  76     -14.609  -0.460   4.096  1.00 33.14           C  \\nATOM    586  C   ILE A  76     -14.183  -1.906   3.849  1.00 33.14           C  \\nATOM    587  O   ILE A  76     -14.581  -2.515   2.852  1.00 33.14           O  \\nATOM    588  CB  ILE A  76     -13.440   0.503   3.793  1.00 33.14           C  \\nATOM    589  CG1 ILE A  76     -13.929   1.955   3.797  1.00 33.14           C  \\nATOM    590  CG2 ILE A  76     -12.783   0.150   2.455  1.00 33.14           C  \\nATOM    591  CD1 ILE A  76     -12.814   2.985   3.674  1.00 33.14           C  \\nATOM    592  N   LEU A  77     -13.415  -2.342   4.774  1.00 32.98           N  \\nATOM    593  CA  LEU A  77     -13.022  -3.733   4.582  1.00 32.98           C  \\nATOM    594  C   LEU A  77     -14.227  -4.660   4.701  1.00 32.98           C  \\nATOM    595  O   LEU A  77     -14.347  -5.628   3.947  1.00 32.98           O  \\nATOM    596  CB  LEU A  77     -11.953  -4.133   5.603  1.00 32.98           C  \\nATOM    597  CG  LEU A  77     -10.540  -3.605   5.347  1.00 32.98           C  \\nATOM    598  CD1 LEU A  77      -9.665  -3.816   6.578  1.00 32.98           C  \\nATOM    599  CD2 LEU A  77      -9.928  -4.282   4.126  1.00 32.98           C  \\nATOM    600  N   ASP A  78     -15.000  -4.403   5.733  1.00 36.22           N  \\nATOM    601  CA  ASP A  78     -16.248  -5.158   5.801  1.00 36.22           C  \\nATOM    602  C   ASP A  78     -17.068  -4.979   4.526  1.00 36.22           C  \\nATOM    603  O   ASP A  78     -17.628  -5.944   4.002  1.00 36.22           O  \\nATOM    604  CB  ASP A  78     -17.069  -4.729   7.019  1.00 36.22           C  \\nATOM    605  CG  ASP A  78     -16.682  -5.470   8.287  1.00 36.22           C  \\nATOM    606  OD1 ASP A  78     -15.903  -6.445   8.210  1.00 36.22           O  \\nATOM    607  OD2 ASP A  78     -17.163  -5.079   9.372  1.00 36.22           O  \\nATOM    608  N   LYS A  79     -17.266  -3.711   4.228  1.00 34.76           N  \\nATOM    609  CA  LYS A  79     -17.937  -3.521   2.945  1.00 34.76           C  \\nATOM    610  C   LYS A  79     -17.203  -4.256   1.827  1.00 34.76           C  \\nATOM    611  O   LYS A  79     -17.834  -4.854   0.952  1.00 34.76           O  \\nATOM    612  CB  LYS A  79     -18.045  -2.032   2.610  1.00 34.76           C  \\nATOM    613  CG  LYS A  79     -19.429  -1.445   2.842  1.00 34.76           C  \\nATOM    614  CD  LYS A  79     -19.444   0.060   2.605  1.00 34.76           C  \\nATOM    615  CE  LYS A  79     -20.794   0.669   2.960  1.00 34.76           C  \\nATOM    616  NZ  LYS A  79     -20.797   2.152   2.781  1.00 34.76           N  \\nATOM    617  N   LEU A  80     -15.910  -4.128   1.923  1.00 32.78           N  \\nATOM    618  CA  LEU A  80     -15.074  -4.881   0.995  1.00 32.78           C  \\nATOM    619  C   LEU A  80     -15.320  -6.380   1.134  1.00 32.78           C  \\nATOM    620  O   LEU A  80     -15.440  -7.089   0.132  1.00 32.78           O  \\nATOM    621  CB  LEU A  80     -13.594  -4.572   1.234  1.00 32.78           C  \\nATOM    622  CG  LEU A  80     -13.080  -3.243   0.677  1.00 32.78           C  \\nATOM    623  CD1 LEU A  80     -11.763  -2.862   1.344  1.00 32.78           C  \\nATOM    624  CD2 LEU A  80     -12.914  -3.327  -0.837  1.00 32.78           C  \\nATOM    625  N   HIS A  81     -15.318  -6.775   2.396  1.00 33.28           N  \\nATOM    626  CA  HIS A  81     -15.602  -8.172   2.704  1.00 33.28           C  \\nATOM    627  C   HIS A  81     -17.042  -8.531   2.354  1.00 33.28           C  \\nATOM    628  O   HIS A  81     -17.315  -9.641   1.890  1.00 33.28           O  \\nATOM    629  CB  HIS A  81     -15.334  -8.461   4.182  1.00 33.28           C  \\nATOM    630  CG  HIS A  81     -14.655  -9.771   4.424  1.00 33.28           C  \\nATOM    631  ND1 HIS A  81     -15.345 -10.960   4.518  1.00 33.28           N  \\nATOM    632  CD2 HIS A  81     -13.346 -10.078   4.588  1.00 33.28           C  \\nATOM    633  CE1 HIS A  81     -14.487 -11.944   4.731  1.00 33.28           C  \\nATOM    634  NE2 HIS A  81     -13.268 -11.436   4.777  1.00 33.28           N  \\nATOM    635  N   ASN A  82     -17.946  -7.765   2.930  1.00 34.62           N  \\nATOM    636  CA  ASN A  82     -19.343  -8.104   2.683  1.00 34.62           C  \\nATOM    637  C   ASN A  82     -19.685  -8.027   1.198  1.00 34.62           C  \\nATOM    638  O   ASN A  82     -20.668  -8.621   0.751  1.00 34.62           O  \\nATOM    639  CB  ASN A  82     -20.268  -7.188   3.489  1.00 34.62           C  \\nATOM    640  CG  ASN A  82     -20.351  -7.582   4.950  1.00 34.62           C  \\nATOM    641  OD1 ASN A  82     -20.024  -8.713   5.320  1.00 34.62           O  \\nATOM    642  ND2 ASN A  82     -20.789  -6.652   5.791  1.00 34.62           N  \\nATOM    643  N   LEU A  83     -18.929  -7.250   0.539  1.00 28.86           N  \\nATOM    644  CA  LEU A  83     -19.284  -7.177  -0.874  1.00 28.86           C  \\nATOM    645  C   LEU A  83     -18.758  -8.393  -1.630  1.00 28.86           C  \\nATOM    646  O   LEU A  83     -17.583  -8.439  -1.999  1.00 28.86           O  \\nATOM    647  CB  LEU A  83     -18.731  -5.894  -1.500  1.00 28.86           C  \\nATOM    648  CG  LEU A  83     -19.530  -4.615  -1.245  1.00 28.86           C  \\nATOM    649  CD1 LEU A  83     -18.606  -3.402  -1.247  1.00 28.86           C  \\nATOM    650  CD2 LEU A  83     -20.632  -4.456  -2.287  1.00 28.86           C  \\nATOM    651  N   ASN A  84     -18.816  -9.487  -0.983  1.00 29.74           N  \\nATOM    652  CA  ASN A  84     -18.746 -10.752  -1.707  1.00 29.74           C  \\nATOM    653  C   ASN A  84     -19.534 -10.695  -3.012  1.00 29.74           C  \\nATOM    654  O   ASN A  84     -19.795 -11.729  -3.630  1.00 29.74           O  \\nATOM    655  CB  ASN A  84     -19.251 -11.900  -0.831  1.00 29.74           C  \\nATOM    656  CG  ASN A  84     -18.127 -12.639  -0.131  1.00 29.74           C  \\nATOM    657  OD1 ASN A  84     -16.948 -12.334  -0.332  1.00 29.74           O  \\nATOM    658  ND2 ASN A  84     -18.483 -13.614   0.696  1.00 29.74           N  \\nATOM    659  N   SER A  85     -20.065  -9.548  -3.356  1.00 26.98           N  \\nATOM    660  CA  SER A  85     -20.500  -9.557  -4.748  1.00 26.98           C  \\nATOM    661  C   SER A  85     -19.336  -9.274  -5.693  1.00 26.98           C  \\nATOM    662  O   SER A  85     -18.702  -8.220  -5.606  1.00 26.98           O  \\nATOM    663  CB  SER A  85     -21.609  -8.528  -4.970  1.00 26.98           C  \\nATOM    664  OG  SER A  85     -22.234  -8.726  -6.227  1.00 26.98           O  \\nATOM    665  N   THR A  86     -18.290 -10.028  -5.606  1.00 28.60           N  \\nATOM    666  CA  THR A  86     -17.216 -10.093  -6.590  1.00 28.60           C  \\nATOM    667  C   THR A  86     -17.409  -9.035  -7.673  1.00 28.60           C  \\nATOM    668  O   THR A  86     -16.447  -8.396  -8.103  1.00 28.60           O  \\nATOM    669  CB  THR A  86     -17.137 -11.487  -7.239  1.00 28.60           C  \\nATOM    670  OG1 THR A  86     -18.416 -12.127  -7.141  1.00 28.60           O  \\nATOM    671  CG2 THR A  86     -16.092 -12.357  -6.549  1.00 28.60           C  \\nATOM    672  N   THR A  87     -18.698  -8.668  -7.947  1.00 27.56           N  \\nATOM    673  CA  THR A  87     -18.824  -7.790  -9.105  1.00 27.56           C  \\nATOM    674  C   THR A  87     -18.652  -6.329  -8.698  1.00 27.56           C  \\nATOM    675  O   THR A  87     -17.992  -5.558  -9.397  1.00 27.56           O  \\nATOM    676  CB  THR A  87     -20.186  -7.974  -9.800  1.00 27.56           C  \\nATOM    677  OG1 THR A  87     -20.488  -9.373  -9.882  1.00 27.56           O  \\nATOM    678  CG2 THR A  87     -20.169  -7.387 -11.208  1.00 27.56           C  \\nATOM    679  N   LEU A  88     -19.110  -6.028  -7.516  1.00 25.56           N  \\nATOM    680  CA  LEU A  88     -19.039  -4.586  -7.310  1.00 25.56           C  \\nATOM    681  C   LEU A  88     -17.646  -4.170  -6.850  1.00 25.56           C  \\nATOM    682  O   LEU A  88     -17.174  -3.083  -7.190  1.00 25.56           O  \\nATOM    683  CB  LEU A  88     -20.083  -4.139  -6.283  1.00 25.56           C  \\nATOM    684  CG  LEU A  88     -21.102  -3.102  -6.759  1.00 25.56           C  \\nATOM    685  CD1 LEU A  88     -22.451  -3.766  -7.016  1.00 25.56           C  \\nATOM    686  CD2 LEU A  88     -21.239  -1.978  -5.737  1.00 25.56           C  \\nATOM    687  N   ILE A  89     -17.014  -5.046  -6.141  1.00 26.85           N  \\nATOM    688  CA  ILE A  89     -15.613  -4.765  -5.846  1.00 26.85           C  \\nATOM    689  C   ILE A  89     -14.855  -4.506  -7.146  1.00 26.85           C  \\nATOM    690  O   ILE A  89     -14.019  -3.602  -7.214  1.00 26.85           O  \\nATOM    691  CB  ILE A  89     -14.957  -5.925  -5.065  1.00 26.85           C  \\nATOM    692  CG1 ILE A  89     -15.548  -6.020  -3.653  1.00 26.85           C  \\nATOM    693  CG2 ILE A  89     -13.436  -5.748  -5.011  1.00 26.85           C  \\nATOM    694  CD1 ILE A  89     -15.074  -7.233  -2.864  1.00 26.85           C  \\nATOM    695  N   LYS A  90     -15.163  -5.349  -8.108  1.00 27.72           N  \\nATOM    696  CA  LYS A  90     -14.685  -5.275  -9.485  1.00 27.72           C  \\nATOM    697  C   LYS A  90     -14.716  -3.839 -10.002  1.00 27.72           C  \\nATOM    698  O   LYS A  90     -13.717  -3.339 -10.521  1.00 27.72           O  \\nATOM    699  CB  LYS A  90     -15.521  -6.177 -10.394  1.00 27.72           C  \\nATOM    700  CG  LYS A  90     -14.965  -7.585 -10.549  1.00 27.72           C  \\nATOM    701  CD  LYS A  90     -15.813  -8.419 -11.501  1.00 27.72           C  \\nATOM    702  CE  LYS A  90     -15.346  -9.867 -11.545  1.00 27.72           C  \\nATOM    703  NZ  LYS A  90     -16.191 -10.695 -12.456  1.00 27.72           N  \\nATOM    704  N   GLU A  91     -15.856  -3.236  -9.700  1.00 27.23           N  \\nATOM    705  CA  GLU A  91     -16.189  -1.999 -10.401  1.00 27.23           C  \\nATOM    706  C   GLU A  91     -15.733  -0.776  -9.610  1.00 27.23           C  \\nATOM    707  O   GLU A  91     -15.195   0.174 -10.182  1.00 27.23           O  \\nATOM    708  CB  GLU A  91     -17.695  -1.922 -10.668  1.00 27.23           C  \\nATOM    709  CG  GLU A  91     -18.126  -2.601 -11.959  1.00 27.23           C  \\nATOM    710  CD  GLU A  91     -19.621  -2.505 -12.218  1.00 27.23           C  \\nATOM    711  OE1 GLU A  91     -20.080  -2.940 -13.298  1.00 27.23           O  \\nATOM    712  OE2 GLU A  91     -20.340  -1.989 -11.333  1.00 27.23           O  \\nATOM    713  N   LYS A  92     -15.817  -0.923  -8.281  1.00 25.89           N  \\nATOM    714  CA  LYS A  92     -15.603   0.369  -7.634  1.00 25.89           C  \\nATOM    715  C   LYS A  92     -14.134   0.565  -7.271  1.00 25.89           C  \\nATOM    716  O   LYS A  92     -13.625   1.687  -7.308  1.00 25.89           O  \\nATOM    717  CB  LYS A  92     -16.474   0.493  -6.383  1.00 25.89           C  \\nATOM    718  CG  LYS A  92     -17.771   1.257  -6.605  1.00 25.89           C  \\nATOM    719  CD  LYS A  92     -18.499   1.515  -5.292  1.00 25.89           C  \\nATOM    720  CE  LYS A  92     -19.777   2.314  -5.508  1.00 25.89           C  \\nATOM    721  NZ  LYS A  92     -20.495   2.569  -4.223  1.00 25.89           N  \\nATOM    722  N   CYS A  93     -13.493  -0.553  -6.875  1.00 27.40           N  \\nATOM    723  CA  CYS A  93     -12.079  -0.337  -6.588  1.00 27.40           C  \\nATOM    724  C   CYS A  93     -11.314   0.025  -7.855  1.00 27.40           C  \\nATOM    725  O   CYS A  93     -10.408   0.859  -7.822  1.00 27.40           O  \\nATOM    726  CB  CYS A  93     -11.465  -1.582  -5.949  1.00 27.40           C  \\nATOM    727  SG  CYS A  93     -11.481  -1.557  -4.143  1.00 27.40           S  \\nATOM    728  N   ILE A  94     -11.724  -0.569  -8.864  1.00 24.70           N  \\nATOM    729  CA  ILE A  94     -11.052  -0.324 -10.135  1.00 24.70           C  \\nATOM    730  C   ILE A  94     -11.310   1.112 -10.586  1.00 24.70           C  \\nATOM    731  O   ILE A  94     -10.413   1.774 -11.114  1.00 24.70           O  \\nATOM    732  CB  ILE A  94     -11.517  -1.319 -11.222  1.00 24.70           C  \\nATOM    733  CG1 ILE A  94     -11.127  -2.751 -10.837  1.00 24.70           C  \\nATOM    734  CG2 ILE A  94     -10.934  -0.941 -12.586  1.00 24.70           C  \\nATOM    735  CD1 ILE A  94     -11.692  -3.817 -11.765  1.00 24.70           C  \\nATOM    736  N   GLU A  95     -12.540   1.565 -10.356  1.00 23.35           N  \\nATOM    737  CA  GLU A  95     -12.775   2.916 -10.856  1.00 23.35           C  \\nATOM    738  C   GLU A  95     -11.888   3.932 -10.141  1.00 23.35           C  \\nATOM    739  O   GLU A  95     -11.378   4.866 -10.763  1.00 23.35           O  \\nATOM    740  CB  GLU A  95     -14.248   3.301 -10.697  1.00 23.35           C  \\nATOM    741  CG  GLU A  95     -14.991   3.451 -12.016  1.00 23.35           C  \\nATOM    742  CD  GLU A  95     -16.446   3.856 -11.843  1.00 23.35           C  \\nATOM    743  OE1 GLU A  95     -17.149   4.045 -12.862  1.00 23.35           O  \\nATOM    744  OE2 GLU A  95     -16.886   3.987 -10.679  1.00 23.35           O  \\nATOM    745  N   VAL A  96     -11.762   3.683  -8.856  1.00 24.17           N  \\nATOM    746  CA  VAL A  96     -11.095   4.784  -8.170  1.00 24.17           C  \\nATOM    747  C   VAL A  96      -9.613   4.801  -8.537  1.00 24.17           C  \\nATOM    748  O   VAL A  96      -9.034   5.868  -8.760  1.00 24.17           O  \\nATOM    749  CB  VAL A  96     -11.262   4.682  -6.637  1.00 24.17           C  \\nATOM    750  CG1 VAL A  96     -10.565   5.848  -5.938  1.00 24.17           C  \\nATOM    751  CG2 VAL A  96     -12.743   4.640  -6.264  1.00 24.17           C  \\nATOM    752  N   CYS A  97      -9.177   3.717  -9.021  1.00 26.35           N  \\nATOM    753  CA  CYS A  97      -7.727   3.796  -9.161  1.00 26.35           C  \\nATOM    754  C   CYS A  97      -7.330   4.010 -10.617  1.00 26.35           C  \\nATOM    755  O   CYS A  97      -6.314   4.647 -10.900  1.00 26.35           O  \\nATOM    756  CB  CYS A  97      -7.066   2.526  -8.624  1.00 26.35           C  \\nATOM    757  SG  CYS A  97      -6.827   2.532  -6.834  1.00 26.35           S  \\nATOM    758  N   LEU A  98      -8.206   3.682 -11.467  1.00 24.29           N  \\nATOM    759  CA  LEU A  98      -7.744   3.699 -12.851  1.00 24.29           C  \\nATOM    760  C   LEU A  98      -7.745   5.119 -13.407  1.00 24.29           C  \\nATOM    761  O   LEU A  98      -6.915   5.460 -14.253  1.00 24.29           O  \\nATOM    762  CB  LEU A  98      -8.623   2.797 -13.721  1.00 24.29           C  \\nATOM    763  CG  LEU A  98      -8.376   1.292 -13.604  1.00 24.29           C  \\nATOM    764  CD1 LEU A  98      -9.614   0.516 -14.044  1.00 24.29           C  \\nATOM    765  CD2 LEU A  98      -7.161   0.884 -14.430  1.00 24.29           C  \\nATOM    766  N   GLU A  99      -8.744   5.873 -12.999  1.00 25.16           N  \\nATOM    767  CA  GLU A  99      -8.764   7.128 -13.745  1.00 25.16           C  \\nATOM    768  C   GLU A  99      -7.526   7.968 -13.445  1.00 25.16           C  \\nATOM    769  O   GLU A  99      -7.053   8.715 -14.304  1.00 25.16           O  \\nATOM    770  CB  GLU A  99     -10.031   7.924 -13.424  1.00 25.16           C  \\nATOM    771  CG  GLU A  99     -11.274   7.420 -14.144  1.00 25.16           C  \\nATOM    772  CD  GLU A  99     -12.480   8.329 -13.969  1.00 25.16           C  \\nATOM    773  OE1 GLU A  99     -13.532   8.069 -14.595  1.00 25.16           O  \\nATOM    774  OE2 GLU A  99     -12.371   9.310 -13.199  1.00 25.16           O  \\nATOM    775  N   ASP A 100      -6.870   7.665 -12.366  1.00 26.54           N  \\nATOM    776  CA  ASP A 100      -5.791   8.628 -12.172  1.00 26.54           C  \\nATOM    777  C   ASP A 100      -4.427   7.974 -12.382  1.00 26.54           C  \\nATOM    778  O   ASP A 100      -3.394   8.563 -12.058  1.00 26.54           O  \\nATOM    779  CB  ASP A 100      -5.868   9.246 -10.774  1.00 26.54           C  \\nATOM    780  CG  ASP A 100      -6.703  10.514 -10.731  1.00 26.54           C  \\nATOM    781  OD1 ASP A 100      -6.921  11.136 -11.793  1.00 26.54           O  \\nATOM    782  OD2 ASP A 100      -7.143  10.895  -9.625  1.00 26.54           O  \\nATOM    783  N   ILE A 101      -4.476   6.774 -12.769  1.00 24.34           N  \\nATOM    784  CA  ILE A 101      -3.147   6.239 -13.044  1.00 24.34           C  \\nATOM    785  C   ILE A 101      -2.693   6.676 -14.435  1.00 24.34           C  \\nATOM    786  O   ILE A 101      -3.385   6.433 -15.426  1.00 24.34           O  \\nATOM    787  CB  ILE A 101      -3.125   4.698 -12.932  1.00 24.34           C  \\nATOM    788  CG1 ILE A 101      -3.615   4.256 -11.549  1.00 24.34           C  \\nATOM    789  CG2 ILE A 101      -1.720   4.156 -13.214  1.00 24.34           C  \\nATOM    790  CD1 ILE A 101      -3.604   2.748 -11.341  1.00 24.34           C  \\nATOM    791  N   PRO A 102      -2.075   7.867 -14.523  1.00 24.89           N  \\nATOM    792  CA  PRO A 102      -1.660   8.281 -15.866  1.00 24.89           C  \\nATOM    793  C   PRO A 102      -1.451   7.098 -16.809  1.00 24.89           C  \\nATOM    794  O   PRO A 102      -1.213   5.976 -16.354  1.00 24.89           O  \\nATOM    795  CB  PRO A 102      -0.343   9.020 -15.616  1.00 24.89           C  \\nATOM    796  CG  PRO A 102      -0.121   8.916 -14.141  1.00 24.89           C  \\nATOM    797  CD  PRO A 102      -1.261   8.146 -13.540  1.00 24.89           C  \\nATOM    798  N   ALA A 103      -2.269   6.962 -17.739  1.00 24.93           N  \\nATOM    799  CA  ALA A 103      -2.271   6.244 -19.011  1.00 24.93           C  \\nATOM    800  C   ALA A 103      -1.049   5.338 -19.130  1.00 24.93           C  \\nATOM    801  O   ALA A 103      -1.043   4.395 -19.926  1.00 24.93           O  \\nATOM    802  CB  ALA A 103      -2.315   7.228 -20.177  1.00 24.93           C  \\nATOM    803  N   TYR A 104      -0.096   5.493 -18.214  1.00 22.82           N  \\nATOM    804  CA  TYR A 104       1.080   4.666 -18.459  1.00 22.82           C  \\nATOM    805  C   TYR A 104       0.882   3.259 -17.907  1.00 22.82           C  \\nATOM    806  O   TYR A 104       1.597   2.330 -18.289  1.00 22.82           O  \\nATOM    807  CB  TYR A 104       2.325   5.302 -17.832  1.00 22.82           C  \\nATOM    808  CG  TYR A 104       3.152   6.105 -18.806  1.00 22.82           C  \\nATOM    809  CD1 TYR A 104       4.157   5.501 -19.559  1.00 22.82           C  \\nATOM    810  CD2 TYR A 104       2.932   7.467 -18.975  1.00 22.82           C  \\nATOM    811  CE1 TYR A 104       4.924   6.238 -20.456  1.00 22.82           C  \\nATOM    812  CE2 TYR A 104       3.693   8.213 -19.869  1.00 22.82           C  \\nATOM    813  CZ  TYR A 104       4.684   7.591 -20.605  1.00 22.82           C  \\nATOM    814  OH  TYR A 104       5.440   8.324 -21.492  1.00 22.82           O  \\nATOM    815  N   GLY A 105      -0.374   2.860 -17.580  1.00 28.74           N  \\nATOM    816  CA  GLY A 105      -0.552   1.696 -16.728  1.00 28.74           C  \\nATOM    817  C   GLY A 105      -1.470   0.649 -17.331  1.00 28.74           C  \\nATOM    818  O   GLY A 105      -1.915  -0.267 -16.638  1.00 28.74           O  \\nATOM    819  N   ARG A 106      -1.855   0.828 -18.582  1.00 28.92           N  \\nATOM    820  CA  ARG A 106      -2.588  -0.329 -19.085  1.00 28.92           C  \\nATOM    821  C   ARG A 106      -1.736  -1.592 -19.009  1.00 28.92           C  \\nATOM    822  O   ARG A 106      -2.227  -2.655 -18.623  1.00 28.92           O  \\nATOM    823  CB  ARG A 106      -3.045  -0.091 -20.526  1.00 28.92           C  \\nATOM    824  CG  ARG A 106      -4.534   0.182 -20.662  1.00 28.92           C  \\nATOM    825  CD  ARG A 106      -5.023  -0.058 -22.084  1.00 28.92           C  \\nATOM    826  NE  ARG A 106      -5.450   1.184 -22.724  1.00 28.92           N  \\nATOM    827  CZ  ARG A 106      -6.413   1.274 -23.637  1.00 28.92           C  \\nATOM    828  NH1 ARG A 106      -7.071   0.192 -24.038  1.00 28.92           N  \\nATOM    829  NH2 ARG A 106      -6.720   2.454 -24.154  1.00 28.92           N  \\nATOM    830  N   ASP A 107      -0.458  -1.355 -19.241  1.00 32.22           N  \\nATOM    831  CA  ASP A 107       0.356  -2.564 -19.334  1.00 32.22           C  \\nATOM    832  C   ASP A 107       0.992  -2.902 -17.988  1.00 32.22           C  \\nATOM    833  O   ASP A 107       1.500  -4.009 -17.797  1.00 32.22           O  \\nATOM    834  CB  ASP A 107       1.439  -2.400 -20.402  1.00 32.22           C  \\nATOM    835  CG  ASP A 107       0.883  -2.396 -21.815  1.00 32.22           C  \\nATOM    836  OD1 ASP A 107      -0.208  -2.962 -22.043  1.00 32.22           O  \\nATOM    837  OD2 ASP A 107       1.543  -1.824 -22.710  1.00 32.22           O  \\nATOM    838  N   SER A 108       1.069  -1.902 -17.152  1.00 31.34           N  \\nATOM    839  CA  SER A 108       1.707  -2.123 -15.858  1.00 31.34           C  \\nATOM    840  C   SER A 108       0.921  -3.124 -15.018  1.00 31.34           C  \\nATOM    841  O   SER A 108       1.502  -3.871 -14.228  1.00 31.34           O  \\nATOM    842  CB  SER A 108       1.848  -0.805 -15.097  1.00 31.34           C  \\nATOM    843  OG  SER A 108       2.703   0.088 -15.790  1.00 31.34           O  \\nATOM    844  N   TRP A 109      -0.314  -3.225 -15.293  1.00 31.60           N  \\nATOM    845  CA  TRP A 109      -1.001  -4.276 -14.548  1.00 31.60           C  \\nATOM    846  C   TRP A 109      -0.397  -5.642 -14.853  1.00 31.60           C  \\nATOM    847  O   TRP A 109      -0.276  -6.487 -13.963  1.00 31.60           O  \\nATOM    848  CB  TRP A 109      -2.496  -4.279 -14.878  1.00 31.60           C  \\nATOM    849  CG  TRP A 109      -3.329  -3.467 -13.933  1.00 31.60           C  \\nATOM    850  CD1 TRP A 109      -3.742  -3.826 -12.680  1.00 31.60           C  \\nATOM    851  CD2 TRP A 109      -3.856  -2.157 -14.168  1.00 31.60           C  \\nATOM    852  NE1 TRP A 109      -4.494  -2.819 -12.121  1.00 31.60           N  \\nATOM    853  CE2 TRP A 109      -4.579  -1.784 -13.013  1.00 31.60           C  \\nATOM    854  CE3 TRP A 109      -3.787  -1.262 -15.244  1.00 31.60           C  \\nATOM    855  CZ2 TRP A 109      -5.228  -0.552 -12.905  1.00 31.60           C  \\nATOM    856  CZ3 TRP A 109      -4.434  -0.037 -15.134  1.00 31.60           C  \\nATOM    857  CH2 TRP A 109      -5.145   0.305 -13.972  1.00 31.60           C  \\nATOM    858  N   LYS A 110      -0.436  -5.933 -16.229  1.00 34.07           N  \\nATOM    859  CA  LYS A 110       0.140  -7.210 -16.640  1.00 34.07           C  \\nATOM    860  C   LYS A 110       1.427  -7.504 -15.874  1.00 34.07           C  \\nATOM    861  O   LYS A 110       1.665  -8.641 -15.464  1.00 34.07           O  \\nATOM    862  CB  LYS A 110       0.412  -7.216 -18.145  1.00 34.07           C  \\nATOM    863  CG  LYS A 110      -0.286  -8.341 -18.895  1.00 34.07           C  \\nATOM    864  CD  LYS A 110      -0.057  -8.236 -20.398  1.00 34.07           C  \\nATOM    865  CE  LYS A 110      -0.780  -9.344 -21.152  1.00 34.07           C  \\nATOM    866  NZ  LYS A 110      -0.656  -9.179 -22.632  1.00 34.07           N  \\nATOM    867  N   LEU A 111       2.050  -6.401 -15.560  1.00 31.56           N  \\nATOM    868  CA  LEU A 111       3.321  -6.564 -14.862  1.00 31.56           C  \\nATOM    869  C   LEU A 111       3.095  -6.835 -13.378  1.00 31.56           C  \\nATOM    870  O   LEU A 111       3.824  -7.622 -12.769  1.00 31.56           O  \\nATOM    871  CB  LEU A 111       4.194  -5.319 -15.039  1.00 31.56           C  \\nATOM    872  CG  LEU A 111       5.028  -5.252 -16.319  1.00 31.56           C  \\nATOM    873  CD1 LEU A 111       5.317  -3.800 -16.687  1.00 31.56           C  \\nATOM    874  CD2 LEU A 111       6.326  -6.035 -16.153  1.00 31.56           C  \\nATOM    875  N   LEU A 112       2.200  -6.016 -12.854  1.00 31.16           N  \\nATOM    876  CA  LEU A 112       1.821  -6.328 -11.481  1.00 31.16           C  \\nATOM    877  C   LEU A 112       1.500  -7.811 -11.329  1.00 31.16           C  \\nATOM    878  O   LEU A 112       1.898  -8.440 -10.346  1.00 31.16           O  \\nATOM    879  CB  LEU A 112       0.617  -5.487 -11.052  1.00 31.16           C  \\nATOM    880  CG  LEU A 112       0.908  -4.316 -10.111  1.00 31.16           C  \\nATOM    881  CD1 LEU A 112       0.053  -3.110 -10.486  1.00 31.16           C  \\nATOM    882  CD2 LEU A 112       0.663  -4.722  -8.662  1.00 31.16           C  \\nATOM    883  N   LEU A 113       0.694  -8.184 -12.265  1.00 31.10           N  \\nATOM    884  CA  LEU A 113       0.323  -9.594 -12.260  1.00 31.10           C  \\nATOM    885  C   LEU A 113       1.559 -10.482 -12.360  1.00 31.10           C  \\nATOM    886  O   LEU A 113       1.656 -11.499 -11.669  1.00 31.10           O  \\nATOM    887  CB  LEU A 113      -0.635  -9.901 -13.415  1.00 31.10           C  \\nATOM    888  CG  LEU A 113      -2.099  -9.513 -13.203  1.00 31.10           C  \\nATOM    889  CD1 LEU A 113      -2.799  -9.330 -14.545  1.00 31.10           C  \\nATOM    890  CD2 LEU A 113      -2.814 -10.565 -12.361  1.00 31.10           C  \\nATOM    891  N   GLU A 114       2.405 -10.033 -13.233  1.00 34.49           N  \\nATOM    892  CA  GLU A 114       3.623 -10.802 -13.469  1.00 34.49           C  \\nATOM    893  C   GLU A 114       4.569 -10.717 -12.275  1.00 34.49           C  \\nATOM    894  O   GLU A 114       5.144 -11.725 -11.859  1.00 34.49           O  \\nATOM    895  CB  GLU A 114       4.330 -10.313 -14.736  1.00 34.49           C  \\nATOM    896  CG  GLU A 114       3.777 -10.915 -16.020  1.00 34.49           C  \\nATOM    897  CD  GLU A 114       4.522 -10.460 -17.265  1.00 34.49           C  \\nATOM    898  OE1 GLU A 114       4.181 -10.921 -18.378  1.00 34.49           O  \\nATOM    899  OE2 GLU A 114       5.454  -9.637 -17.126  1.00 34.49           O  \\nATOM    900  N   ASP A 115       4.610  -9.539 -11.731  1.00 33.62           N  \\nATOM    901  CA  ASP A 115       5.580  -9.339 -10.659  1.00 33.62           C  \\nATOM    902  C   ASP A 115       5.029  -9.830  -9.322  1.00 33.62           C  \\nATOM    903  O   ASP A 115       5.791 -10.236  -8.442  1.00 33.62           O  \\nATOM    904  CB  ASP A 115       5.971  -7.864 -10.557  1.00 33.62           C  \\nATOM    905  CG  ASP A 115       7.340  -7.571 -11.147  1.00 33.62           C  \\nATOM    906  OD1 ASP A 115       7.989  -8.501 -11.672  1.00 33.62           O  \\nATOM    907  OD2 ASP A 115       7.774  -6.400 -11.083  1.00 33.62           O  \\nATOM    908  N   GLY A 116       3.802  -9.584  -9.134  1.00 33.02           N  \\nATOM    909  CA  GLY A 116       3.309 -10.001  -7.830  1.00 33.02           C  \\nATOM    910  C   GLY A 116       3.146 -11.504  -7.707  1.00 33.02           C  \\nATOM    911  O   GLY A 116       2.819 -12.012  -6.632  1.00 33.02           O  \\nATOM    912  N   GLY A 117       3.798 -12.256  -8.608  1.00 33.20           N  \\nATOM    913  CA  GLY A 117       3.718 -13.695  -8.417  1.00 33.20           C  \\nATOM    914  C   GLY A 117       2.327 -14.252  -8.657  1.00 33.20           C  \\nATOM    915  O   GLY A 117       1.928 -15.233  -8.026  1.00 33.20           O  \\nATOM    916  N   CYS A 118       1.515 -13.377  -9.122  1.00 29.76           N  \\nATOM    917  CA  CYS A 118       0.220 -13.948  -9.474  1.00 29.76           C  \\nATOM    918  C   CYS A 118       0.376 -15.067 -10.497  1.00 29.76           C  \\nATOM    919  O   CYS A 118       1.168 -14.950 -11.434  1.00 29.76           O  \\nATOM    920  CB  CYS A 118      -0.711 -12.867 -10.025  1.00 29.76           C  \\nATOM    921  SG  CYS A 118      -2.409 -13.428 -10.276  1.00 29.76           S  \\nATOM    922  N   PRO A 119       0.147 -16.280 -10.015  1.00 31.04           N  \\nATOM    923  CA  PRO A 119       0.256 -17.295 -11.065  1.00 31.04           C  \\nATOM    924  C   PRO A 119      -0.242 -16.799 -12.420  1.00 31.04           C  \\nATOM    925  O   PRO A 119      -1.214 -16.042 -12.485  1.00 31.04           O  \\nATOM    926  CB  PRO A 119      -0.623 -18.435 -10.545  1.00 31.04           C  \\nATOM    927  CG  PRO A 119      -1.217 -17.912  -9.277  1.00 31.04           C  \\nATOM    928  CD  PRO A 119      -0.669 -16.536  -9.026  1.00 31.04           C  \\nATOM    929  N   ARG A 120       0.627 -16.348 -13.238  1.00 34.00           N  \\nATOM    930  CA  ARG A 120       0.318 -16.194 -14.656  1.00 34.00           C  \\nATOM    931  C   ARG A 120      -1.065 -16.751 -14.978  1.00 34.00           C  \\nATOM    932  O   ARG A 120      -1.830 -16.135 -15.724  1.00 34.00           O  \\nATOM    933  CB  ARG A 120       1.376 -16.889 -15.516  1.00 34.00           C  \\nATOM    934  CG  ARG A 120       2.159 -15.942 -16.413  1.00 34.00           C  \\nATOM    935  CD  ARG A 120       3.092 -16.696 -17.349  1.00 34.00           C  \\nATOM    936  NE  ARG A 120       3.721 -15.803 -18.318  1.00 34.00           N  \\nATOM    937  CZ  ARG A 120       4.762 -16.124 -19.081  1.00 34.00           C  \\nATOM    938  NH1 ARG A 120       5.314 -17.329 -19.004  1.00 34.00           N  \\nATOM    939  NH2 ARG A 120       5.255 -15.233 -19.929  1.00 34.00           N  \\nATOM    940  N   ASN A 121      -1.306 -17.883 -14.489  1.00 34.78           N  \\nATOM    941  CA  ASN A 121      -2.472 -18.562 -15.043  1.00 34.78           C  \\nATOM    942  C   ASN A 121      -3.679 -18.450 -14.116  1.00 34.78           C  \\nATOM    943  O   ASN A 121      -4.516 -19.353 -14.068  1.00 34.78           O  \\nATOM    944  CB  ASN A 121      -2.154 -20.031 -15.326  1.00 34.78           C  \\nATOM    945  CG  ASN A 121      -1.678 -20.263 -16.747  1.00 34.78           C  \\nATOM    946  OD1 ASN A 121      -1.729 -19.360 -17.586  1.00 34.78           O  \\nATOM    947  ND2 ASN A 121      -1.211 -21.474 -17.026  1.00 34.78           N  \\nATOM    948  N   SER A 122      -3.596 -17.709 -13.082  1.00 35.29           N  \\nATOM    949  CA  SER A 122      -4.869 -17.440 -12.421  1.00 35.29           C  \\nATOM    950  C   SER A 122      -5.772 -16.573 -13.293  1.00 35.29           C  \\nATOM    951  O   SER A 122      -5.395 -15.463 -13.675  1.00 35.29           O  \\nATOM    952  CB  SER A 122      -4.640 -16.756 -11.073  1.00 35.29           C  \\nATOM    953  OG  SER A 122      -3.600 -15.798 -11.165  1.00 35.29           O  \\nATOM    954  N   GLY A 123      -5.745 -16.878 -14.580  1.00 34.67           N  \\nATOM    955  CA  GLY A 123      -6.494 -16.408 -15.734  1.00 34.67           C  \\nATOM    956  C   GLY A 123      -7.565 -15.394 -15.379  1.00 34.67           C  \\nATOM    957  O   GLY A 123      -7.831 -14.469 -16.148  1.00 34.67           O  \\nATOM    958  N   THR A 124      -7.870 -15.193 -14.152  1.00 37.20           N  \\nATOM    959  CA  THR A 124      -8.918 -14.178 -14.147  1.00 37.20           C  \\nATOM    960  C   THR A 124      -8.509 -12.983 -13.291  1.00 37.20           C  \\nATOM    961  O   THR A 124      -7.626 -13.097 -12.438  1.00 37.20           O  \\nATOM    962  CB  THR A 124     -10.250 -14.752 -13.629  1.00 37.20           C  \\nATOM    963  OG1 THR A 124     -10.048 -15.308 -12.324  1.00 37.20           O  \\nATOM    964  CG2 THR A 124     -10.777 -15.842 -14.557  1.00 37.20           C  \\nATOM    965  N   LYS A 125      -8.484 -11.830 -13.923  1.00 35.39           N  \\nATOM    966  CA  LYS A 125      -8.479 -10.553 -13.216  1.00 35.39           C  \\nATOM    967  C   LYS A 125      -8.897 -10.730 -11.759  1.00 35.39           C  \\nATOM    968  O   LYS A 125      -8.308 -10.125 -10.860  1.00 35.39           O  \\nATOM    969  CB  LYS A 125      -9.405  -9.552 -13.909  1.00 35.39           C  \\nATOM    970  CG  LYS A 125      -8.705  -8.663 -14.926  1.00 35.39           C  \\nATOM    971  CD  LYS A 125      -9.681  -7.705 -15.596  1.00 35.39           C  \\nATOM    972  CE  LYS A 125      -9.019  -6.934 -16.730  1.00 35.39           C  \\nATOM    973  NZ  LYS A 125      -9.970  -5.988 -17.386  1.00 35.39           N  \\nATOM    974  N   THR A 126      -9.706 -11.726 -11.548  1.00 36.34           N  \\nATOM    975  CA  THR A 126     -10.302 -12.023 -10.251  1.00 36.34           C  \\nATOM    976  C   THR A 126      -9.260 -12.598  -9.296  1.00 36.34           C  \\nATOM    977  O   THR A 126      -9.172 -12.180  -8.139  1.00 36.34           O  \\nATOM    978  CB  THR A 126     -11.476 -13.010 -10.389  1.00 36.34           C  \\nATOM    979  OG1 THR A 126     -12.388 -12.522 -11.381  1.00 36.34           O  \\nATOM    980  CG2 THR A 126     -12.219 -13.169  -9.066  1.00 36.34           C  \\nATOM    981  N   GLU A 127      -8.621 -13.590  -9.794  1.00 39.03           N  \\nATOM    982  CA  GLU A 127      -7.674 -14.304  -8.943  1.00 39.03           C  \\nATOM    983  C   GLU A 127      -6.513 -13.403  -8.531  1.00 39.03           C  \\nATOM    984  O   GLU A 127      -6.037 -13.475  -7.397  1.00 39.03           O  \\nATOM    985  CB  GLU A 127      -7.144 -15.551  -9.656  1.00 39.03           C  \\nATOM    986  CG  GLU A 127      -8.072 -16.754  -9.562  1.00 39.03           C  \\nATOM    987  CD  GLU A 127      -7.522 -17.991 -10.253  1.00 39.03           C  \\nATOM    988  OE1 GLU A 127      -8.013 -19.110  -9.978  1.00 39.03           O  \\nATOM    989  OE2 GLU A 127      -6.592 -17.842 -11.077  1.00 39.03           O  \\nATOM    990  N   ALA A 128      -6.179 -12.575  -9.468  1.00 36.53           N  \\nATOM    991  CA  ALA A 128      -5.092 -11.637  -9.202  1.00 36.53           C  \\nATOM    992  C   ALA A 128      -5.464 -10.669  -8.083  1.00 36.53           C  \\nATOM    993  O   ALA A 128      -4.666 -10.420  -7.176  1.00 36.53           O  \\nATOM    994  CB  ALA A 128      -4.732 -10.866 -10.470  1.00 36.53           C  \\nATOM    995  N   ILE A 129      -6.675 -10.219  -8.188  1.00 38.12           N  \\nATOM    996  CA  ILE A 129      -7.181  -9.276  -7.196  1.00 38.12           C  \\nATOM    997  C   ILE A 129      -7.234  -9.948  -5.826  1.00 38.12           C  \\nATOM    998  O   ILE A 129      -6.819  -9.361  -4.823  1.00 38.12           O  \\nATOM    999  CB  ILE A 129      -8.577  -8.741  -7.585  1.00 38.12           C  \\nATOM   1000  CG1 ILE A 129      -8.481  -7.859  -8.835  1.00 38.12           C  \\nATOM   1001  CG2 ILE A 129      -9.205  -7.972  -6.419  1.00 38.12           C  \\nATOM   1002  CD1 ILE A 129      -9.829  -7.409  -9.381  1.00 38.12           C  \\nATOM   1003  N   HIS A 130      -7.764 -11.175  -5.874  1.00 40.63           N  \\nATOM   1004  CA  HIS A 130      -7.859 -11.893  -4.608  1.00 40.63           C  \\nATOM   1005  C   HIS A 130      -6.489 -12.044  -3.955  1.00 40.63           C  \\nATOM   1006  O   HIS A 130      -6.357 -11.897  -2.738  1.00 40.63           O  \\nATOM   1007  CB  HIS A 130      -8.494 -13.269  -4.819  1.00 40.63           C  \\nATOM   1008  CG  HIS A 130      -9.952 -13.314  -4.487  1.00 40.63           C  \\nATOM   1009  ND1 HIS A 130     -10.422 -13.255  -3.193  1.00 40.63           N  \\nATOM   1010  CD2 HIS A 130     -11.042 -13.410  -5.283  1.00 40.63           C  \\nATOM   1011  CE1 HIS A 130     -11.743 -13.314  -3.208  1.00 40.63           C  \\nATOM   1012  NE2 HIS A 130     -12.145 -13.408  -4.464  1.00 40.63           N  \\nATOM   1013  N   HIS A 131      -5.572 -12.238  -4.816  1.00 40.79           N  \\nATOM   1014  CA  HIS A 131      -4.220 -12.450  -4.311  1.00 40.79           C  \\nATOM   1015  C   HIS A 131      -3.643 -11.166  -3.725  1.00 40.79           C  \\nATOM   1016  O   HIS A 131      -3.049 -11.184  -2.644  1.00 40.79           O  \\nATOM   1017  CB  HIS A 131      -3.310 -12.976  -5.422  1.00 40.79           C  \\nATOM   1018  CG  HIS A 131      -1.991 -13.480  -4.929  1.00 40.79           C  \\nATOM   1019  ND1 HIS A 131      -1.871 -14.609  -4.148  1.00 40.79           N  \\nATOM   1020  CD2 HIS A 131      -0.735 -13.008  -5.110  1.00 40.79           C  \\nATOM   1021  CE1 HIS A 131      -0.594 -14.810  -3.869  1.00 40.79           C  \\nATOM   1022  NE2 HIS A 131       0.116 -13.853  -4.440  1.00 40.79           N  \\nATOM   1023  N   ILE A 132      -3.853 -10.111  -4.470  1.00 38.66           N  \\nATOM   1024  CA  ILE A 132      -3.362  -8.823  -3.993  1.00 38.66           C  \\nATOM   1025  C   ILE A 132      -4.000  -8.494  -2.645  1.00 38.66           C  \\nATOM   1026  O   ILE A 132      -3.318  -8.047  -1.720  1.00 38.66           O  \\nATOM   1027  CB  ILE A 132      -3.648  -7.696  -5.011  1.00 38.66           C  \\nATOM   1028  CG1 ILE A 132      -2.803  -7.891  -6.275  1.00 38.66           C  \\nATOM   1029  CG2 ILE A 132      -3.388  -6.323  -4.385  1.00 38.66           C  \\nATOM   1030  CD1 ILE A 132      -3.151  -6.933  -7.406  1.00 38.66           C  \\nATOM   1031  N   LEU A 133      -5.274  -8.771  -2.550  1.00 39.66           N  \\nATOM   1032  CA  LEU A 133      -5.979  -8.494  -1.303  1.00 39.66           C  \\nATOM   1033  C   LEU A 133      -5.413  -9.331  -0.161  1.00 39.66           C  \\nATOM   1034  O   LEU A 133      -5.257  -8.837   0.958  1.00 39.66           O  \\nATOM   1035  CB  LEU A 133      -7.476  -8.770  -1.461  1.00 39.66           C  \\nATOM   1036  CG  LEU A 133      -8.275  -7.740  -2.262  1.00 39.66           C  \\nATOM   1037  CD1 LEU A 133      -9.675  -8.269  -2.557  1.00 39.66           C  \\nATOM   1038  CD2 LEU A 133      -8.345  -6.415  -1.511  1.00 39.66           C  \\nATOM   1039  N   GLU A 134      -5.138 -10.509  -0.474  1.00 41.35           N  \\nATOM   1040  CA  GLU A 134      -4.553 -11.396   0.527  1.00 41.35           C  \\nATOM   1041  C   GLU A 134      -3.221 -10.852   1.036  1.00 41.35           C  \\nATOM   1042  O   GLU A 134      -2.944 -10.898   2.236  1.00 41.35           O  \\nATOM   1043  CB  GLU A 134      -4.361 -12.802  -0.048  1.00 41.35           C  \\nATOM   1044  CG  GLU A 134      -5.012 -13.900   0.782  1.00 41.35           C  \\nATOM   1045  CD  GLU A 134      -4.913 -15.275   0.142  1.00 41.35           C  \\nATOM   1046  OE1 GLU A 134      -5.416 -16.257   0.732  1.00 41.35           O  \\nATOM   1047  OE2 GLU A 134      -4.325 -15.371  -0.959  1.00 41.35           O  \\nATOM   1048  N   LEU A 135      -2.496 -10.385   0.074  1.00 38.21           N  \\nATOM   1049  CA  LEU A 135      -1.176  -9.874   0.430  1.00 38.21           C  \\nATOM   1050  C   LEU A 135      -1.292  -8.642   1.320  1.00 38.21           C  \\nATOM   1051  O   LEU A 135      -0.541  -8.496   2.287  1.00 38.21           O  \\nATOM   1052  CB  LEU A 135      -0.376  -9.534  -0.830  1.00 38.21           C  \\nATOM   1053  CG  LEU A 135       0.274 -10.712  -1.559  1.00 38.21           C  \\nATOM   1054  CD1 LEU A 135       0.641 -10.315  -2.985  1.00 38.21           C  \\nATOM   1055  CD2 LEU A 135       1.504 -11.197  -0.800  1.00 38.21           C  \\nATOM   1056  N   ILE A 136      -2.218  -7.828   0.962  1.00 38.12           N  \\nATOM   1057  CA  ILE A 136      -2.485  -6.633   1.755  1.00 38.12           C  \\nATOM   1058  C   ILE A 136      -2.859  -7.033   3.181  1.00 38.12           C  \\nATOM   1059  O   ILE A 136      -2.366  -6.446   4.146  1.00 38.12           O  \\nATOM   1060  CB  ILE A 136      -3.607  -5.777   1.127  1.00 38.12           C  \\nATOM   1061  CG1 ILE A 136      -3.137  -5.170  -0.200  1.00 38.12           C  \\nATOM   1062  CG2 ILE A 136      -4.061  -4.683   2.099  1.00 38.12           C  \\nATOM   1063  CD1 ILE A 136      -4.238  -4.472  -0.986  1.00 38.12           C  \\nATOM   1064  N   GLU A 137      -3.697  -8.018   3.289  1.00 38.61           N  \\nATOM   1065  CA  GLU A 137      -4.160  -8.511   4.583  1.00 38.61           C  \\nATOM   1066  C   GLU A 137      -2.993  -8.989   5.441  1.00 38.61           C  \\nATOM   1067  O   GLU A 137      -2.934  -8.698   6.637  1.00 38.61           O  \\nATOM   1068  CB  GLU A 137      -5.174  -9.642   4.396  1.00 38.61           C  \\nATOM   1069  CG  GLU A 137      -6.624  -9.188   4.480  1.00 38.61           C  \\nATOM   1070  CD  GLU A 137      -7.620 -10.316   4.263  1.00 38.61           C  \\nATOM   1071  OE1 GLU A 137      -8.845 -10.060   4.296  1.00 38.61           O  \\nATOM   1072  OE2 GLU A 137      -7.171 -11.467   4.059  1.00 38.61           O  \\nATOM   1073  N   LEU A 138      -2.200  -9.788   4.817  1.00 37.67           N  \\nATOM   1074  CA  LEU A 138      -1.081 -10.387   5.535  1.00 37.67           C  \\nATOM   1075  C   LEU A 138      -0.132  -9.313   6.057  1.00 37.67           C  \\nATOM   1076  O   LEU A 138       0.335  -9.392   7.196  1.00 37.67           O  \\nATOM   1077  CB  LEU A 138      -0.321 -11.359   4.628  1.00 37.67           C  \\nATOM   1078  CG  LEU A 138      -0.925 -12.755   4.473  1.00 37.67           C  \\nATOM   1079  CD1 LEU A 138      -0.412 -13.416   3.197  1.00 37.67           C  \\nATOM   1080  CD2 LEU A 138      -0.605 -13.614   5.692  1.00 37.67           C  \\nATOM   1081  N   GLU A 139       0.026  -8.331   5.222  1.00 36.33           N  \\nATOM   1082  CA  GLU A 139       0.957  -7.274   5.606  1.00 36.33           C  \\nATOM   1083  C   GLU A 139       0.376  -6.404   6.717  1.00 36.33           C  \\nATOM   1084  O   GLU A 139       1.085  -6.027   7.652  1.00 36.33           O  \\nATOM   1085  CB  GLU A 139       1.319  -6.409   4.396  1.00 36.33           C  \\nATOM   1086  CG  GLU A 139       2.729  -6.641   3.874  1.00 36.33           C  \\nATOM   1087  CD  GLU A 139       3.081  -5.762   2.684  1.00 36.33           C  \\nATOM   1088  OE1 GLU A 139       4.226  -5.844   2.186  1.00 36.33           O  \\nATOM   1089  OE2 GLU A 139       2.203  -4.985   2.247  1.00 36.33           O  \\nATOM   1090  N   GLN A 140      -0.845  -6.023   6.441  1.00 41.00           N  \\nATOM   1091  CA  GLN A 140      -1.501  -5.212   7.461  1.00 41.00           C  \\nATOM   1092  C   GLN A 140      -1.412  -5.875   8.832  1.00 41.00           C  \\nATOM   1093  O   GLN A 140      -1.216  -5.197   9.844  1.00 41.00           O  \\nATOM   1094  CB  GLN A 140      -2.964  -4.964   7.091  1.00 41.00           C  \\nATOM   1095  CG  GLN A 140      -3.156  -3.904   6.015  1.00 41.00           C  \\nATOM   1096  CD  GLN A 140      -4.614  -3.547   5.796  1.00 41.00           C  \\nATOM   1097  OE1 GLN A 140      -5.496  -4.004   6.530  1.00 41.00           O  \\nATOM   1098  NE2 GLN A 140      -4.878  -2.730   4.782  1.00 41.00           N  \\nATOM   1099  N   HIS A 141      -1.576  -7.171   8.698  1.00 39.41           N  \\nATOM   1100  CA  HIS A 141      -1.461  -7.906   9.953  1.00 39.41           C  \\nATOM   1101  C   HIS A 141      -0.080  -7.721  10.574  1.00 39.41           C  \\nATOM   1102  O   HIS A 141       0.037  -7.497  11.781  1.00 39.41           O  \\nATOM   1103  CB  HIS A 141      -1.743  -9.393   9.732  1.00 39.41           C  \\nATOM   1104  CG  HIS A 141      -1.821 -10.184  10.999  1.00 39.41           C  \\nATOM   1105  ND1 HIS A 141      -2.840 -10.026  11.913  1.00 39.41           N  \\nATOM   1106  CD2 HIS A 141      -1.007 -11.142  11.500  1.00 39.41           C  \\nATOM   1107  CE1 HIS A 141      -2.647 -10.854  12.925  1.00 39.41           C  \\nATOM   1108  NE2 HIS A 141      -1.542 -11.543  12.699  1.00 39.41           N  \\nATOM   1109  N   GLU A 142       0.866  -7.749   9.676  1.00 41.86           N  \\nATOM   1110  CA  GLU A 142       2.227  -7.660  10.196  1.00 41.86           C  \\nATOM   1111  C   GLU A 142       2.539  -6.251  10.690  1.00 41.86           C  \\nATOM   1112  O   GLU A 142       3.156  -6.079  11.743  1.00 41.86           O  \\nATOM   1113  CB  GLU A 142       3.240  -8.079   9.127  1.00 41.86           C  \\nATOM   1114  CG  GLU A 142       3.638  -9.546   9.198  1.00 41.86           C  \\nATOM   1115  CD  GLU A 142       4.642  -9.948   8.129  1.00 41.86           C  \\nATOM   1116  OE1 GLU A 142       5.016 -11.141   8.066  1.00 41.86           O  \\nATOM   1117  OE2 GLU A 142       5.057  -9.062   7.349  1.00 41.86           O  \\nATOM   1118  N   GLU A 143       2.043  -5.311   9.880  1.00 39.82           N  \\nATOM   1119  CA  GLU A 143       2.288  -3.926  10.269  1.00 39.82           C  \\nATOM   1120  C   GLU A 143       1.577  -3.586  11.576  1.00 39.82           C  \\nATOM   1121  O   GLU A 143       2.147  -2.922  12.444  1.00 39.82           O  \\nATOM   1122  CB  GLU A 143       1.840  -2.969   9.162  1.00 39.82           C  \\nATOM   1123  CG  GLU A 143       2.706  -1.724   9.037  1.00 39.82           C  \\nATOM   1124  CD  GLU A 143       2.444  -0.936   7.763  1.00 39.82           C  \\nATOM   1125  OE1 GLU A 143       3.167   0.052   7.501  1.00 39.82           O  \\nATOM   1126  OE2 GLU A 143       1.508  -1.309   7.022  1.00 39.82           O  \\nATOM   1127  N   ARG A 144       0.343  -4.065  11.613  1.00 38.38           N  \\nATOM   1128  CA  ARG A 144      -0.425  -3.878  12.840  1.00 38.38           C  \\nATOM   1129  C   ARG A 144       0.338  -4.407  14.049  1.00 38.38           C  \\nATOM   1130  O   ARG A 144       0.376  -3.761  15.098  1.00 38.38           O  \\nATOM   1131  CB  ARG A 144      -1.785  -4.571  12.734  1.00 38.38           C  \\nATOM   1132  CG  ARG A 144      -2.841  -4.001  13.668  1.00 38.38           C  \\nATOM   1133  CD  ARG A 144      -4.141  -4.790  13.597  1.00 38.38           C  \\nATOM   1134  NE  ARG A 144      -5.304  -3.931  13.800  1.00 38.38           N  \\nATOM   1135  CZ  ARG A 144      -6.398  -4.282  14.471  1.00 38.38           C  \\nATOM   1136  NH1 ARG A 144      -6.499  -5.488  15.019  1.00 38.38           N  \\nATOM   1137  NH2 ARG A 144      -7.398  -3.422  14.596  1.00 38.38           N  \\nATOM   1138  N   THR A 145       0.823  -5.579  13.808  1.00 41.78           N  \\nATOM   1139  CA  THR A 145       1.515  -6.259  14.897  1.00 41.78           C  \\nATOM   1140  C   THR A 145       2.738  -5.463  15.342  1.00 41.78           C  \\nATOM   1141  O   THR A 145       2.978  -5.301  16.541  1.00 41.78           O  \\nATOM   1142  CB  THR A 145       1.947  -7.679  14.484  1.00 41.78           C  \\nATOM   1143  OG1 THR A 145       0.816  -8.379  13.949  1.00 41.78           O  \\nATOM   1144  CG2 THR A 145       2.494  -8.456  15.676  1.00 41.78           C  \\nATOM   1145  N   GLN A 146       3.351  -4.877  14.335  1.00 38.58           N  \\nATOM   1146  CA  GLN A 146       4.551  -4.114  14.664  1.00 38.58           C  \\nATOM   1147  C   GLN A 146       4.196  -2.810  15.372  1.00 38.58           C  \\nATOM   1148  O   GLN A 146       4.877  -2.405  16.316  1.00 38.58           O  \\nATOM   1149  CB  GLN A 146       5.366  -3.824  13.403  1.00 38.58           C  \\nATOM   1150  CG  GLN A 146       6.170  -5.016  12.903  1.00 38.58           C  \\nATOM   1151  CD  GLN A 146       7.046  -4.674  11.711  1.00 38.58           C  \\nATOM   1152  OE1 GLN A 146       6.906  -3.606  11.107  1.00 38.58           O  \\nATOM   1153  NE2 GLN A 146       7.955  -5.579  11.364  1.00 38.58           N  \\nATOM   1154  N   LEU A 147       3.188  -2.213  14.814  1.00 33.95           N  \\nATOM   1155  CA  LEU A 147       2.769  -0.953  15.420  1.00 33.95           C  \\nATOM   1156  C   LEU A 147       2.307  -1.168  16.857  1.00 33.95           C  \\nATOM   1157  O   LEU A 147       2.613  -0.362  17.739  1.00 33.95           O  \\nATOM   1158  CB  LEU A 147       1.645  -0.312  14.601  1.00 33.95           C  \\nATOM   1159  CG  LEU A 147       1.774   1.187  14.330  1.00 33.95           C  \\nATOM   1160  CD1 LEU A 147       2.271   1.428  12.908  1.00 33.95           C  \\nATOM   1161  CD2 LEU A 147       0.441   1.890  14.562  1.00 33.95           C  \\nATOM   1162  N   ILE A 148       1.640  -2.289  17.036  1.00 37.46           N  \\nATOM   1163  CA  ILE A 148       1.179  -2.701  18.357  1.00 37.46           C  \\nATOM   1164  C   ILE A 148       2.370  -2.827  19.304  1.00 37.46           C  \\nATOM   1165  O   ILE A 148       2.325  -2.339  20.436  1.00 37.46           O  \\nATOM   1166  CB  ILE A 148       0.401  -4.034  18.294  1.00 37.46           C  \\nATOM   1167  CG1 ILE A 148      -0.933  -3.840  17.563  1.00 37.46           C  \\nATOM   1168  CG2 ILE A 148       0.178  -4.597  19.701  1.00 37.46           C  \\nATOM   1169  CD1 ILE A 148      -1.672  -5.138  17.269  1.00 37.46           C  \\nATOM   1170  N   SER A 149       3.330  -3.576  18.761  1.00 40.09           N  \\nATOM   1171  CA  SER A 149       4.516  -3.829  19.574  1.00 40.09           C  \\nATOM   1172  C   SER A 149       5.244  -2.531  19.907  1.00 40.09           C  \\nATOM   1173  O   SER A 149       5.658  -2.321  21.049  1.00 40.09           O  \\nATOM   1174  CB  SER A 149       5.466  -4.787  18.854  1.00 40.09           C  \\nATOM   1175  OG  SER A 149       6.730  -4.817  19.494  1.00 40.09           O  \\nATOM   1176  N   TYR A 150       5.237  -1.720  18.876  1.00 34.15           N  \\nATOM   1177  CA  TYR A 150       5.913  -0.439  19.052  1.00 34.15           C  \\nATOM   1178  C   TYR A 150       5.167   0.439  20.050  1.00 34.15           C  \\nATOM   1179  O   TYR A 150       5.783   1.078  20.906  1.00 34.15           O  \\nATOM   1180  CB  TYR A 150       6.044   0.289  17.711  1.00 34.15           C  \\nATOM   1181  CG  TYR A 150       6.913   1.521  17.772  1.00 34.15           C  \\nATOM   1182  CD1 TYR A 150       6.349   2.793  17.849  1.00 34.15           C  \\nATOM   1183  CD2 TYR A 150       8.300   1.417  17.750  1.00 34.15           C  \\nATOM   1184  CE1 TYR A 150       7.146   3.931  17.903  1.00 34.15           C  \\nATOM   1185  CE2 TYR A 150       9.107   2.549  17.803  1.00 34.15           C  \\nATOM   1186  CZ  TYR A 150       8.522   3.799  17.880  1.00 34.15           C  \\nATOM   1187  OH  TYR A 150       9.316   4.923  17.933  1.00 34.15           O  \\nATOM   1188  N   GLY A 151       3.867   0.438  19.913  1.00 33.68           N  \\nATOM   1189  CA  GLY A 151       3.052   1.234  20.817  1.00 33.68           C  \\nATOM   1190  C   GLY A 151       3.113   0.755  22.255  1.00 33.68           C  \\nATOM   1191  O   GLY A 151       3.188   1.564  23.182  1.00 33.68           O  \\nATOM   1192  N   ARG A 152       3.137  -0.500  22.440  1.00 37.17           N  \\nATOM   1193  CA  ARG A 152       3.268  -1.075  23.774  1.00 37.17           C  \\nATOM   1194  C   ARG A 152       4.593  -0.674  24.415  1.00 37.17           C  \\nATOM   1195  O   ARG A 152       4.642  -0.357  25.606  1.00 37.17           O  \\nATOM   1196  CB  ARG A 152       3.154  -2.600  23.717  1.00 37.17           C  \\nATOM   1197  CG  ARG A 152       1.735  -3.118  23.887  1.00 37.17           C  \\nATOM   1198  CD  ARG A 152       1.690  -4.640  23.910  1.00 37.17           C  \\nATOM   1199  NE  ARG A 152       0.333  -5.135  24.125  1.00 37.17           N  \\nATOM   1200  CZ  ARG A 152      -0.036  -6.408  24.008  1.00 37.17           C  \\nATOM   1201  NH1 ARG A 152       0.846  -7.342  23.675  1.00 37.17           N  \\nATOM   1202  NH2 ARG A 152      -1.298  -6.749  24.227  1.00 37.17           N  \\nATOM   1203  N   ASN A 153       5.633  -0.656  23.538  1.00 38.01           N  \\nATOM   1204  CA  ASN A 153       6.950  -0.338  24.079  1.00 38.01           C  \\nATOM   1205  C   ASN A 153       7.037   1.119  24.524  1.00 38.01           C  \\nATOM   1206  O   ASN A 153       7.774   1.444  25.456  1.00 38.01           O  \\nATOM   1207  CB  ASN A 153       8.041  -0.645  23.050  1.00 38.01           C  \\nATOM   1208  CG  ASN A 153       8.313  -2.130  22.913  1.00 38.01           C  \\nATOM   1209  OD1 ASN A 153       8.002  -2.916  23.812  1.00 38.01           O  \\nATOM   1210  ND2 ASN A 153       8.895  -2.525  21.787  1.00 38.01           N  \\nATOM   1211  N   LEU A 154       6.149   1.874  23.966  1.00 37.24           N  \\nATOM   1212  CA  LEU A 154       6.212   3.287  24.326  1.00 37.24           C  \\nATOM   1213  C   LEU A 154       5.149   3.632  25.363  1.00 37.24           C  \\nATOM   1214  O   LEU A 154       5.073   4.773  25.825  1.00 37.24           O  \\nATOM   1215  CB  LEU A 154       6.035   4.164  23.084  1.00 37.24           C  \\nATOM   1216  CG  LEU A 154       7.210   4.201  22.106  1.00 37.24           C  \\nATOM   1217  CD1 LEU A 154       6.757   4.740  20.753  1.00 37.24           C  \\nATOM   1218  CD2 LEU A 154       8.349   5.044  22.670  1.00 37.24           C  \\nATOM   1219  N   GLY A 155       4.348   2.596  25.694  1.00 38.22           N  \\nATOM   1220  CA  GLY A 155       3.325   2.821  26.703  1.00 38.22           C  \\nATOM   1221  C   GLY A 155       2.077   3.483  26.150  1.00 38.22           C  \\nATOM   1222  O   GLY A 155       1.397   4.227  26.860  1.00 38.22           O  \\nATOM   1223  N   LEU A 156       1.975   3.544  24.804  1.00 32.14           N  \\nATOM   1224  CA  LEU A 156       0.886   4.243  24.130  1.00 32.14           C  \\nATOM   1225  C   LEU A 156      -0.312   3.321  23.931  1.00 32.14           C  \\nATOM   1226  O   LEU A 156      -1.090   3.500  22.991  1.00 32.14           O  \\nATOM   1227  CB  LEU A 156       1.354   4.789  22.778  1.00 32.14           C  \\nATOM   1228  CG  LEU A 156       2.471   5.834  22.819  1.00 32.14           C  \\nATOM   1229  CD1 LEU A 156       3.040   6.057  21.422  1.00 32.14           C  \\nATOM   1230  CD2 LEU A 156       1.957   7.143  23.408  1.00 32.14           C  \\nATOM   1231  N   ASP A 157      -0.456   2.373  24.849  1.00 29.26           N  \\nATOM   1232  CA  ASP A 157      -1.521   1.380  24.746  1.00 29.26           C  \\nATOM   1233  C   ASP A 157      -2.882   2.049  24.569  1.00 29.26           C  \\nATOM   1234  O   ASP A 157      -3.748   1.530  23.862  1.00 29.26           O  \\nATOM   1235  CB  ASP A 157      -1.535   0.477  25.981  1.00 29.26           C  \\nATOM   1236  CG  ASP A 157      -0.712  -0.787  25.802  1.00 29.26           C  \\nATOM   1237  OD1 ASP A 157      -0.294  -1.086  24.663  1.00 29.26           O  \\nATOM   1238  OD2 ASP A 157      -0.483  -1.492  26.808  1.00 29.26           O  \\nATOM   1239  N   ASP A 158      -2.997   3.126  25.254  1.00 29.77           N  \\nATOM   1240  CA  ASP A 158      -4.304   3.776  25.277  1.00 29.77           C  \\nATOM   1241  C   ASP A 158      -4.663   4.334  23.902  1.00 29.77           C  \\nATOM   1242  O   ASP A 158      -5.834   4.338  23.514  1.00 29.77           O  \\nATOM   1243  CB  ASP A 158      -4.329   4.894  26.322  1.00 29.77           C  \\nATOM   1244  CG  ASP A 158      -4.106   4.389  27.736  1.00 29.77           C  \\nATOM   1245  OD1 ASP A 158      -4.399   3.206  28.015  1.00 29.77           O  \\nATOM   1246  OD2 ASP A 158      -3.635   5.182  28.580  1.00 29.77           O  \\nATOM   1247  N   ILE A 159      -3.674   4.844  23.275  1.00 27.55           N  \\nATOM   1248  CA  ILE A 159      -3.891   5.431  21.957  1.00 27.55           C  \\nATOM   1249  C   ILE A 159      -4.218   4.330  20.950  1.00 27.55           C  \\nATOM   1250  O   ILE A 159      -5.102   4.495  20.106  1.00 27.55           O  \\nATOM   1251  CB  ILE A 159      -2.661   6.238  21.487  1.00 27.55           C  \\nATOM   1252  CG1 ILE A 159      -2.368   7.383  22.465  1.00 27.55           C  \\nATOM   1253  CG2 ILE A 159      -2.874   6.772  20.068  1.00 27.55           C  \\nATOM   1254  CD1 ILE A 159      -1.049   8.097  22.205  1.00 27.55           C  \\nATOM   1255  N   ILE A 160      -3.443   3.233  21.173  1.00 27.13           N  \\nATOM   1256  CA  ILE A 160      -3.547   2.154  20.197  1.00 27.13           C  \\nATOM   1257  C   ILE A 160      -4.890   1.444  20.354  1.00 27.13           C  \\nATOM   1258  O   ILE A 160      -5.567   1.158  19.364  1.00 27.13           O  \\nATOM   1259  CB  ILE A 160      -2.387   1.145  20.344  1.00 27.13           C  \\nATOM   1260  CG1 ILE A 160      -1.039   1.846  20.136  1.00 27.13           C  \\nATOM   1261  CG2 ILE A 160      -2.555  -0.019  19.362  1.00 27.13           C  \\nATOM   1262  CD1 ILE A 160       0.159   0.907  20.168  1.00 27.13           C  \\nATOM   1263  N   TYR A 161      -5.308   1.248  21.616  1.00 30.81           N  \\nATOM   1264  CA  TYR A 161      -6.567   0.531  21.784  1.00 30.81           C  \\nATOM   1265  C   TYR A 161      -7.722   1.501  22.005  1.00 30.81           C  \\nATOM   1266  O   TYR A 161      -8.883   1.159  21.769  1.00 30.81           O  \\nATOM   1267  CB  TYR A 161      -6.476  -0.448  22.958  1.00 30.81           C  \\nATOM   1268  CG  TYR A 161      -5.652  -1.677  22.661  1.00 30.81           C  \\nATOM   1269  CD1 TYR A 161      -6.192  -2.745  21.946  1.00 30.81           C  \\nATOM   1270  CD2 TYR A 161      -4.335  -1.775  23.094  1.00 30.81           C  \\nATOM   1271  CE1 TYR A 161      -5.437  -3.881  21.671  1.00 30.81           C  \\nATOM   1272  CE2 TYR A 161      -3.571  -2.906  22.825  1.00 30.81           C  \\nATOM   1273  CZ  TYR A 161      -4.130  -3.952  22.114  1.00 30.81           C  \\nATOM   1274  OH  TYR A 161      -3.379  -5.074  21.844  1.00 30.81           O  \\nATOM   1275  N   GLY A 162      -7.552   2.791  21.777  1.00 26.08           N  \\nATOM   1276  CA  GLY A 162      -8.743   3.522  22.178  1.00 26.08           C  \\nATOM   1277  C   GLY A 162      -9.328   3.036  23.490  1.00 26.08           C  \\nATOM   1278  O   GLY A 162      -9.262   1.846  23.803  1.00 26.08           O  \\nATOM   1279  N   SER A 163      -8.631   3.341  24.599  1.00 23.13           N  \\nATOM   1280  CA  SER A 163      -9.186   3.141  25.934  1.00 23.13           C  \\nATOM   1281  C   SER A 163     -10.555   2.472  25.869  1.00 23.13           C  \\nATOM   1282  O   SER A 163     -11.456   2.955  25.181  1.00 23.13           O  \\nATOM   1283  CB  SER A 163      -9.296   4.474  26.675  1.00 23.13           C  \\nATOM   1284  OG  SER A 163      -9.078   5.560  25.790  1.00 23.13           O  \\nATOM   1285  N   ASP A 164     -10.637   1.351  25.467  1.00 25.30           N  \\nATOM   1286  CA  ASP A 164     -11.866   0.610  25.738  1.00 25.30           C  \\nATOM   1287  C   ASP A 164     -12.616   1.207  26.927  1.00 25.30           C  \\nATOM   1288  O   ASP A 164     -13.592   0.626  27.406  1.00 25.30           O  \\nATOM   1289  CB  ASP A 164     -11.556  -0.865  25.999  1.00 25.30           C  \\nATOM   1290  CG  ASP A 164     -11.423  -1.678  24.723  1.00 25.30           C  \\nATOM   1291  OD1 ASP A 164     -11.766  -1.166  23.636  1.00 25.30           O  \\nATOM   1292  OD2 ASP A 164     -10.974  -2.842  24.807  1.00 25.30           O  \\nATOM   1293  N   SER A 165     -12.095   2.268  27.518  1.00 26.30           N  \\nATOM   1294  CA  SER A 165     -12.850   2.618  28.717  1.00 26.30           C  \\nATOM   1295  C   SER A 165     -14.002   3.561  28.389  1.00 26.30           C  \\nATOM   1296  O   SER A 165     -14.710   4.021  29.288  1.00 26.30           O  \\nATOM   1297  CB  SER A 165     -11.933   3.263  29.758  1.00 26.30           C  \\nATOM   1298  OG  SER A 165     -10.742   2.511  29.914  1.00 26.30           O  \\nATOM   1299  N   GLU A 166     -14.498   3.611  27.149  1.00 27.27           N  \\nATOM   1300  CA  GLU A 166     -15.722   4.406  27.180  1.00 27.27           C  \\nATOM   1301  C   GLU A 166     -16.956   3.527  27.002  1.00 27.27           C  \\nATOM   1302  O   GLU A 166     -18.075   4.033  26.894  1.00 27.27           O  \\nATOM   1303  CB  GLU A 166     -15.687   5.489  26.099  1.00 27.27           C  \\nATOM   1304  CG  GLU A 166     -15.158   6.830  26.587  1.00 27.27           C  \\nATOM   1305  CD  GLU A 166     -14.053   7.395  25.709  1.00 27.27           C  \\nATOM   1306  OE1 GLU A 166     -13.591   8.529  25.969  1.00 27.27           O  \\nATOM   1307  OE2 GLU A 166     -13.647   6.699  24.752  1.00 27.27           O  \\nATOM   1308  N   GLU A 167     -16.797   2.224  26.758  1.00 25.69           N  \\nATOM   1309  CA  GLU A 167     -18.060   1.496  26.666  1.00 25.69           C  \\nATOM   1310  C   GLU A 167     -18.579   1.115  28.049  1.00 25.69           C  \\nATOM   1311  O   GLU A 167     -19.786   0.958  28.244  1.00 25.69           O  \\nATOM   1312  CB  GLU A 167     -17.896   0.243  25.802  1.00 25.69           C  \\nATOM   1313  CG  GLU A 167     -18.190   0.471  24.326  1.00 25.69           C  \\nATOM   1314  CD  GLU A 167     -17.979  -0.770  23.474  1.00 25.69           C  \\nATOM   1315  OE1 GLU A 167     -18.175  -0.699  22.239  1.00 25.69           O  \\nATOM   1316  OE2 GLU A 167     -17.617  -1.823  24.045  1.00 25.69           O  \\nATOM   1317  N   GLU A 168     -17.795   1.360  29.107  1.00 24.06           N  \\nATOM   1318  CA  GLU A 168     -18.479   0.985  30.341  1.00 24.06           C  \\nATOM   1319  C   GLU A 168     -19.124   2.199  31.004  1.00 24.06           C  \\nATOM   1320  O   GLU A 168     -19.728   2.081  32.072  1.00 24.06           O  \\nATOM   1321  CB  GLU A 168     -17.507   0.310  31.311  1.00 24.06           C  \\nATOM   1322  CG  GLU A 168     -17.579  -1.210  31.297  1.00 24.06           C  \\nATOM   1323  CD  GLU A 168     -16.537  -1.869  32.186  1.00 24.06           C  \\nATOM   1324  OE1 GLU A 168     -16.478  -3.119  32.228  1.00 24.06           O  \\nATOM   1325  OE2 GLU A 168     -15.774  -1.129  32.847  1.00 24.06           O  \\nATOM   1326  N   LYS A 169     -19.215   3.328  30.322  1.00 27.60           N  \\nATOM   1327  CA  LYS A 169     -20.064   4.316  30.981  1.00 27.60           C  \\nATOM   1328  C   LYS A 169     -21.482   4.285  30.417  1.00 27.60           C  \\nATOM   1329  O   LYS A 169     -22.356   5.019  30.882  1.00 27.60           O  \\nATOM   1330  CB  LYS A 169     -19.471   5.718  30.832  1.00 27.60           C  \\nATOM   1331  CG  LYS A 169     -18.528   6.114  31.958  1.00 27.60           C  \\nATOM   1332  CD  LYS A 169     -18.061   7.557  31.815  1.00 27.60           C  \\nATOM   1333  CE  LYS A 169     -17.048   7.927  32.890  1.00 27.60           C  \\nATOM   1334  NZ  LYS A 169     -16.505   9.304  32.689  1.00 27.60           N  \\nATOM   1335  N   ALA A 170     -21.738   3.367  29.521  1.00 26.90           N  \\nATOM   1336  CA  ALA A 170     -23.161   3.451  29.203  1.00 26.90           C  \\nATOM   1337  C   ALA A 170     -23.936   2.309  29.854  1.00 26.90           C  \\nATOM   1338  O   ALA A 170     -25.169   2.295  29.831  1.00 26.90           O  \\nATOM   1339  CB  ALA A 170     -23.371   3.438  27.691  1.00 26.90           C  \\nATOM   1340  N   GLU A 171     -23.315   1.665  30.806  1.00 25.00           N  \\nATOM   1341  CA  GLU A 171     -24.312   0.884  31.532  1.00 25.00           C  \\nATOM   1342  C   GLU A 171     -24.296   1.217  33.022  1.00 25.00           C  \\nATOM   1343  O   GLU A 171     -25.202   0.827  33.760  1.00 25.00           O  \\nATOM   1344  CB  GLU A 171     -24.076  -0.615  31.326  1.00 25.00           C  \\nATOM   1345  CG  GLU A 171     -24.866  -1.208  30.169  1.00 25.00           C  \\nATOM   1346  CD  GLU A 171     -24.586  -2.686  29.946  1.00 25.00           C  \\nATOM   1347  OE1 GLU A 171     -25.155  -3.275  28.999  1.00 25.00           O  \\nATOM   1348  OE2 GLU A 171     -23.793  -3.259  30.725  1.00 25.00           O  \\nATOM   1349  N   GLU A 172     -24.195   2.525  33.374  1.00 16.39           N  \\nATOM   1350  CA  GLU A 172     -24.758   2.905  34.667  1.00 16.39           C  \\nATOM   1351  C   GLU A 172     -25.470   4.252  34.584  1.00 16.39           C  \\nATOM   1352  O   GLU A 172     -26.483   4.466  35.252  1.00 16.39           O  \\nATOM   1353  CB  GLU A 172     -23.664   2.953  35.736  1.00 16.39           C  \\nATOM   1354  CG  GLU A 172     -23.370   1.604  36.376  1.00 16.39           C  \\nATOM   1355  CD  GLU A 172     -22.367   1.687  37.517  1.00 16.39           C  \\nATOM   1356  OE1 GLU A 172     -22.064   0.641  38.135  1.00 16.39           O  \\nATOM   1357  OE2 GLU A 172     -21.881   2.806  37.794  1.00 16.39           O  \",\"pdb\",{\"hbondCutoff\": 4.0});\n",
              "\tviewer_17225171870958037.setStyle({\"cartoon\": {\"colorscheme\": {\"prop\": \"b\", \"gradient\": \"roygb\", \"min\": 50, \"max\": 90}}});\n",
              "\tviewer_17225171870958037.zoomTo();\n",
              "viewer_17225171870958037.render();\n",
              "});\n",
              "</script>"
            ]
          },
          "metadata": {}
        }
      ],
      "source": [
        "#@markdown ##Display\n",
        "import py3Dmol\n",
        "\n",
        "seq_idx = 0 #@param {type:\"integer\"}\n",
        "\n",
        "pymol_color_list = [\"#33ff33\",\"#00ffff\",\"#ff33cc\",\"#ffff00\",\"#ff9999\",\"#e5e5e5\",\"#7f7fff\",\"#ff7f00\",\n",
        "                    \"#7fff7f\",\"#199999\",\"#ff007f\",\"#ffdd5e\",\"#8c3f99\",\"#b2b2b2\",\"#007fff\",\"#c4b200\",\n",
        "                    \"#8cb266\",\"#00bfbf\",\"#b27f7f\",\"#fcd1a5\",\"#ff7f7f\",\"#ffbfdd\",\"#7fffff\",\"#ffff7f\",\n",
        "                    \"#00ff7f\",\"#337fcc\",\"#d8337f\",\"#bfff3f\",\"#ff7fff\",\"#d8d8ff\",\"#3fffbf\",\"#b78c4c\",\n",
        "                    \"#339933\",\"#66b2b2\",\"#ba8c84\",\"#84bf00\",\"#b24c66\",\"#7f7f7f\",\"#3f3fa5\",\"#a5512b\"]\n",
        "\n",
        "def show_pdb(pdb_str, show_sidechains=False, show_mainchains=False,\n",
        "             color=\"pLDDT\", chains=None, vmin=50, vmax=90,\n",
        "             size=(800,480), hbondCutoff=4.0,\n",
        "             Ls=None,\n",
        "             animate=False):\n",
        "\n",
        "  if chains is None:\n",
        "    chains = 1 if Ls is None else len(Ls)\n",
        "  view = py3Dmol.view(js='https://3dmol.org/build/3Dmol.js', width=size[0], height=size[1])\n",
        "  if animate:\n",
        "    view.addModelsAsFrames(pdb_str,'pdb',{'hbondCutoff':hbondCutoff})\n",
        "  else:\n",
        "    view.addModel(pdb_str,'pdb',{'hbondCutoff':hbondCutoff})\n",
        "  if color == \"pLDDT\":\n",
        "    view.setStyle({'cartoon': {'colorscheme': {'prop':'b','gradient': 'roygb','min':vmin,'max':vmax}}})\n",
        "  elif color == \"rainbow\":\n",
        "    view.setStyle({'cartoon': {'color':'spectrum'}})\n",
        "  elif color == \"chain\":\n",
        "    for n,chain,color in zip(range(chains),alphabet_list,pymol_color_list):\n",
        "       view.setStyle({'chain':chain},{'cartoon': {'color':color}})\n",
        "  if show_sidechains:\n",
        "    BB = ['C','O','N']\n",
        "    HP = [\"ALA\",\"GLY\",\"VAL\",\"ILE\",\"LEU\",\"PHE\",\"MET\",\"PRO\",\"TRP\",\"CYS\",\"TYR\"]\n",
        "    view.addStyle({'and':[{'resn':[\"GLY\",\"PRO\"],'invert':True},{'atom':BB,'invert':True}]},\n",
        "                  {'stick':{'colorscheme':f\"WhiteCarbon\",'radius':0.3}})\n",
        "    view.addStyle({'and':[{'resn':\"GLY\"},{'atom':'CA'}]},\n",
        "                  {'sphere':{'colorscheme':f\"WhiteCarbon\",'radius':0.3}})\n",
        "    view.addStyle({'and':[{'resn':\"PRO\"},{'atom':['C','O'],'invert':True}]},\n",
        "                  {'stick':{'colorscheme':f\"WhiteCarbon\",'radius':0.3}})\n",
        "  if show_mainchains:\n",
        "    BB = ['C','O','N','CA']\n",
        "    view.addStyle({'atom':BB},{'stick':{'colorscheme':f\"WhiteCarbon\",'radius':0.3}})\n",
        "  view.zoomTo()\n",
        "  if animate: view.animate()\n",
        "  return view\n",
        "\n",
        "color = \"confidence\" #@param [\"confidence\", \"rainbow\", \"chain\"]\n",
        "if color == \"confidence\": color = \"pLDDT\"\n",
        "show_sidechains = False #@param {type:\"boolean\"}\n",
        "show_mainchains = False #@param {type:\"boolean\"}\n",
        "show_pdb(pdb_strs[seq_idx], color=color, show_sidechains=show_sidechains, show_mainchains=show_mainchains,\n",
        "         Ls=lengths).show()"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "techsiyGBgsr"
      },
      "source": [
        "#Run TMalign\n",
        "\n",
        "\n",
        "To check if structure of scaffolded motif is kept in our generated protein.\n",
        "\n",
        "This will output a TM-score between 0-1, where 1 is the best. Decide on a cut-off value (e.g. 0.9) to decide on \"goodness\" of protein."
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 34,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "cVIGB1OnBgsr",
        "outputId": "7ab898e8-ee5a-44d6-c4d6-a2ff3d78b2f7"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\r0% [Working]\r            \rGet:1 https://cloud.r-project.org/bin/linux/ubuntu jammy-cran40/ InRelease [3,626 B]\n",
            "Get:2 https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/x86_64  InRelease [1,581 B]\n",
            "Hit:3 http://archive.ubuntu.com/ubuntu jammy InRelease\n",
            "Get:4 http://security.ubuntu.com/ubuntu jammy-security InRelease [129 kB]\n",
            "Ign:5 https://r2u.stat.illinois.edu/ubuntu jammy InRelease\n",
            "Get:6 https://r2u.stat.illinois.edu/ubuntu jammy Release [5,713 B]\n",
            "Get:7 https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/x86_64  Packages [861 kB]\n",
            "Get:8 https://r2u.stat.illinois.edu/ubuntu jammy Release.gpg [793 B]\n",
            "Get:9 http://archive.ubuntu.com/ubuntu jammy-updates InRelease [128 kB]\n",
            "Get:10 https://r2u.stat.illinois.edu/ubuntu jammy/main amd64 Packages [2,552 kB]\n",
            "Hit:11 https://ppa.launchpadcontent.net/deadsnakes/ppa/ubuntu jammy InRelease\n",
            "Hit:12 https://ppa.launchpadcontent.net/graphics-drivers/ppa/ubuntu jammy InRelease\n",
            "Hit:13 https://ppa.launchpadcontent.net/ubuntugis/ppa/ubuntu jammy InRelease\n",
            "Get:14 http://archive.ubuntu.com/ubuntu jammy-backports InRelease [127 kB]\n",
            "Get:15 http://security.ubuntu.com/ubuntu jammy-security/universe amd64 Packages [1,130 kB]\n",
            "Get:16 https://r2u.stat.illinois.edu/ubuntu jammy/main all Packages [8,217 kB]\n",
            "Get:17 http://archive.ubuntu.com/ubuntu jammy-updates/universe amd64 Packages [1,421 kB]\n",
            "Get:18 http://security.ubuntu.com/ubuntu jammy-security/main amd64 Packages [2,115 kB]\n",
            "Get:19 http://security.ubuntu.com/ubuntu jammy-security/restricted amd64 Packages [2,785 kB]\n",
            "Get:20 http://archive.ubuntu.com/ubuntu jammy-updates/restricted amd64 Packages [2,884 kB]\n",
            "Get:21 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 Packages [2,396 kB]\n",
            "Fetched 24.8 MB in 3s (9,257 kB/s)\n",
            "Reading package lists... Done\n",
            "W: Skipping acquire of configured file 'main/source/Sources' as repository 'https://r2u.stat.illinois.edu/ubuntu jammy InRelease' does not seem to provide it (sources.list entry misspelt?)\n",
            "Reading package lists... Done\n",
            "Building dependency tree... Done\n",
            "Reading state information... Done\n",
            "build-essential is already the newest version (12.9ubuntu3).\n",
            "0 upgraded, 0 newly installed, 0 to remove and 62 not upgraded.\n",
            "--2024-08-01 13:00:03--  https://zhanglab.ccmb.med.umich.edu/TM-align/TMalign.cpp\n",
            "Resolving zhanglab.ccmb.med.umich.edu (zhanglab.ccmb.med.umich.edu)... 141.213.137.249\n",
            "Connecting to zhanglab.ccmb.med.umich.edu (zhanglab.ccmb.med.umich.edu)|141.213.137.249|:443... connected.\n",
            "HTTP request sent, awaiting response... 301 Moved Permanently\n",
            "Location: https://seq2fun.dcmb.med.umich.edu//TM-align/TMalign.cpp [following]\n",
            "--2024-08-01 13:00:03--  https://seq2fun.dcmb.med.umich.edu//TM-align/TMalign.cpp\n",
            "Resolving seq2fun.dcmb.med.umich.edu (seq2fun.dcmb.med.umich.edu)... 141.213.137.249\n",
            "Connecting to seq2fun.dcmb.med.umich.edu (seq2fun.dcmb.med.umich.edu)|141.213.137.249|:443... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 182099 (178K) [text/plain]\n",
            "Saving to: ‘TMalign.cpp’\n",
            "\n",
            "TMalign.cpp         100%[===================>] 177.83K   749KB/s    in 0.2s    \n",
            "\n",
            "2024-08-01 13:00:04 (749 KB/s) - ‘TMalign.cpp’ saved [182099/182099]\n",
            "\n"
          ]
        }
      ],
      "source": [
        "# Step 1: Install necessary tools\n",
        "!apt-get update\n",
        "!apt-get install -y build-essential\n",
        "\n",
        "# Step 2: Download the TM-align source code\n",
        "!wget https://zhanglab.ccmb.med.umich.edu/TM-align/TMalign.cpp -O TMalign.cpp\n",
        "\n",
        "# Step 3: Compile the source code\n",
        "!g++ -O3 -ffast-math -lm -o TMalign TMalign.cpp\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 35,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "giTMWwlcBgsr",
        "outputId": "4794586d-5b2e-4b81-acbe-4295b12c5b36"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Results for data/processed/evodiff_seq_0_be355.pdb:\n",
            "\n",
            " *********************************************************************\n",
            " * TM-align (Version 20220412): protein structure alignment          *\n",
            " * References: Y Zhang, J Skolnick. Nucl Acids Res 33, 2302-9 (2005) *\n",
            " * Please email comments and suggestions to zhanglab@zhanggroup.org   *\n",
            " *********************************************************************\n",
            "\n",
            "Name of Chain_1: /content/S2DS-Summer24-Polyploy/data/interim/test_extracted_motif.pdb (to be superimposed onto Chain_2)\n",
            "Name of Chain_2: /content/S2DS-Summer24-Polyploy/data/processed/evodiff_seq_0_be355.pdb\n",
            "Length of Chain_1: 22 residues\n",
            "Length of Chain_2: 172 residues\n",
            "\n",
            "Aligned length= 20, RMSD=   2.84, Seq_ID=n_identical/n_aligned= 0.150\n",
            "TM-score= 0.19861 (if normalized by length of Chain_1, i.e., LN=22, d0=0.57)\n",
            "TM-score= 0.09244 (if normalized by length of Chain_2, i.e., LN=172, d0=4.89)\n",
            "(You should use TM-score normalized by length of the reference structure)\n",
            "\n",
            "(\":\" denotes residue pairs of d <  5.0 Angstrom, \".\" denotes other aligned residues)\n",
            "--DKLHNL-NSNWFPEG-S--KP---------------------------------------------------FIYQE-----------------------------------------------------------------------------------------------\n",
            "  ::::.: :::::::. :  .                                                    ::::                                                                                                \n",
            "MVKISYLCDASKHMWPGDIKAI-LDKLHNLNSEKEIECWADEGRASDYIKSISSIEHDFNFGCSEIQFGHEHCYLIFA-KKGLGEGKDLKNGAEARDFDDNEIAEDNIGAALINDHGLILGLYDDDFETDTKNYDRNDRKRMWGKIRAGFNSSEIVNNKDRVKYTASDVEAAKK\n",
            "\n",
            "Total CPU time is  0.01 seconds\n",
            "\n",
            "TM-score: 0.19861, if lower than 0.9 run pipeline again\n",
            "Results for data/processed/evodiff_seq_1_824dc.pdb:\n",
            "\n",
            " *********************************************************************\n",
            " * TM-align (Version 20220412): protein structure alignment          *\n",
            " * References: Y Zhang, J Skolnick. Nucl Acids Res 33, 2302-9 (2005) *\n",
            " * Please email comments and suggestions to zhanglab@zhanggroup.org   *\n",
            " *********************************************************************\n",
            "\n",
            "Name of Chain_1: /content/S2DS-Summer24-Polyploy/data/interim/test_extracted_motif.pdb (to be superimposed onto Chain_2)\n",
            "Name of Chain_2: /content/S2DS-Summer24-Polyploy/data/processed/evodiff_seq_1_824dc.pdb\n",
            "Length of Chain_1: 22 residues\n",
            "Length of Chain_2: 172 residues\n",
            "\n",
            "Aligned length= 11, RMSD=   2.66, Seq_ID=n_identical/n_aligned= 0.091\n",
            "TM-score= 0.12311 (if normalized by length of Chain_1, i.e., LN=22, d0=0.57)\n",
            "TM-score= 0.05203 (if normalized by length of Chain_2, i.e., LN=172, d0=4.89)\n",
            "(You should use TM-score normalized by length of the reference structure)\n",
            "\n",
            "(\":\" denotes residue pairs of d <  5.0 Angstrom, \".\" denotes other aligned residues)\n",
            "DKLHNLNS-----------------------------------------------------------------------------------------------------------------------------------------------------------------NWFPEGSKPFIYQE\n",
            "                                                                                                                                                                         :::   :.::::::\n",
            "--------MDDQTIQKKIAASLLSLILTTYEANEADTLTEENKNPIIDDEWARLFYLGLAILGKLSVKHDSASKHMWPGDIKAILDKLHNLNSNILANFFLQHICTLGKGQQKIKALDSALKAFTATLDSEEPTYRLGKKSNSGSTKVKKGAAPKSKPVTQRVEEDNTRGIV---SQNQATVP\n",
            "\n",
            "Total CPU time is  0.00 seconds\n",
            "\n",
            "TM-score: 0.12311, if lower than 0.9 run pipeline again\n",
            "Results for data/processed/evodiff_seq_0_2d04c.pdb:\n",
            "\n",
            " *********************************************************************\n",
            " * TM-align (Version 20220412): protein structure alignment          *\n",
            " * References: Y Zhang, J Skolnick. Nucl Acids Res 33, 2302-9 (2005) *\n",
            " * Please email comments and suggestions to zhanglab@zhanggroup.org   *\n",
            " *********************************************************************\n",
            "\n",
            "Name of Chain_1: /content/S2DS-Summer24-Polyploy/data/interim/test_extracted_motif.pdb (to be superimposed onto Chain_2)\n",
            "Name of Chain_2: /content/S2DS-Summer24-Polyploy/data/processed/evodiff_seq_0_2d04c.pdb\n",
            "Length of Chain_1: 22 residues\n",
            "Length of Chain_2: 172 residues\n",
            "\n",
            "Aligned length= 19, RMSD=   2.82, Seq_ID=n_identical/n_aligned= 0.053\n",
            "TM-score= 0.16188 (if normalized by length of Chain_1, i.e., LN=22, d0=0.57)\n",
            "TM-score= 0.08742 (if normalized by length of Chain_2, i.e., LN=172, d0=4.89)\n",
            "(You should use TM-score normalized by length of the reference structure)\n",
            "\n",
            "(\":\" denotes residue pairs of d <  5.0 Angstrom, \".\" denotes other aligned residues)\n",
            "-------DK--L--HN-LN--SNWFPEGSKPFIYQE-------------------------------------------------------------------------------------------------------------------------------------------\n",
            "       .:  :  :: ..  :   .::::::::::                                                                                                                                           \n",
            "MNFSSKDRGEFLTFAAKCGKKH---KRKAIFVTGSRGTRSDEMAGYEQFRGKPNAIIGAGIIFLDEASKHMWPGDIKAILDKLHNLNSTTLIKEKCIEVCLEDIPAYGRDSWKLLLEDGGCPRNSGTKTEAIHHILELIELEQHEERTQLISYGRNLGLDDIIYGSDSEEEKAEE\n",
            "\n",
            "Total CPU time is  0.00 seconds\n",
            "\n",
            "TM-score: 0.16188, if lower than 0.9 run pipeline again\n",
            "Results for data/processed/evodiff_seq_1_0b87c.pdb:\n",
            "\n",
            " *********************************************************************\n",
            " * TM-align (Version 20220412): protein structure alignment          *\n",
            " * References: Y Zhang, J Skolnick. Nucl Acids Res 33, 2302-9 (2005) *\n",
            " * Please email comments and suggestions to zhanglab@zhanggroup.org   *\n",
            " *********************************************************************\n",
            "\n",
            "Name of Chain_1: /content/S2DS-Summer24-Polyploy/data/interim/test_extracted_motif.pdb (to be superimposed onto Chain_2)\n",
            "Name of Chain_2: /content/S2DS-Summer24-Polyploy/data/processed/evodiff_seq_1_0b87c.pdb\n",
            "Length of Chain_1: 22 residues\n",
            "Length of Chain_2: 172 residues\n",
            "\n",
            "Aligned length= 18, RMSD=   3.28, Seq_ID=n_identical/n_aligned= 0.000\n",
            "TM-score= 0.18496 (if normalized by length of Chain_1, i.e., LN=22, d0=0.57)\n",
            "TM-score= 0.08156 (if normalized by length of Chain_2, i.e., LN=172, d0=4.89)\n",
            "(You should use TM-score normalized by length of the reference structure)\n",
            "\n",
            "(\":\" denotes residue pairs of d <  5.0 Angstrom, \".\" denotes other aligned residues)\n",
            "------------------------------------------DKLHN-LNSNWFPEGSKPFIYQE---------------------------------------------------------------------------------------------------------------\n",
            "                                          :.::. ..:    :.::::::::                                                                                                               \n",
            "MKTVVIFNGASKHMWPGDIKAILDKLHNLNSLVELKPMPYNFTLEAGDASD----SSGTSACFVAQLFDGEKYYYNALEQQPPINYSQVKVNLGTHSLPKAEDENLGPFVPGLDGFISLVGPHEVRTRELVKEIERKHSPSAKEGNTSLKKQIEEKKEAQKLRGKQQVHGKDNSKN\n",
            "\n",
            "Total CPU time is  0.00 seconds\n",
            "\n",
            "TM-score: 0.18496, if lower than 0.9 run pipeline again\n"
          ]
        }
      ],
      "source": [
        "import re\n",
        "import os\n",
        "import subprocess\n",
        "import glob\n",
        "\n",
        "# Define the reference protein file path: it's where the motif pbd has been extracted to\n",
        "ref_protein_abs = os.path.abspath(output_pdb_file)\n",
        "\n",
        "# Use glob to find all PDB files in the omaga_fold output directory\n",
        "generated_proteins = glob.glob(os.path.join(output_path, '*.pdb'))\n",
        "\n",
        "# Loop through each generated protein file\n",
        "for generated_protein in generated_proteins:\n",
        "    generated_protein_abs = os.path.abspath(generated_protein)\n",
        "\n",
        "    # Run the TMalign command\n",
        "    tmalign_command = [\"./TMalign\", ref_protein_abs, generated_protein_abs]\n",
        "    result = subprocess.run(tmalign_command, capture_output=True, text=True)\n",
        "\n",
        "    # Print and save the results\n",
        "    if result.returncode == 0:\n",
        "        output = result.stdout\n",
        "        print(f\"Results for {generated_protein}:\")\n",
        "        print(output)\n",
        "\n",
        "        # Extract TM-scores from the output using regex\n",
        "        tm_scores = re.findall(r\"TM-score=\\s+([\\d.]+)\", output)\n",
        "        if tm_scores:\n",
        "            tm_score_1 = float(tm_scores[0])\n",
        "            tm_score_2 = float(tm_scores[1])\n",
        "            print(f\"TM-score: {tm_score_1}, if lower than 0.9 run pipeline again\")\n",
        "\n",
        "            # Use tm_score_1 as needed in subsequent commands\n",
        "        else:\n",
        "            print(\"TM-score not found in the output.\")\n",
        "    else:\n",
        "        print(f\"Error running TMalign for {generated_protein}. Return code: {result.returncode}\")\n",
        "        print(result.stderr)\n"
      ]
    }
  ],
  "metadata": {
    "accelerator": "GPU",
    "colab": {
      "gpuType": "T4",
      "provenance": []
    },
    "kernelspec": {
      "display_name": "Python 3",
      "name": "python3"
    },
    "language_info": {
      "codemirror_mode": {
        "name": "ipython",
        "version": 3
      },
      "file_extension": ".py",
      "mimetype": "text/x-python",
      "name": "python",
      "nbconvert_exporter": "python",
      "pygments_lexer": "ipython3",
      "version": "3.11.9"
    }
  },
  "nbformat": 4,
  "nbformat_minor": 0
}